<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<meta name="dcterms.date" content="2024-08-27">
<title>subject_teaching - Lab Work Organization</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>

<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script><script src="../../site_libs/kePrint-0.0.1/kePrint.js"></script><link href="../../site_libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script><script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script><link rel="stylesheet" href="../../include/styles.css">
</head>
<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="navbar navbar-expand-lg navbar-dark "><div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">PythRas</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
<li class="nav-item">
    <a class="nav-link" href="../../index.html" rel="" target="">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-r" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">R</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-r">
<li>
    <a class="dropdown-item" href="../../ch/rbasics/firststeps.html" rel="" target="">
 <span class="dropdown-text">First Steps</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/rbasics/tidyverse.html" rel="" target="">
 <span class="dropdown-text">The tidyverse</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/rbasics/correlation_regression.html" rel="" target="">
 <span class="dropdown-text">Correlation &amp; Regression</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/rbasics/baddata.html" rel="" target="">
 <span class="dropdown-text">Bad data &amp; Outliers</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-python" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Python</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-python">
<li>
    <a class="dropdown-item" href="../../ch/python/basics.html" rel="" target="">
 <span class="dropdown-text">Basics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/python/panda.html" rel="" target="">
 <span class="dropdown-text">Advanced Python</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/python/MLA.html" rel="" target="">
 <span class="dropdown-text">Machine Learning Algorithms</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/python/AI.html" rel="" target="">
 <span class="dropdown-text">AI</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-linux-and-ctl" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Linux and CTL</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-linux-and-ctl">
<li>
    <a class="dropdown-item" href="../../ch/linux-and-ctl/basics.html" rel="" target="">
 <span class="dropdown-text">Linux Basics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/linux-and-ctl/advanced.html" rel="" target="">
 <span class="dropdown-text">Advanced Command Line</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-probability-and-statistics" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Probability and Statistics</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-probability-and-statistics">
<li>
    <a class="dropdown-item" href="../../ch/prob_stat/prob_basic.html" rel="" target="">
 <span class="dropdown-text">Probability Basics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/prob_stat/stat.html" rel="" target="">
 <span class="dropdown-text">Statistics</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-genomics" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Genomics</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-genomics">
<li>
    <a class="dropdown-item" href="../../ch/genomics/basics.html" rel="" target="">
 <span class="dropdown-text">Genomics Basics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/genomics/advanced.html" rel="" target="">
 <span class="dropdown-text">Advanced Genomics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../ch/genomics/pipeline.html" rel="" target="">
 <span class="dropdown-text">Pipelines</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item">
    <a class="nav-link active" href="../../ch/phd_org/organization.html" rel="" target="" aria-current="page">
 <span class="menu-text">Lab-works</span></a>
  </li>  
</ul>
<ul class="navbar-nav navbar-nav-scroll ms-auto">
<li class="nav-item">
    <a class="nav-link" href="https://github.com/MdRasheduz-zaman" rel="" target="">
 <span class="menu-text">Contact</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/MdRasheduz-zaman/Data_Analysis" rel="" target=""><i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
<div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav></header><!-- content --><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">On this page</h2>
   
  <ul>
<li><a href="#troubleshooting-sample-preparation" id="toc-troubleshooting-sample-preparation" class="nav-link active" data-scroll-target="#troubleshooting-sample-preparation">Troubleshooting: Sample Preparation</a></li>
  <li>
<a href="#troubleshooting-rna-extraction" id="toc-troubleshooting-rna-extraction" class="nav-link" data-scroll-target="#troubleshooting-rna-extraction">Troubleshooting: RNA Extraction</a>
  <ul class="collapse">
<li><a href="#procedural-guidelines" id="toc-procedural-guidelines" class="nav-link" data-scroll-target="#procedural-guidelines">Procedural Guidelines</a></li>
  <li><a href="#lyse-samples-and-separate-phases" id="toc-lyse-samples-and-separate-phases" class="nav-link" data-scroll-target="#lyse-samples-and-separate-phases">Lyse samples and separate phases</a></li>
  </ul>
</li>
  <li>
<a href="#troubleshooting-rt-qpcr" id="toc-troubleshooting-rt-qpcr" class="nav-link" data-scroll-target="#troubleshooting-rt-qpcr">Troubleshooting: RT-qPCR</a>
  <ul class="collapse">
<li><a href="#organizational-stuffs" id="toc-organizational-stuffs" class="nav-link" data-scroll-target="#organizational-stuffs">Organizational Stuffs</a></li>
  <li><a href="#troubleshooting-reverse-transcription" id="toc-troubleshooting-reverse-transcription" class="nav-link" data-scroll-target="#troubleshooting-reverse-transcription">Troubleshooting: Reverse Transcription</a></li>
  <li><a href="#dnase-i-treatment" id="toc-dnase-i-treatment" class="nav-link" data-scroll-target="#dnase-i-treatment">DNase I treatment</a></li>
  <li><a href="#removal-of-dnase-i" id="toc-removal-of-dnase-i" class="nav-link" data-scroll-target="#removal-of-dnase-i">Removal of DNase I</a></li>
  <li><a href="#polyadenylating-using-e.-coli-polya-polymerase" id="toc-polyadenylating-using-e.-coli-polya-polymerase" class="nav-link" data-scroll-target="#polyadenylating-using-e.-coli-polya-polymerase">Polyadenylating using E. coli poly(A) polymerase</a></li>
  <li><a href="#reverse-transcription-and-strand-switching" id="toc-reverse-transcription-and-strand-switching" class="nav-link" data-scroll-target="#reverse-transcription-and-strand-switching">Reverse Transcription and Strand-switching</a></li>
  <li><a href="#rna-degradation-and-second-strand-synthesis" id="toc-rna-degradation-and-second-strand-synthesis" class="nav-link" data-scroll-target="#rna-degradation-and-second-strand-synthesis">RNA Degradation and Second Strand Synthesis</a></li>
  </ul>
</li>
  <li>
<a href="#troubleshooting-minion-sequencing" id="toc-troubleshooting-minion-sequencing" class="nav-link" data-scroll-target="#troubleshooting-minion-sequencing">Troubleshooting: MinION Sequencing</a>
  <ul class="collapse">
<li><a href="#dna-repair-and-end-prep" id="toc-dna-repair-and-end-prep" class="nav-link" data-scroll-target="#dna-repair-and-end-prep">DNA Repair and End-prep</a></li>
  <li><a href="#adapter-ligation-for-cdna" id="toc-adapter-ligation-for-cdna" class="nav-link" data-scroll-target="#adapter-ligation-for-cdna">Adapter Ligation for cDNA</a></li>
  <li><a href="#ampure-xp-bead-binding" id="toc-ampure-xp-bead-binding" class="nav-link" data-scroll-target="#ampure-xp-bead-binding">AMPure XP bead binding</a></li>
  <li><a href="#priming-and-loading-on-spoton-flowcell" id="toc-priming-and-loading-on-spoton-flowcell" class="nav-link" data-scroll-target="#priming-and-loading-on-spoton-flowcell">Priming and Loading on SpotON flowCell</a></li>
  </ul>
</li>
  <li>
<a href="#troubleshooting-sispa-protocol" id="toc-troubleshooting-sispa-protocol" class="nav-link" data-scroll-target="#troubleshooting-sispa-protocol">Troubleshooting: SISPA Protocol</a>
  <ul class="collapse">
<li><a href="#cdna-synthesis" id="toc-cdna-synthesis" class="nav-link" data-scroll-target="#cdna-synthesis">cDNA Synthesis</a></li>
  <li><a href="#double-strand-synthesis" id="toc-double-strand-synthesis" class="nav-link" data-scroll-target="#double-strand-synthesis">Double-strand synthesis</a></li>
  <li><a href="#cdna-amplification" id="toc-cdna-amplification" class="nav-link" data-scroll-target="#cdna-amplification">cDNA Amplification</a></li>
  <li><a href="#clean-up-of-cdna-amplicon" id="toc-clean-up-of-cdna-amplicon" class="nav-link" data-scroll-target="#clean-up-of-cdna-amplicon">Clean-up of cDNA amplicon</a></li>
  <li><a href="#end-prep-and-barcode-ligation" id="toc-end-prep-and-barcode-ligation" class="nav-link" data-scroll-target="#end-prep-and-barcode-ligation">End Prep and Barcode Ligation</a></li>
  <li><a href="#cleanup-of-barcoded-dna" id="toc-cleanup-of-barcoded-dna" class="nav-link" data-scroll-target="#cleanup-of-barcoded-dna">Cleanup of barcoded DNA</a></li>
  <li><a href="#sispa-vs-maxima-h-minus-rtssp-mmlv-rt" id="toc-sispa-vs-maxima-h-minus-rtssp-mmlv-rt" class="nav-link" data-scroll-target="#sispa-vs-maxima-h-minus-rtssp-mmlv-rt">SISPA vs Maxima H Minus RT/SSP (MMLV RT)</a></li>
  </ul>
</li>
  <li><a href="#troubleshooting-data-analysis" id="toc-troubleshooting-data-analysis" class="nav-link" data-scroll-target="#troubleshooting-data-analysis">Troubleshooting: Data Analysis</a></li>
  <li><a href="#troubleshooting-flowcell-count-issue" id="toc-troubleshooting-flowcell-count-issue" class="nav-link" data-scroll-target="#troubleshooting-flowcell-count-issue">Troubleshooting: FlowCell Count Issue</a></li>
  <li><a href="#troubleshooting-contamination-issue" id="toc-troubleshooting-contamination-issue" class="nav-link" data-scroll-target="#troubleshooting-contamination-issue">Troubleshooting: Contamination Issue</a></li>
  </ul><div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="organization.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block default"><div class="quarto-title">
<div class="quarto-title-block"><div><h1 class="title">Lab Work Organization</h1><button type="button" class="btn code-tools-button" id="quarto-code-tools-source"><i class="bi"></i> Code</button></div></div>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Md Rasheduzzaman </p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://www.fli.de/de/institute/institut-fuer-infektionsmedizin-imed/">
            FLI Insel Riems
            </a>
          </p>
        <p class="affiliation">
            <a href="https://github.com/MdRasheduz-zaman">
            &amp; Freelancer
            </a>
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Last updated</div>
    <div class="quarto-title-meta-contents">
      <p class="date">August 27, 2024</p>
    </div>
  </div>
  
    
  </div>
  
<div>
  <div class="abstract">
    <div class="abstract-title">Content summary</div>
    Sample prep, RNA extraction, RT-qPCR, cDNA conversion, library prep, ONT sequencing, etc.
  </div>
</div>

</header><section id="troubleshooting-sample-preparation" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-sample-preparation">Troubleshooting: Sample Preparation</h2>
<ul>
<li>Extract all trizol samples according to manufacturers protocol</li>
<li>Add glycogen (0.5µl) for the samples after the sucrose cushion (samples 7-11).</li>
<li>Determine viral RNA content through RT-qPCR.</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Comments for Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">1. Get 50 mosquitoes from insectary.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">1. Obtain the necessary number of mosquitoes for the experiment.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">1. Ensure mosquitoes are healthy and of the correct species/strain.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">2. Make 5 pools each consisting of 10 mosquitoes in 2ml tubes containing steel balls.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">2. Pooling increases sample size and reduces variability.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">2. Avoid cross-contamination between pools.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">3. Make 3 Medium aliquots containing 2ml medium each.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">3. Medium is used as a negative control.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">3. Prepare medium under sterile conditions.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">4. Add 10^4/ml particles to one 2ml aliquot -&gt; mix.</td>
<td style="text-align: left; width: 25%;">4. Create a spike-in sample with a known concentration of particles.</td>
<td style="text-align: left; width: 25%;">4. Verify particle concentration and mix thoroughly.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">5. Add 10^6/ml particles to the second 2ml aliquot -&gt; mix.</td>
<td style="text-align: left; width: 25%;">5. Create a spike-in sample with a higher concentration of particles.</td>
<td style="text-align: left; width: 25%;">5. Verify particle concentration and mix thoroughly.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">6. Add 750µl of the 10^4 solution to two pools each and 250µl of the 10^4 solution to Trizol LS (Sample 1).</td>
<td style="text-align: left; width: 25%;">6. Spike-in controls help assess extraction efficiency.</td>
<td style="text-align: left; width: 25%;">6. Pipette accurately to ensure proper spike-in concentrations.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">7. Add 750µl of the 10^6 solution to two pools each and 250µl of the 10^6 solution to Trizol LS (Sample 2).</td>
<td style="text-align: left; width: 25%;">7. Spike-in controls help assess extraction efficiency.</td>
<td style="text-align: left; width: 25%;">7. Pipette accurately to ensure proper spike-in concentrations.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">8. Add 750µl of the control medium to the last mosquito pool and 250µl of the medium to Trizol LS (Sample 3).</td>
<td style="text-align: left; width: 25%;">8. The control medium sample serves as a baseline for comparison.</td>
<td style="text-align: left; width: 25%;">8. Ensure control medium is sterile and particle-free.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">9. Store Trizol LS samples @ -80°C.</td>
<td style="text-align: left; width: 25%;">9. Trizol LS preserves RNA/DNA integrity.</td>
<td style="text-align: left; width: 25%;">9. Maintain consistent -80°C storage for all samples.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">10. Homogenize the pools 2 min @ 30 Hz.</td>
<td style="text-align: left; width: 25%;">10. Disrupts mosquito tissues and releases nucleic acids.</td>
<td style="text-align: left; width: 25%;">10. Check homogenizer settings and ensure thorough disruption.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">11. Centrifuge all samples for 3 min @ 20000g.</td>
<td style="text-align: left; width: 25%;">11. Separates cellular debris from the supernatant containing nucleic acids.</td>
<td style="text-align: left; width: 25%;">11. Use appropriate centrifuge settings and time.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">12. Take 250µl of the supernatant of the smaller control and pools and add it to Trizol LS -&gt; -80°C (Samples 4-6).</td>
<td style="text-align: left; width: 25%;">12. Collects a representative sample for later RNA/DNA extraction.</td>
<td style="text-align: left; width: 25%;">12. Avoid disturbing the pellet when collecting supernatant.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">13. Transfer the remaining supernatant to a fresh tube (for all pools).</td>
<td style="text-align: left; width: 25%;">13. Removes debris and potential inhibitors from the supernatant.</td>
<td style="text-align: left; width: 25%;">13. Transfer supernatant carefully to avoid contamination.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">14. Centrifuge for 10 min @ 20000g.</td>
<td style="text-align: left; width: 25%;">14. Further clarifies the supernatant.</td>
<td style="text-align: left; width: 25%;">14. Check centrifuge settings and time.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">14_1. Take out the plunger from a filtering syringe, and put the empty cylinder on top of a new tube.</td>
<td style="text-align: left; width: 25%;">14_1. Prepare the syringe for filtration.</td>
<td style="text-align: left; width: 25%;">14_1. Ensure the filter is compatible with the sample type.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">14_2. Transfer the sample into the empty cylinder.</td>
<td style="text-align: left; width: 25%;">14_2. Begin the filtration process.</td>
<td style="text-align: left; width: 25%;">14_2. Avoid spilling or losing sample during transfer.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">14_3. Hold the cylinder firmly, and insert the plunger. Push it to filter the sample through the filter. Some dead volume will not be passed through.</td>
<td style="text-align: left; width: 25%;">14_3. Complete the filtration step.</td>
<td style="text-align: left; width: 25%;">14_3. Apply consistent pressure to the plunger for even filtration.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">15. Transfer supernatant/filtered volume to ultracentrifuge tubes.</td>
<td style="text-align: left; width: 25%;">15. Prepare for ultracentrifugation.</td>
<td style="text-align: left; width: 25%;">15. Use appropriate ultracentrifuge tubes.<br>
These tubes are in the ultrcentrifugation room in the closet.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">16. Underlay the sample with 20% Sucrose in PBS.</td>
<td style="text-align: left; width: 25%;">16. Sucrose cushion helps to concentrate particles during ultracentrifugation.</td>
<td style="text-align: left; width: 25%;">16. Prepare sucrose solution accurately.<br>
Find out the heaviest one and add PBS to other tubes to make the same volume.<br>
Add 1.2mL PBS first to the sample and then add the 20% sucrose.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">17. Pellet particles 1h @ 483 750 g (or higher).</td>
<td style="text-align: left; width: 25%;">17. Ultracentrifugation pellets particles.</td>
<td style="text-align: left; width: 25%;">17. Use correct ultracentrifuge settings and time.<br>
We need 60WS to get 48375g.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">18. Carefully remove supernatant (best by careful decanting).</td>
<td style="text-align: left; width: 25%;">18. Removes unwanted supernatant.</td>
<td style="text-align: left; width: 25%;">18. Avoid disturbing the pellet when removing supernatant.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">19. Leave the tubes bottoms up for a few minutes to drain any fluid.</td>
<td style="text-align: left; width: 25%;">19. Ensure all supernatant is removed.</td>
<td style="text-align: left; width: 25%;">19. Complete drainage before proceeding to the next step.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">20. Add 1ml Trizol to the pellets (Samples 7-11) -&gt; store @ -80°C till processing.</td>
<td style="text-align: left; width: 25%;">20. Trizol preserves RNA/DNA in the pelleted particles.</td>
<td style="text-align: left; width: 25%;">20. Add Trizol directly to the pellet to prevent degradation.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="troubleshooting-rna-extraction" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-rna-extraction">Troubleshooting: RNA Extraction</h2>
<section id="procedural-guidelines" class="level3"><h3 class="anchored" data-anchor-id="procedural-guidelines">Procedural Guidelines</h3>
<ul>
<li>Perform all steps at room temperature (20–25°C) unless otherwise noted.<br>
</li>
<li>Use cold TRIzol™ Reagent if the starting material contains high levels of RNase, such as spleen or pancreas samples.<br>
</li>
<li>Use disposable, individually wrapped, sterile plastic ware and sterile, disposable RNase-free pipettes &amp; pipette tips, and tubes.<br>
</li>
<li>Wear disposable gloves while handling reagents and RNA samples to prevent RNase contamination from the surface of the skin; change gloves frequently, particularly as the protocol progresses from crude extracts to more purified materials.<br>
</li>
<li>Always use proper microbiological aseptic techniques when working with RNA.<br>
</li>
<li>Use RNaseZap™ RNase Decontamination Solution (Cat. no. AM9780) to remove RNase contamination from work surfaces and non-disposable items such as centrifuges and pipettes used during purification.</li>
</ul></section><section id="lyse-samples-and-separate-phases" class="level3"><h3 class="anchored" data-anchor-id="lyse-samples-and-separate-phases">Lyse samples and separate phases</h3>
<ol type="1">
<li>Lyse and homogenize samples in TRIzol™ Reagent according to your starting material.</li>
</ol>
<ul>
<li><p>Tissues:<br>
Add 1 mL of TRIzol™ Reagent per 50–100 mg of tissue to the sample and homogenize using a homogenizer.</p></li>
<li><p>Cell grown in monolayer:<br></p></li>
</ul>
<ol type="a">
<li>Remove growth media.<br>
</li>
<li>Add 0.3–0.4 mL of TRIzol™ Reagent per 1 × <span class="math inline">\(10^{5}—10^{7}\)</span> cells directly to the culture dish to lyse the cells.<br>
</li>
<li>Pipet the lysate up and down several times to homogenize.</li>
</ol>
<ul>
<li>Cells grown in suspension:<br>
</li>
</ul>
<ol type="a">
<li>Pellet the cells by centrifugation and discard the supernatant.<br>
</li>
<li>Add 0.75 mL of TRIzol™ Reagent per 0.25 mL of sample (5–10 × <span class="math inline">\(10^{6}\)</span> cells from animal, plant, or yeasty origin or 1 × <span class="math inline">\(10^{7}\)</span> cells of bacterial origin) to the pellet.<br><em>Note:</em> Do not wash cells before addition of TRIzol™ Reagent to avoid mRNA degradation.<br>
</li>
<li>Pipette the lysate up and down several times to homogenize.<br><strong>Note:</strong> The sample volume should not exceed 10% of the volume of TRIzol™ Reagent used for lysis.</li>
</ol>
<p><strong>STOPPING POINT</strong> Samples can be stored at 4°C overnight or at –20°C for up to a year.</p>
<ol start="2" type="1">
<li>
<strong>(Optional)</strong> If samples have a high fat content, centrifuge the lysate for 5 minutes at 12,000 g at 4–10°C, then transfer the clear supernatant to a new tube.</li>
</ol>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Comments for Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">3. Incubate for 5 minutes to permit complete dissociation of the nucleoproteins complex.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">3. Incubation allows the denaturing agents in TRIzol to disrupt cellular structures and dissolve membranes.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">3. If complete dissociation doesn't occur, increase incubation time or use a higher concentration of TRIzol.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">4. Add 0.2 mL of chloroform per 1 mL of TRIzol™ Reagent used for lysis, then securely cap the tube.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">4. Chloroform further denatures proteins and promotes phase separation.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">4. Ensure proper mixing of chloroform and TRIzol. Vortex briefly after adding chloroform.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%; background-color: #f2f2f2 !important;">5. Incubate for 2–3 minutes.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">5. This step enhances phase separation and ensures thorough mixing.</td>
<td style="text-align: left; width: 25%; background-color: #f2f2f2 !important;">5. Check sample temperature to ensure it's close to room temperature.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 50%;">6. Centrifuge the sample for 15 minutes at 12,000 × g at 4°C. The mixture separates into a lower red phenol-chloroform, and interphase, and a colorless upper aqueous phase.</td>
<td style="text-align: left; width: 25%;">6. Centrifugation causes the sample to separate into distinct layers based on density.</td>
<td style="text-align: left; width: 25%;">6. Ensure correct centrifuge speed and time. Incomplete separation may indicate issues with centrifugation.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 50%;">7. Transfer the aqueous phase containing the RNA to a new tube by angling the tube at 45° and pipette the solution out.</td>
<td style="text-align: left; width: 25%;">7. The aqueous phase contains RNA, while DNA and proteins are in the interphase and organic phase, respectively.</td>
<td style="text-align: left; width: 25%;">7. Use a micropipette with a wide-bore tip to avoid shearing RNA. Be careful not to disturb the interphase.</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>IMPORTANT!</strong> Avoid transferring any of the interphase or organic layer into the pipette when removing the aqueous phase. <br></p>
<p>Proceed directly to “Isolate RNA” on page 2.<br></p>
<p>Save the interphase and organic phase if you want to isolate DNA or protein. See “Isolate DNA” on page 3 or “Isolate proteins” on page 4 for detailed procedures. The organic phase can be stored at 4°C overnight.</p>
</section></section><section id="troubleshooting-rt-qpcr" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-rt-qpcr">Troubleshooting: RT-qPCR</h2>
<ul>
<li>I will make a shiny app for the calculation online for any number of <code>#sample</code>.</li>
</ul>
<p><strong>N.B.</strong> For the NTC, we need everything except the RNA sample (use RNase-free water instead of sample).</p>
<ul>
<li>Using <code>WNV-Ag-ID-Path.prcl</code> One-Step RT-PCR protocol.</li>
</ul>
<div class="column" style="width:90%;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="../../img/RTqPCR_info.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">RT-qPCR Run Info</figcaption></figure>
</div>
</div>
<ul>
<li>There are 6 steps in the RT-qPCR. Here goes the mechanistic details of the steps:</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 50%">
<col style="width: 50%">
</colgroup>
<thead><tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Steps</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Mechanism</th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Step 1: Reverse Transcription (RT)</td>
<td style="text-align: left; width: 80%; background-color: #f2f2f2 !important;">The reverse transcriptase enzyme is active at this temperature, converting the RNA template into complementary DNA (cDNA). This is a crucial step as qPCR amplifies DNA, not RNA.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Step 2: Initial Denaturation and Enzyme Activation</td>
<td style="text-align: left; width: 80%; background-color: #f2f2f2 !important;">The high temperature serves two purposes:<br>
Denaturation: Separates double-stranded RNA/DNA into single strands, making them accessible for primers.<br>
Hot-start activation: Activates the hot-start Taq polymerase, which prevents non-specific amplification at lower temperatures.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 20%;">Step 3: Denaturation</td>
<td style="text-align: left; width: 80%;">Repeated in each cycle, this step ensures complete separation of newly synthesized DNA strands, allowing primers to bind in the next step.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 20%;">Step 4: Annealing</td>
<td style="text-align: left; width: 80%;">The primers, designed to be complementary to specific regions of the target DNA, bind (anneal) to their respective targets. Lowering the temperature facilitates this binding.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 20%;">Step 5: Extension</td>
<td style="text-align: left; width: 80%;">The Taq polymerase extends the primers, synthesizing new complementary DNA strands. This results in the doubling of the target DNA in each cycle.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 20%;">Step 6: Data Acquisition and Analysis</td>
<td style="text-align: left; width: 80%;">The duration varies (usually a brief step at a temperature between annealing and extension)<br>
FAM Fluorescence: The machine measures the fluorescence of the FAM dye, which is typically incorporated into the amplifying DNA (either through a probe or intercalating dye like SYBR Green). The FAM signal increases proportionally with the amount of target DNA.<br>
ROX Fluorescence: The ROX dye acts as a passive reference. Its fluorescence should remain constant throughout the run. The machine uses the ROX signal to normalize the FAM signal, correcting for well-to-well variations and potential errors in pipetting.</td>
</tr>
</tbody>
</table>
</div>
</div>
<section id="organizational-stuffs" class="level3"><h3 class="anchored" data-anchor-id="organizational-stuffs">Organizational Stuffs</h3>
<ul>
<li>Reagents are in -20°C fridge (opposite of -80°C fridge).<br>
</li>
<li>Turn on all the switches on the sample hood and rotate the key to turn on. Take the slash up (folded).<br>
</li>
<li>Look if the scissor is there to cut your plate and sealing accordingly.<br>
</li>
<li>Print out the design and keep there to organize the Master Mix (MM) and samples accordingly. Use FAM and ROX for this experiment.<br>
</li>
<li>Keep the enzyme mix always on the cooling block till use.<br>
</li>
<li>Take gloves.<br>
</li>
<li>The cooling blocks change color depending on how hot they are.<br>
</li>
<li>Take 10µL, 20µL, 100µL, 200µL tip boxes if needed. Pipettes are there already inside the hoods.<br>
</li>
<li>Samples on ice.<br>
</li>
<li>Take 1 black box and take ice in it (from downstairs, left hand side while entering). Put the ice boxes in it.<br>
</li>
<li>Take another black box with plate &amp; sealing (touch only with gloves).<br>
</li>
<li>Take 2 Eppendorf tube holders, one for sample and one for H2O, Buffer.<br>
</li>
<li>Touch the RNA sample and reagents only with gloves.<br>
</li>
<li>Keep FAM and Enzyme inside the black box, because it is color sensitive.<br>
</li>
<li>Take the small white box to keep the well plate (with sealing) on the ice box inside it to remove potential contamination issue. Take out when needed after making the MM.<br>
</li>
<li>Clean the MM hood and Sample hood with alcohol first and then start working/putting things inside.<br>
</li>
<li>The centrifuge machine for well plate works this way: Turn on –&gt; Place the plate –&gt; Run for 10-20 secs –&gt; Take out –&gt; Turn off<br>
</li>
<li>Use the first point to pipette off the 11.5 µL MM, don’t push too much. A bit will remain in the tip. That’s fine. It will be the same for all the wells. No air bubble (aerosol) will be formed this way.</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<thead><tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Consumables</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Location</th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 65%; background-color: #f2f2f2 !important;">Cooling blocks (blue and yellow) for PCR tubes and reagents</td>
<td style="text-align: left; width: 35%; background-color: #f2f2f2 !important;">Shelf XIV, middle room</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 65%; background-color: #f2f2f2 !important;">RNase-free water</td>
<td style="text-align: left; width: 35%; background-color: #f2f2f2 !important;">Rack A, Box 3</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 65%;">2x RT-PCR buffer (has things for ROX)</td>
<td style="text-align: left; width: 35%;">Rack A, Box 3</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 65%;">25x RT-PCR enzyme mix (Rev Transcriptase + Pol)</td>
<td style="text-align: left; width: 35%;">Rack A, Box 3</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 65%;">FAM mix (Primer + Probe)</td>
<td style="text-align: left; width: 35%;">Rack B2, Box 2</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 65%;">96-Well-Plate + sealing</td>
<td style="text-align: left; width: 35%;">Top-right drawer of RNA side</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 65%;">RNA Sample(s)</td>
<td style="text-align: left; width: 35%;">-80°C fridge (Rack N, Box 4)</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 65%;">A waste bag</td>
<td style="text-align: left; width: 35%;">You know where</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 65%;">Scissors</td>
<td style="text-align: left; width: 35%;">Inside MM hood or you take</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>No.&nbsp;of sample = depends on your experiment &amp; used in the equation to calculate the total volume of MM.<br>
</li>
<li>Take the waste bag out of MM hood to the sample hood, never the other way around.<br>
</li>
<li>Design the well-plate beforehand using correct fluorophores (FAM/ROX), standard samples, concentration of the standards, unknown and/or negative samples, NTC, biological/technical replicates, etc.<br>
</li>
<li>Prepare the reaction mix following this exact order. <strong>&amp; See the points on precautions below:</strong>
</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Reagent
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
1x (µL)
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
38x (µL)
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanism
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">RNase-free water (100µL aliquot)</td>
<td style="text-align: right; width: 15%; background-color: #f2f2f2 !important;">3.75</td>
<td style="text-align: left; width: 15%; background-color: #f2f2f2 !important;">142.5</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">To make the correct volume</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2x RT-PCR buffer (130µL aliquot?)</td>
<td style="text-align: right; width: 15%; background-color: #f2f2f2 !important;">6.25</td>
<td style="text-align: left; width: 15%; background-color: #f2f2f2 !important;">237.5</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Has buffering activity and ROX for internal control</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">FAM mix (10µL aliquot)</td>
<td style="text-align: right; width: 15%; background-color: #f2f2f2 !important;">1.00</td>
<td style="text-align: left; width: 15%; background-color: #f2f2f2 !important;">38</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Has primer pairs and probe for amplification and fluorophore detection, respectively</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">25x RT-PCR enzyme mix (10µL aliquot)</td>
<td style="text-align: right; width: 15%;">0.50</td>
<td style="text-align: left; width: 15%;">19</td>
<td style="text-align: left; width: 40%;">Has the polymerase for amplification</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">Total vol of MM</td>
<td style="text-align: right; width: 15%;">11.50</td>
<td style="text-align: left; width: 15%;">437</td>
<td style="text-align: left; width: 40%;">X</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">RNA template (20µL)</td>
<td style="text-align: right; width: 15%;">1.00</td>
<td style="text-align: left; width: 15%;">x</td>
<td style="text-align: left; width: 40%;">Inital element/RNA for RT-qPCR</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">Total vol of reaction mix</td>
<td style="text-align: right; width: 15%;">12.50</td>
<td style="text-align: left; width: 15%;">475</td>
<td style="text-align: left; width: 40%;">X</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Cut the plate and the sealing accordingly using the scissors. Name/date &amp; orientation should be written.<br>
</li>
<li>Turn on air circulation, &amp; white light in the MM hood.<br>
</li>
<li>Vortex everything (MM, samples) before adding (so, prep the MM in a Eppendorf tube).<br>
</li>
<li>
<strong>MM handling:</strong> Put the MM in each and every well carefully. Don’t release the pipette inside the well or near the plate. Keep it held. Release only on the waste bag and discard. Otherwise, you might take something from the well and make aerosol (contaminant).<br>
</li>
<li>
<strong>Sample handling:</strong> Don’t press hard (second point), pipette multiple times and take carefully. Then put in the well carefully. Again follow the same procedure of pipetting.<br>
</li>
<li>Put on the sealing nicely; otherwise aerosol will form and disperse.<br>
</li>
<li>Clean, turn on the UV light in th MM hood.<br>
</li>
<li>Clean, turn off the switches in the sample hood. Turn on the UV. Go back after 20-30 mins to turn of the UV (for the sample hood, MM hood is automatic). Turn off using the key.<br>
</li>
<li>Centrifuge the plate in the other lab (Short pulse for 20 secs). <strong>Procedure:</strong> Open lid –&gt; Place on the plate –&gt; Switch on –&gt; Close lid –&gt; Hold the button for “Impuls” for 20 secs –&gt; Take out and turn off<br>
</li>
<li>RT-qPCR running: Open the program –&gt; Click on “open lid” –&gt; place on the well-plate as designed –&gt; Put a balance for even heat circulation (if not using the full plate) –&gt; Click on “close lid” –&gt; Choose the right options.<br>
</li>
<li>Run the RT-qPCR using the right protocol (<code>WNV-Ag-ID-Path.prcl</code>) and set the right location for saving the result.<br>
</li>
<li>Open the result using Bio-Rad CFX Maestro and explore!</li>
</ul></section><section id="troubleshooting-reverse-transcription" class="level3"><h3 class="anchored" data-anchor-id="troubleshooting-reverse-transcription">Troubleshooting: Reverse Transcription</h3>
<ul>
<li>
<a href="https://www.tandfonline.com/doi/pdf/10.2144/01304pf02">The main paper</a> should help us understanding and designing the primers. Primer (1) and (3) are needed.</li>
</ul>
<iframe src="../../pdf/cDNA_MMLV.pdf" width="100%" height="600px">
</iframe>
<p>The ONT protocol for direct cDNA sequencing also provides with the information about the primers required.</p>
<div class="column" style="width:90%;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="../../img/cDNA_ONT.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">cDNA conversion by ONT</figcaption></figure>
</div>
</div>
<p><a href="https://eu.idtdna.com/pages/education/decoded/article/use-of-template-switching-oligos-(ts-oligos-tsos)-for-efficient-cdna-library-construction">Another website</a> is quite good for understanding as well.</p>
<p><strong>Here goes the step-by-step mechanism.</strong></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="../../Video/cDNA_mechanism.gif" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">cDNA conversion mechanism</figcaption></figure>
</div>
</section><section id="dnase-i-treatment" class="level3"><h3 class="anchored" data-anchor-id="dnase-i-treatment">DNase I treatment</h3>
</section><section id="removal-of-dnase-i" class="level3"><h3 class="anchored" data-anchor-id="removal-of-dnase-i">Removal of DNase I</h3>
</section><section id="polyadenylating-using-e.-coli-polya-polymerase" class="level3"><h3 class="anchored" data-anchor-id="polyadenylating-using-e.-coli-polya-polymerase">Polyadenylating using E. coli poly(A) polymerase</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Start with clean, non-polyadenylated RNA of interest. The sample should be EDTA-free, with no salts present.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures that the RNA sample is free of contaminants that could interfere with the polyadenylation reaction.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">If the RNA sample contains EDTA or salts, perform a cleanup step using ethanol precipitation or a commercial RNA purification kit.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. In a 1.5 ml microcentrifuge tube, set up 3’ polyadenylation reaction as follows:<br>
* Non-polyadenylated RNA X μl ≤ 10 μg<br>
* 10X E. coli poly(A) polymerase buffer 2 μl 1X<br>
* ATP (10 mM) 2 μl 1 mM<br>
* Nuclease-free water 15-X μl<br>
* E. coli poly(A) polymerase (5 U/μl) 1 μl 5U<br>
* TOTAL 20 μl</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Combines the RNA template with the necessary reagents for polyadenylation, including the enzyme, buffer, ATP (the source of adenosine nucleotides), and nuclease-free water to adjust the final volume.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure accurate pipetting, especially with small volumes. Use high-quality, RNase-free reagents.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Incubate at 37°C for 0.5–1.5 minutes.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Provides the optimal temperature for the E. coli poly(A) polymerase enzyme to add adenosine nucleotides to the 3' end of the RNA molecules.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure accurate temperature control and timing during incubation. Longer incubation times will result in longer poly(A) tails, but may decrease overall sequencing yield.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. After 0.5–1.5 minutes of incubation, stop the reaction by adding 5 μl of 50 mM EDTA (to a final concentration of 10 mM EDTA). Final volume will be 25 μl.</td>
<td style="text-align: left; width: 40%;">EDTA chelates divalent cations (like Mg2+), which are essential for the polymerase enzyme's activity, effectively stopping the reaction.</td>
<td style="text-align: left; width: 20%;">Ensure thorough mixing after adding EDTA to stop the reaction completely.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. Add 45 μl of RNase-free SPRI beads to the reaction.</td>
<td style="text-align: left; width: 40%;">AMPure XP beads selectively bind to RNA, allowing for purification and removal of contaminants like enzymes, salts, and unincorporated nucleotides.</td>
<td style="text-align: left; width: 20%;">Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Incubate on a Hula Mixer for 5 minutes at room temperature.</td>
<td style="text-align: left; width: 40%;">Mixing promotes efficient binding of RNA to the beads.</td>
<td style="text-align: left; width: 20%;">Use a thermomixer or rotator for consistent mixing.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Spin down the sample and pellet on a magnet. Keep the tube on the magnet and pipette off and discard the supernatant.</td>
<td style="text-align: left; width: 40%;">Magnetic separation facilitates removal of unwanted components in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and avoid disturbing the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. Keep the tube on the magnet and wash the beads with 200 μl of freshly-prepared 70% ethanol. Carefully turn the tube 180° twice in the rack to wash pelleted beads. Pippette off and discard the supernatant.</td>
<td style="text-align: left; width: 40%;">Ethanol washes remove residual contaminants and salts from the beads.</td>
<td style="text-align: left; width: 20%;">Use fresh 70% ethanol. Carefully turn the tube to ensure all beads are washed.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Repeat the previous step for a total of two washes.</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 20%;">Repeat washing if the supernatant is still cloudy.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Briefly spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for 30 seconds, but do not dry the pellet to the point of cracking.</td>
<td style="text-align: left; width: 40%;">Removes any remaining ethanol, which can interfere with downstream steps.</td>
<td style="text-align: left; width: 20%;">Avoid over-drying the beads, which can make elution difficult.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">11. Remove the tube from the magnetic rack and resuspend the pellet in 12 μl of nuclease-free water. Incubate on ice for 5 minutes.</td>
<td style="text-align: left; width: 40%;">Elution buffer releases the purified RNA from the beads. Incubation on ice helps maintain RNA integrity.</td>
<td style="text-align: left; width: 20%;">Use nuclease-free water for elution. Ensure complete resuspension of the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">12. Pellet the beads on a magnet until the eluate is clear and colourless.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified RNA in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">13. Remove and retain 12 μl of eluate containing the 3’-polyadenylated RNA in a clean 1.5 ml microcentrifuge tube.</td>
<td style="text-align: left; width: 40%;">Collects the purified, polyadenylated RNA for further analysis or downstream applications.</td>
<td style="text-align: left; width: 20%;">Avoid transferring any beads with the eluate.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">14. Remove 1 μl of final eluate and quantify the final concentration using HS RNA Qubit kit.</td>
<td style="text-align: left; width: 40%;">Measures the concentration of the purified, polyadenylated RNA to assess yield and quality.</td>
<td style="text-align: left; width: 20%;">Ensure accurate calibration of the Qubit fluorometer. If yield is low, troubleshoot previous steps.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">Optional step: Dilute the final eluate to 100 ng/μl and quantify 1 μl using Agilent 6000 Nano RNA kit, using total RNA or mRNA setting depending on the sample.</td>
<td style="text-align: left; width: 40%;">Provides additional information about RNA size distribution and integrity.</td>
<td style="text-align: left; width: 20%;">Ensure proper chip preparation and instrument calibration for the Agilent Bioanalyzer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">15. Store the final product at -80°C.</td>
<td style="text-align: left; width: 40%;">Long-term storage preserves the RNA for future use.</td>
<td style="text-align: left; width: 20%;">Maintain consistent -80°C storage to prevent RNA degradation.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="reverse-transcription-and-strand-switching" class="level3"><h3 class="anchored" data-anchor-id="reverse-transcription-and-strand-switching">Reverse Transcription and Strand-switching</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Prepare the following reaction in a 0.2 ml PCR tube:<br>
x μl poly A+ RNA, 100 ng<br>
2.5 μl VNP<br>
1 μl 10 mM dNTPs<br>
7.5-x μl RNase-free water</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Combines mRNA template, VNP (a template switching oligo), dNTPs, and water. VNP anneals to the polyA tail of mRNA and provides a template for reverse transcription.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure accurate quantification and integrity of RNA. Use high-quality, RNase-free water.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. Mix gently by flicking the tube, and spin down</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures all components are well mixed and collects any condensation on the tube walls.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Avoid vigorous mixing, which can shear RNA.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Incubate at 65°C for 5 minutes and then snap cool on a pre-chilled freezer block for 1 minute</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">High temperature denatures RNA secondary structures. Snap cooling prevents re-formation of secondary structures and promotes annealing of VNP to the polyA tail.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Use a pre-chilled freezer block for efficient snap cooling.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. In a separate tube, mix together the following:<br>
4 μl 5x RT Buffer<br>
1 μl RNaseOUT<br>
1 μl Nuclease-free water<br>
2 μl Strand-Switching Primer (SSP)</td>
<td style="text-align: left; width: 40%;">Prepares the reverse transcription and strand-switching buffer containing the necessary reagents.</td>
<td style="text-align: left; width: 20%;">Prepare the buffer fresh and keep it on ice. Ensure RNase-free conditions.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. Mix gently by flicking the tube, and spin down.</td>
<td style="text-align: left; width: 40%;">Ensures thorough mixing of components.</td>
<td style="text-align: left; width: 20%;">Avoid vigorous mixing, which can shear RNA.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Add the strand-switching buffer to the snap-cooled, annealed mRNA, mix by flicking the tube and spin down.</td>
<td style="text-align: left; width: 40%;">Combines the mRNA template with the RT and strand-switching buffer.</td>
<td style="text-align: left; width: 20%;">Mix gently to avoid disrupting the annealed mRNA/VNP complex.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Incubate at 42°C for 2 minutes in the thermal cycler.</td>
<td style="text-align: left; width: 40%;">Allows the SSP to anneal to the mRNA/cDNA hybrid.</td>
<td style="text-align: left; width: 20%;">Ensure accurate temperature and timing in the thermal cycler.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. Add 1 µl of Maxima H Minus Reverse Transcriptase. The total volume is now 20 µl.</td>
<td style="text-align: left; width: 40%;">Adds the reverse transcriptase enzyme.</td>
<td style="text-align: left; width: 20%;">Keep the enzyme on ice until use. Avoid pipetting errors.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Mix gently by flicking the tube, and spin down.</td>
<td style="text-align: left; width: 40%;">Ensures proper mixing of the enzyme with the reaction mixture.</td>
<td style="text-align: left; width: 20%;">Avoid vigorous mixing, which can shear the cDNA.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Incubate using the following protocol using a thermal cycler:<br>
Reverse transcription and strand-switching 90 mins @ 42°C (1 cycle)<br>
Heat inactivation 5 mins @ 85°C (1 cycle)<br>
Hold @ 4°C</td>
<td style="text-align: left; width: 40%;">Reverse transcriptase synthesizes cDNA from the mRNA template, and the strand-switching activity of the enzyme generates the second cDNA strand. Heat inactivation denatures the enzyme and stops the reaction.</td>
<td style="text-align: left; width: 20%;">Ensure accurate temperature and timing in the thermal cycler.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="rna-degradation-and-second-strand-synthesis" class="level3"><h3 class="anchored" data-anchor-id="rna-degradation-and-second-strand-synthesis">RNA Degradation and Second Strand Synthesis</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Add 1 µl RiboShredder or RNase Cocktail Enzyme Mix (ThermoFisher, cat # AM2286) to the reverse transcription reaction.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Degrades any remaining RNA, preventing it from interfering with downstream applications.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Use fresh RNase enzyme. Ensure complete RNA degradation by checking for absence of RNA bands on a gel.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. Incubate the reaction for 10 minutes at 37° C in a thermal cycler.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Provides optimal temperature for RNase activity.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Verify accurate temperature control in the thermal cycler.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Resuspend the AMPure XP beads by vortexing.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures consistent bead suspension for efficient binding of cDNA.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Vortex thoroughly until no clumps remain.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. Transfer the sample to a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Transfers the sample to a tube compatible with magnetic separation.</td>
<td style="text-align: left; width: 20%;">Avoid contamination by using clean tubes and pipette tips.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. Add 17 µl of resuspended AMPure XP beads to the reaction and mix by flicking the tube.</td>
<td style="text-align: left; width: 40%;">AMPure XP beads selectively bind to cDNA, allowing for purification.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of beads for the sample volume. Mix gently to avoid foaming.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Mixing promotes efficient binding of cDNA to the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water.</td>
<td style="text-align: left; width: 40%;">Prepares ethanol for washing the beads.</td>
<td style="text-align: left; width: 20%;">Prepare fresh ethanol and use high-quality, nuclease-free water.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant.</td>
<td style="text-align: left; width: 40%;">Magnetic beads are attracted to the magnet, separating them from the supernatant.</td>
<td style="text-align: left; width: 20%;">Ensure the magnet is strong enough to hold the beads. Avoid disturbing the pellet when removing supernatant.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet. Remove the ethanol using a pipette and discard.</td>
<td style="text-align: left; width: 40%;">Ethanol wash removes impurities and salts from the beads.</td>
<td style="text-align: left; width: 20%;">Use fresh 70% ethanol and pipette gently to avoid disturbing the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Repeat the previous step.</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 20%;">Repeat washing if the supernatant is still cloudy.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">11. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking.</td>
<td style="text-align: left; width: 40%;">Removes any remaining ethanol.</td>
<td style="text-align: left; width: 20%;">Avoid over-drying the beads, which can make elution difficult.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">12. Remove the tube from the magnetic rack and resuspend pellet in 20 µl Nuclease-free water.</td>
<td style="text-align: left; width: 40%;">Elute the purified cDNA from the beads.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of nuclease-free water. Ensure complete resuspension.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">13. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Allows the cDNA to rehydrate and detach from the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">14. Pellet the beads on the magnet until the eluate is clear and colourless.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified cDNA in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">15. Remove and retain 20 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Collects the purified cDNA for downstream applications.</td>
<td style="text-align: left; width: 20%;">Avoid transferring any beads with the eluate.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">16. Prepare the following reaction in a 0.2 ml thin-walled PCR tube:<br>
&nbsp; 25 μl 2x LongAmp Taq Master Mix<br>
&nbsp; 2 μl PR2 Primer, 10 μM<br>
&nbsp; 20 μl Reverse-transcribed sample from above<br>
&nbsp; 3 μl Nuclease-free water</td>
<td style="text-align: left; width: 40%;">Sets up the PCR reaction with LongAmp Taq polymerase, primers, and the cDNA template.</td>
<td style="text-align: left; width: 20%;">Prepare the PCR master mix carefully. Avoid pipetting errors and ensure proper primer concentrations.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">17. Incubate using the following protocol:<br>
94°C<br>
- Time: 1 mins<br>
- No. of cycles: 1<br>
50°C<br>
- Time: 1 mins<br>
- No. of cycles: 1<br>
65°C<br>
- Time: 15 mins<br>
- No. of cycles: 1<br>
4°C<br>
- Time: ∞</td>
<td style="text-align: left; width: 40%;">Initial denaturation, primer annealing, extension (second strand synthesis), and final hold.</td>
<td style="text-align: left; width: 20%;">Verify correct thermal cycler settings. Check for amplification using gel electrophoresis.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">18. Resuspend the AMPure XP beads by vortexing.</td>
<td style="text-align: left; width: 40%;">Resuspends the AMPure XP beads for purification.</td>
<td style="text-align: left; width: 20%;">Vortex thoroughly until no clumps remain.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">19. Transfer the sample to a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Transfers the sample to a tube compatible with magnetic separation.</td>
<td style="text-align: left; width: 20%;">Avoid contamination by using clean tubes and pipette tips.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">20. Add 40 µl of resuspended AMPure XP beads to the reaction and mix by flicking the tube.</td>
<td style="text-align: left; width: 40%;">AMPure XP beads bind to the amplified cDNA, allowing for purification.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of beads for the sample volume. Mix gently to avoid foaming.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">21. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Mixing promotes efficient binding of cDNA to the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">22. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water.</td>
<td style="text-align: left; width: 40%;">Prepares ethanol for washing the beads.</td>
<td style="text-align: left; width: 20%;">Prepare fresh ethanol and use high-quality, nuclease-free water.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">23. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, separating them from the supernatant.</td>
<td style="text-align: left; width: 20%;">Ensure the magnet is strong enough to hold the beads. Avoid disturbing the pellet when removing supernatant.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">24. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet.</td>
<td style="text-align: left; width: 40%;">Ethanol wash removes impurities and salts from the beads.</td>
<td style="text-align: left; width: 20%;">Use fresh 70% ethanol and pipette gently to avoid disturbing the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">25. Remove the ethanol using a pipette and discard.</td>
<td style="text-align: left; width: 40%;">Removes the ethanol wash.</td>
<td style="text-align: left; width: 20%;">Ensure complete removal of ethanol.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">26. Repeat the previous step.</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 20%;">Repeat washing if the supernatant is still cloudy.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">27. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking.</td>
<td style="text-align: left; width: 40%;">Removes any remaining ethanol.</td>
<td style="text-align: left; width: 20%;">Avoid over-drying the beads, which can make elution difficult.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">28. Remove the tube from the magnetic rack and resuspend pellet in 21 µl Nuclease-free water.</td>
<td style="text-align: left; width: 40%;">Elute the purified double-stranded cDNA from the beads.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of nuclease-free water. Ensure complete resuspension.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">29. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Allows the cDNA to rehydrate and detach from the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">30. Pellet the beads on the magnet until the eluate is clear and colourless.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified cDNA in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">31. Remove and retain 21 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Collects the purified cDNA for downstream applications.</td>
<td style="text-align: left; width: 20%;">Avoid transferring any beads with the eluate.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">32. Analyse 1 µl of the strand-switched DNA for size, quantity and quality.</td>
<td style="text-align: left; width: 40%;">Assess the size distribution, concentration, and integrity of the cDNA to ensure it's suitable for further analysis.</td>
<td style="text-align: left; width: 20%;">Use appropriate methods for analysis, such as Bioanalyzer or gel electrophoresis.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section></section><section id="troubleshooting-minion-sequencing" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-minion-sequencing">Troubleshooting: MinION Sequencing</h2>
<p>I am basically doing gDNA and cDNA sequencing. There are some subtle differences in the protocols. Please, refer to the ONT guidelines for details. I am attaching the pdf here though. The reagents highlighted yellow are different in the 2 protocols. Check them out below.</p>
<p>For gDNA sequencing using SQK-LSK109:<br><iframe src="../../pdf/SQK-LSK109-XL.pdf" width="100%" height="600px"></iframe></p>
<p>For polyA tailing:<br><iframe src="../../pdf/3-poly-rna-ecoli-pap.pdf" width="100%" height="600px"></iframe></p>
<p>For direct cDNA sequencing using SQK-DCS109:<br><iframe src="../../pdf/cDNA_SQK-DCS109.pdf" width="100%" height="600px"></iframe></p>
<p><strong>The cDNA sequencing kit uses the same Ligation sequencing kit (SQK-LSK109) + Priming kit (EXP-FLP002). But the adapter mix could be different here (inherently) to ease addition of the adapters to the ds-cDNA sequences.</strong></p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<thead><tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Reagent</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Function</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Protocol</th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">NEBNext FFPE Repair Mix (NEB, M6630)</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Repair the end of the dsDNA. Not needed for cDNA</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">gDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Adapter Mix (AMX) and Quick T4 Ligase</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">T4 DNA ligase lygate the adapter mix</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">gDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">E. coli Poly(A) Polymerase (NEB: M0276L)</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Adds poly-A tail</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">cDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">50 mM RNase-free EDTA (Invitrogen: AM9260G)</td>
<td style="text-align: left; width: 40%;">Stops poly-A tailing after the required time period</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">NEB Blunt/TA Ligase Master Mix (M0367)</td>
<td style="text-align: left; width: 40%;">Ligate the dA tail</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10 mM dNTP solution (e.g. NEB N0447)</td>
<td style="text-align: left; width: 40%;">Nucleotides to be added in the cDNA strands</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">LongAmp Taq 2X Master Mix (e.g. NEB M0287)</td>
<td style="text-align: left; width: 40%;">Polymerase for 2nd strand synthesis</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">Maxima H Minus Reverse Transcriptase (200 U/μl) with 5x RT Buffer (ThermoFisher, cat # EP0751)</td>
<td style="text-align: left; width: 40%;">For 1st strand synthesis</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">RNaseOUTTM, 40 U/μl (Life Technologies, cat # 10777019)</td>
<td style="text-align: left; width: 40%;">To protect RNA template from any RNase</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">RiboShredder (Epicentre, cat # RS12500), or RNase Cocktail Enzyme Mix (ThermoFisher, cat # AM2286)</td>
<td style="text-align: left; width: 40%;">To degrade the RNA template before the 2nd strand synthesis</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">2.5 μl VNP/DTT</td>
<td style="text-align: left; width: 40%;">As a primer for the 1st strand synthesis, complementary to the poly-A tail in the RNA</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">2 μl Strand-Switching Primer (SSP)</td>
<td style="text-align: left; width: 40%;">Required for the Maxima H RT (modified MMLV) for full length+ 1st strand synthesis</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">2 μl PR2 Primer, 10 μM</td>
<td style="text-align: left; width: 40%;">Primer to initiate the 2nd strand synthesis</td>
<td style="text-align: left; width: 20%;">cDNA seq</td>
</tr>
</tbody>
</table>
</div>
</div>
<section id="dna-repair-and-end-prep" class="level3"><h3 class="anchored" data-anchor-id="dna-repair-and-end-prep">DNA Repair and End-prep</h3>
<section id="end-prep-for-cdna-sequencing" class="level4"><h4 class="anchored" data-anchor-id="end-prep-for-cdna-sequencing">End-prep for cDNA sequencing</h4>
<p><strong>IMPORTANT</strong> If you have prepared your own cDNA instead of performing reverse transcription using the Direct cDNA Sequencing Kit, please start this step with 70-200 ng cDNA in 20 µl Nuclease-free water.</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Perform end repair and dA-tailing of the cDNA sample as follows:<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;20 µl cDNA sample<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;30 µl Nuclease-free water<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;7 µl Ultra II End-prep reaction buffer<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;3 µl Ultra II End-prep enzyme mix</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Repairs any damaged DNA ends and adds an adenosine (A) nucleotide to the 3' ends of the cDNA, preparing it for ligation to sequencing adapters.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure proper enzyme activity and storage. Use high-quality reagents and follow the manufacturer's instructions.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. Mix gently by pipetting and spin down.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures proper mixing of reagents.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Avoid vigorous mixing, which can damage DNA.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Using a thermal cycler, incubate at 20°C for 5 minutes and 65°C for 5 minutes.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Provides optimal temperature for enzymatic activity: 20°C for end repair and 65°C for A-tailing.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure accurate temperature control and timing in the thermal cycler.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. Resuspend the AMPure XP beads by vortexing.</td>
<td style="text-align: left; width: 40%;">Ensures consistent bead suspension for efficient binding of cDNA.</td>
<td style="text-align: left; width: 20%;">Vortex thoroughly until no clumps remain.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. Transfer the sample to a 1.5 ml DNA LoBind Eppendorf tube.</td>
<td style="text-align: left; width: 40%;">Transfers the sample to a tube compatible with magnetic separation.</td>
<td style="text-align: left; width: 20%;">Use a clean tube to avoid contamination.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Add 60 µl of resuspended AMPure XP beads to the end-prep reaction and mix by pipetting.</td>
<td style="text-align: left; width: 40%;">AMPure XP beads bind to the cDNA, allowing for purification and removal of excess reagents.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of beads. Mix gently to avoid foaming.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Mixing promotes efficient binding of cDNA to the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water.</td>
<td style="text-align: left; width: 40%;">Prepares ethanol for washing the beads.</td>
<td style="text-align: left; width: 20%;">Prepare fresh ethanol and use high-quality nuclease-free water.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant.</td>
<td style="text-align: left; width: 40%;">Magnetic beads are attracted to the magnet, separating them from the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and avoid disturbing the pellet when removing supernatant.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet. Remove the ethanol using a pipette and discard.</td>
<td style="text-align: left; width: 40%;">Ethanol wash removes impurities and salts from the beads.</td>
<td style="text-align: left; width: 20%;">Use fresh 70% ethanol. Pipette gently to avoid disturbing the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">11. Repeat the previous step.</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 20%;">Repeat the wash if the supernatant is cloudy.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">12. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking.</td>
<td style="text-align: left; width: 40%;">Removes any remaining ethanol.</td>
<td style="text-align: left; width: 20%;">Avoid over-drying the beads, which can hinder elution.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">13. Remove the tube from the magnetic rack and resuspend pellet in 30 µl Nuclease-free water. Incubate for 2 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Elution buffer rehydrates the DNA and allows it to detach from the beads.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of nuclease-free water. Ensure complete resuspension.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">14. Pellet the beads on the magnet until the eluate is clear and colourless, for at least 1 minute</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified DNA in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">15. Remove and retain 30 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Collects the purified, end-repaired, and A-tailed cDNA for further processing.</td>
<td style="text-align: left; width: 20%;">Avoid transferring any beads with the eluate.</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Take forward 30 µl of end-prepped cDNA into adapter ligation.</p>
</section></section><section id="adapter-ligation-for-cdna" class="level3"><h3 class="anchored" data-anchor-id="adapter-ligation-for-cdna">Adapter Ligation for cDNA</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%;">1. Check the contents of each tube are clear of any precipitate and are thoroughly mixed before setting up the reaction.<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Mix the contents of each tube by flicking.<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Check that there is no precipitate present (DTT in the Blunt/TA Master Mix can sometimes form a precipitate)<br>
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Spin down briefly before accurately pipetting the contents in the reaction.</td>
<td style="text-align: left; width: 40%;">Ensures that all reagents are properly mixed and no precipitates are present, which could interfere with the ligation reaction.</td>
<td style="text-align: left; width: 20%;">If precipitate is observed, warm the tube briefly at 37°C and vortex gently until the precipitate dissolves.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">2. Taking the end-prepped DNA, perform adapter ligation as follows, mixing by flicking the tube between each sequential addition.<br>
30 µl End-prepped DNA<br>
5 µl Adapter Mix<br>
50 µl Blunt/TA Ligation Master Mix<br>
15 µl Nuclease-free water</td>
<td style="text-align: left; width: 40%;">Combines the end-prepped cDNA with adapters and the ligation master mix. The adapters contain sequences necessary for sequencing and bridge amplification.</td>
<td style="text-align: left; width: 20%;">Use fresh, high-quality reagents. Ensure accurate pipetting and avoid contamination.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">3. Mix gently by flicking the tube, and spin down.</td>
<td style="text-align: left; width: 40%;">Ensures all components are well mixed and collects any condensation on the tube walls.</td>
<td style="text-align: left; width: 20%;">Avoid vigorous mixing, which can damage DNA.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. Incubate the reaction for 10 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Provides optimal temperature for the ligase enzyme to join the adapters to the cDNA ends.</td>
<td style="text-align: left; width: 20%;">Ensure accurate temperature control. Avoid extending the incubation time, as this could lead to non-specific ligation.</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Output: Adapted and tethered DNA library.</strong></p>
</section><section id="ampure-xp-bead-binding" class="level3"><h3 class="anchored" data-anchor-id="ampure-xp-bead-binding">AMPure XP bead binding</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Resuspend the AMPure XP beads by vortexing.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures consistent bead suspension for efficient binding of DNA.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Vortex thoroughly until no clumps remain.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. Add 40 µl of resuspended AMPure XP beads to the adapter ligation reaction from the previous step and mix by pipetting.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">AMPure XP beads selectively bind to DNA fragments larger than a certain size, allowing for size selection and purification.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Use the correct volume of beads for the sample volume. Mix gently to avoid foaming.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Mixing promotes efficient binding of DNA to the beads.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. Place on a magnetic rack, allow beads to pellet and pipette off supernatant.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, separating them from the supernatant containing unwanted fragments and reagents.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and avoid disturbing the pellet when removing supernatant.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. Add 200 μl of Wash Buffer (WSB) to the beads. Resuspend the beads by pipetting up and down. Return the tube to the magnetic rack, allow beads to pellet and pipette off the supernatant.</td>
<td style="text-align: left; width: 40%;">Wash buffer removes impurities and salts from the beads.</td>
<td style="text-align: left; width: 20%;">Use fresh wash buffer and pipette gently to avoid disturbing the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Repeat the previous step.</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 20%;">Repeat washing if the supernatant is still cloudy.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Spin down and place the tube back on the magnet. Pipette off any residual supernatant. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking.</td>
<td style="text-align: left; width: 40%;">Removes any remaining wash buffer, which can interfere with downstream applications.</td>
<td style="text-align: left; width: 20%;">Avoid over-drying the beads, which can make elution difficult.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. Remove the tube from the magnetic rack and resuspend pellet in 13 µl of Elution Buffer (EB).</td>
<td style="text-align: left; width: 40%;">Elution buffer rehydrates the DNA and allows it to detach from the beads.</td>
<td style="text-align: left; width: 20%;">Use the correct volume of elution buffer. Ensure complete resuspension.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT.</td>
<td style="text-align: left; width: 40%;">Incubation allows the DNA to fully rehydrate and elute from the beads.</td>
<td style="text-align: left; width: 20%;">Ensure proper rotation on the Hula mixer.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Pellet the beads on a magnet until the eluate is clear and colourless.</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified DNA in the supernatant.</td>
<td style="text-align: left; width: 20%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">11. Remove and retain 13 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube.</td>
<td style="text-align: left; width: 40%;">Collects the purified DNA for downstream applications.</td>
<td style="text-align: left; width: 20%;">Avoid transferring any beads with the eluate.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">12. Quantify 1 µl of eluted cDNA using a Qubit fluorometer - recovery aim ~60 ng.</td>
<td style="text-align: left; width: 40%;">Measures the concentration of the purified cDNA to assess yield and quality.</td>
<td style="text-align: left; width: 20%;">Ensure accurate calibration of the Qubit fluorometer. If yield is low, troubleshoot previous steps.</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>The prepared library is used for loading onto the flow cell. Store the library on ice until ready to load.</strong></p>
</section><section id="priming-and-loading-on-spoton-flowcell" class="level3"><h3 class="anchored" data-anchor-id="priming-and-loading-on-spoton-flowcell">Priming and Loading on SpotON flowCell</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">1. Thaw the Sequencing Buffer (SQB), Loading Beads (LB), Flush Tether (FLT) and one tube of Flush Buffer (FB) at RT before mixing the reagents by vortexing, and spin down the SQB, FB and FLT at RT.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Thawing and mixing reagents ensures they are at the correct temperature and consistency for optimal performance.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Ensure all reagents are completely thawed and mixed before use. Check for any signs of contamination or degradation.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">2. Prepare the flow cell priming mix: Add 30 µl of thawed and mixed Flush Tether (FLT) directly to the tube of thawed and mixed Flush Buffer (FB), and mix by vortexing.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">The priming mix prepares the flow cell for loading by coating the channels and pores with reagents that facilitate DNA binding and sequencing.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Avoid introducing air bubbles into the priming mix. Mix gently to avoid foaming.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">3. Open the MinION Mk1C lid and slide the flow cell under the clip.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Provides access to the flow cell ports for loading reagents.</td>
<td style="text-align: left; width: 20%; background-color: #f2f2f2 !important;">Handle the flow cell with care to avoid damage. Make sure it is properly seated under the clip.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">4. Slide the priming port cover clockwise to open the priming port.</td>
<td style="text-align: left; width: 40%;">Opens the port for loading the priming mix.</td>
<td style="text-align: left; width: 20%;">Open the priming port slowly and carefully to prevent damage.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">5. IMPORTANT: Take care when drawing back buffer from the flow cell. Do not remove more than 20-30 µl, and make sure that the array of pores are covered by buffer at all times. Introducing air bubbles into the array can irreversibly damage pores.<br>
After opening the priming port, check for a small air bubble under the cover. Draw back a small volume to remove any bubbles:<br>
Set a P1000 pipette to 200 µl<br>
Insert the tip into the priming port<br>
Turn the wheel until the dial shows 220-230 µl, to draw back 20-30 µl, or until you can see a small volume of buffer entering the pipette tip<br><br>
Note: Visually check that there is continuous buffer from the priming port across the sensor array.</td>
<td style="text-align: left; width: 40%;">Removes any air bubbles that may have formed under the priming port cover, which could interfere with flow cell function.</td>
<td style="text-align: left; width: 20%;">Use caution when drawing back buffer. Avoid removing too much or introducing air bubbles.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">6. Load 800 µl of the priming mix into the flow cell via the priming port, avoiding the introduction of air bubbles. Wait for five minutes. During this time, prepare the library for loading by following the steps below.</td>
<td style="text-align: left; width: 40%;">The priming mix helps to wet the flow cell channels and pores, ensuring proper fluid flow and DNA capture.</td>
<td style="text-align: left; width: 20%;">Avoid introducing air bubbles when loading the priming mix. Ensure the full volume is loaded.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">7. Thoroughly mix the contents of the Loading Beads (LB) by pipetting.</td>
<td style="text-align: left; width: 40%;">Ensures uniform distribution of beads for optimal library loading.</td>
<td style="text-align: left; width: 20%;">Mix gently to avoid damaging the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">8. IMPORTANT: The Loading Beads (LB) tube contains a suspension of beads. These beads settle very quickly. It is vital that they are mixed immediately before use<br>
In a new tube, prepare the library for loading as follows:<br><br>
37.5 µl Sequencing Buffer (SQB)<br>
25.5 µl Loading Beads (LB), mixed immediately before use<br>
12 µl DNA library</td>
<td style="text-align: left; width: 40%;">Combines the library with sequencing buffer and loading beads. The loading beads help to concentrate the library and facilitate its entry into the flow cell pores.</td>
<td style="text-align: left; width: 20%;">Mix the loading beads immediately before use to prevent settling. Avoid over-mixing, which can damage DNA.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">9. Complete the flow cell priming:<br>
Gently lift the SpotON sample port cover to make the SpotON sample port accessible.<br>
Load 200 µl of the priming mix into the flow cell priming port (not the SpotON sample port), avoiding the introduction of air bubbles.</td>
<td style="text-align: left; width: 40%;">Completes the priming process, ensuring all channels and pores are properly coated and ready for sequencing.</td>
<td style="text-align: left; width: 20%;">Avoid introducing air bubbles when loading the priming mix. Ensure the full volume is loaded.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">10. Mix the prepared library gently by pipetting up and down just prior to loading.</td>
<td style="text-align: left; width: 40%;">Ensures the library is well mixed and evenly distributed before loading.</td>
<td style="text-align: left; width: 20%;">Mix gently to avoid damaging the DNA library.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">11. Add 75 μl of the prepared library to the flow cell via the SpotON sample port in a dropwise fashion. Ensure each drop flows into the port before adding the next.</td>
<td style="text-align: left; width: 40%;">The SpotON port allows for gentle and controlled loading of the library onto the flow cell.</td>
<td style="text-align: left; width: 20%;">Load the library slowly and carefully to avoid air bubbles or spills. Ensure complete loading.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 40%;">12. Gently replace the SpotON sample port cover, making sure the bung enters the SpotON port and close the priming port.</td>
<td style="text-align: left; width: 40%;">Prevents contamination and leakage of reagents.</td>
<td style="text-align: left; width: 20%;">Close the ports securely to prevent leakage.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 40%;">13. IMPORTANT: Install the light shield on your flow cell as soon as the library has been loaded for optimal sequencing output.<br><br>
Place the light shield onto the flow cell, as follows:<br>
Carefully place the leading edge of the light shield against the clip.<br>
Note: Do not force the light shield underneath the clip.<br>
Gently lower the light shield onto the flow cell. The light shield should sit around the SpotON cover, covering the entire top section of the flow cell.</td>
<td style="text-align: left; width: 40%;">The light shield protects the flow cell from ambient light, which can interfere with sequencing.</td>
<td style="text-align: left; width: 20%;">Install the light shield gently to avoid damaging the flow cell. Make sure it is properly seated.</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Close the device lid and set up a sequencing run on MinKNOW.</p>
</section></section><section id="troubleshooting-sispa-protocol" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-sispa-protocol">Troubleshooting: SISPA Protocol</h2>
<section id="cdna-synthesis" class="level3"><h3 class="anchored" data-anchor-id="cdna-synthesis">cDNA Synthesis</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Detail
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Mix in a clean 0.2 ml PCR tube:<br>
* 1 μl dNTP Mix (10 mM)<br>
* 1 μl FR26RV-N (Stock = 50 μM)<br>
* 11 μl Viral RNA (ideally the Cq should be &lt;23-26, depending on the virus)</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Combines the viral RNA template, dNTPs (building blocks for DNA synthesis), and the first SISPA primer (FR26RV-N). The primer contains a barcode sequence and random hexamers that will anneal to the viral RNA.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure accurate quantification and integrity of RNA. Use high-quality, RNase-free reagents. The Cq value of the viral RNA should be within the optimal range for efficient reverse transcription.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Incubate for 5 min at 65 °C in the thermal cycler. Place on ice for 1 min.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">The high temperature denatures RNA secondary structures, making it more accessible for primer binding. The subsequent cooling on ice helps maintain the denatured state and promotes primer annealing.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Verify accurate temperature control in the thermal cycler. Use a pre-chilled ice bath for rapid cooling.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Pre-warm 5x SSIV Buffer at 37 °C for 10 minutes before use, mix by vortexing and spin down.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Pre-warming the buffer ensures optimal enzyme activity during reverse transcription. Mixing and spinning down ensure homogeneity and prevent reagent loss due to condensation.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Use a water bath or heat block for accurate temperature control. Mix gently to avoid introducing air bubbles.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">4. Add per sample:<br>
* 4 μl SSIV buffer 5x<br>
* 1 μl DTT 100 mM<br>
* 1 μl RNase Inhibitor (RNase OUT; 40 U/μl)<br>
* 1 μl SSIV Reverse Transcriptase (200 U/μl)</td>
<td style="text-align: left; width: 40%;">Provides the necessary buffer conditions, reducing agents (DTT), and RNase inhibitor to protect the RNA template and ensure efficient reverse transcription. The reverse transcriptase enzyme catalyzes the synthesis of cDNA from the RNA template.</td>
<td style="text-align: left; width: 30%;">Prepare the reaction mix on ice to minimize RNase activity. Use fresh, high-quality reagents. Avoid pipetting errors, especially with small volumes.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">5. Close lids thoroughly, spin down quickly using bench top centrifuge</td>
<td style="text-align: left; width: 40%;">Prevents evaporation and contamination during the reaction.</td>
<td style="text-align: left; width: 30%;">Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">6. Place in the thermal cycler: 23 °C for 10 min, 50 °C for 50 min, 80 °C for 10 min, Hold at 4 °C</td>
<td style="text-align: left; width: 40%;">* 23 °C: Allows for primer annealing to the RNA template.<br>
* 50 °C: Optimal temperature for reverse transcriptase activity, enabling cDNA synthesis.<br>
* 80 °C: Inactivates the reverse transcriptase enzyme.<br>
* 4 °C: Holds the reaction until further processing.</td>
<td style="text-align: left; width: 30%;">Verify accurate temperature control and timing in the thermal cycler.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="double-strand-synthesis" class="level3"><h3 class="anchored" data-anchor-id="double-strand-synthesis">Double-strand synthesis</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Detail
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Add 1 μl of Klenow polymerase (5 U/μl) per sample directly to the tube.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Klenow polymerase, a DNA polymerase lacking 5'-&gt;3' exonuclease activity, synthesizes the second cDNA strand using the annealed SISPA primer as a template. This results in double-stranded cDNA with barcode sequences incorporated into both strands.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Keep the Klenow polymerase on ice until use to maintain its activity. Avoid pipetting errors, especially with small volumes.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Close lids thoroughly, spin down quickly using bench top centrifuge</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Prevents evaporation and contamination during the reaction.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Place in the thermal cycler: 37 °C for 60 min, 75 °C for 10 min</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">* 37 °C: Optimal temperature for Klenow polymerase activity, allowing for efficient second strand synthesis.<br>
* 75 °C: Inactivates the Klenow polymerase enzyme.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Verify accurate temperature control and timing in the thermal cycler.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="cdna-amplification" class="level3"><h3 class="anchored" data-anchor-id="cdna-amplification">cDNA Amplification</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Detail
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Mix:<br>
* 5 μl dsDNA<br>
* 1 μl FR20RV (Stock 40 μM, final concentration 0.8 μM)<br>
* 5 μl 10 X PfU Ultra II reaction Buffer<br>
* 1.25 μl dNTP Mix 10 mM<br>
* 1 μl PfU Ultra II Fusion HS DNA polymerase (keep on cooling rack)<br>
* 36.75 μl Nuclease-free Water</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Combines the double-stranded cDNA template, the second SISPA primer (FR20RV), dNTPs, buffer, and a high-fidelity DNA polymerase (PfU Ultra II Fusion HS). The second SISPA primer binds to the barcode sequences incorporated in the previous steps, enabling amplification of the cDNA.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure accurate pipetting, especially with small volumes. Keep the polymerase enzyme on ice until use to maintain its activity. Use high-quality, nuclease-free water.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Close lids thoroughly, spin down quickly using bench top centrifuge</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Prevents evaporation and contamination during the reaction.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Place in the thermal cycler:<br>
* Denaturation 95 °C 1 min<br>
* 45 Cycles<br>
* 95 °C 20 sec<br>
* 65 °C 20 sec<br>
* 72 °C 3 min<br>
* Final Extension 72 °C 3 min<br>
* Hold 4 °C</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">* 95 °C (Denaturation): Separates the double-stranded cDNA into single strands.<br>
* 65 °C (Annealing): Allows the second SISPA primer to bind to the cDNA.<br>
* 72 °C (Extension): The polymerase enzyme synthesizes new DNA strands, amplifying the cDNA.<br>
* 45 Cycles: Repeats the denaturation, annealing, and extension steps to generate a sufficient amount of amplified cDNA.<br>
* Final Extension: Ensures complete extension of all cDNA strands.<br>
* 4 °C: Holds the reaction until further processing.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Verify accurate temperature control and timing in the thermal cycler. Optimize the number of PCR cycles if needed to achieve sufficient amplification without introducing artifacts.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="clean-up-of-cdna-amplicon" class="level3"><h3 class="anchored" data-anchor-id="clean-up-of-cdna-amplicon">Clean-up of cDNA amplicon</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Transfer 25 μl of the dsDNA to a 1.5 mL DNA LoBind tubes; the spare dsDNA should be stored at 4 °C as backup.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Transfer the amplified cDNA to a suitable tube for purification.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Use LoBind tubes to minimize DNA loss.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Bring AMPure beads to room temperature. Mix the beads thoroughly before use! Add 1.8 x (sample volume) beads, e.g. for 25 μl sample add 45 μl beads;</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">AMPure XP beads selectively bind DNA, allowing for purification and concentration.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Mix sample thoroughly by pipetting up and down at least 15 x</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures efficient binding of cDNA to the beads.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Mix gently to avoid foaming or DNA damage.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">4. Incubate at room temperature for 7 min on a thermomix (room temperature, 550 rpm)</td>
<td style="text-align: left; width: 40%;">Incubation allows the cDNA to bind to the beads.</td>
<td style="text-align: left; width: 30%;">Use a thermomixer or rotator for consistent mixing.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">5. Quickly centrifuge</td>
<td style="text-align: left; width: 40%;">Brief centrifugation ensures all liquid is at the bottom of the tube.</td>
<td style="text-align: left; width: 30%;">Ensure complete pelleting of beads before removing supernatant.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">6. Place tubes in the magnetic stand and leave tubes in the magnetic rack until elution (!)</td>
<td style="text-align: left; width: 40%;">Magnetic separation facilitates removal of unwanted components.</td>
<td style="text-align: left; width: 30%;">Use a strong magnet and avoid disturbing the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">7. After 5 minutes remove the supernatant slowly; hold the tube with one hand while opening the cap!</td>
<td style="text-align: left; width: 40%;">Careful removal prevents disturbing the bound cDNA.</td>
<td style="text-align: left; width: 30%;">Remove supernatant carefully to avoid losing beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">8. Add 1 ml 80 % ethanol and incubate for 30 sec at room temperature</td>
<td style="text-align: left; width: 40%;">Ethanol washes remove contaminants and salts.</td>
<td style="text-align: left; width: 30%;">Use fresh 80% ethanol.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">9. Slightly rotate the tubes in the magnetic rack to concentrate the beads</td>
<td style="text-align: left; width: 40%;">Helps concentrate the beads for efficient washing.</td>
<td style="text-align: left; width: 30%;">Avoid excessive agitation, which can dislodge the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">10. Slowly remove the supernatant and discard the ethanol</td>
<td style="text-align: left; width: 40%;">Removes the ethanol wash.</td>
<td style="text-align: left; width: 30%;">Remove supernatant completely without disturbing the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">11. Add another 1 ml 80 % ethanol and incubate for 30 sec at room temperature</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 30%;">Use fresh 80% ethanol.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">12. Slowly remove the supernatant and discard the ethanol</td>
<td style="text-align: left; width: 40%;">Removes residual ethanol, which can interfere with downstream steps.</td>
<td style="text-align: left; width: 30%;">Remove supernatant completely without disturbing the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">13. Remove all remaining ethanol with a 10 μl pipet and a clean tip and let air dry for 10 min with the tube opened</td>
<td style="text-align: left; width: 40%;">Ensure complete removal of ethanol before elution. Avoid over-drying the beads.</td>
<td style="text-align: left; width: 30%;">Ensure complete removal of ethanol before elution. Avoid over-drying the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">14. Elution: Take the tube from the magnetic stand and add 25 μl nuclease-free water directly onto the beads. Resuspend the beads by pipette mixing. Incubate 2 min at room temperature. Place tube on magnetic stand and incubate for 10 min. Transfer the eluate containing the DNA into a clean reaction tube</td>
<td style="text-align: left; width: 40%;">Elution buffer releases the purified cDNA from the beads.</td>
<td style="text-align: left; width: 30%;">Use nuclease-free water for elution. Ensure complete resuspension of the beads.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="end-prep-and-barcode-ligation" class="level3"><h3 class="anchored" data-anchor-id="end-prep-and-barcode-ligation">End Prep and Barcode Ligation</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Mix in a clean 1.5 ml DNA LoBind tube:<br>
o 10 μl DNA<br>
o 1.4 μl Ultra II End Reaction Buffer<br>
o 0.6 μl Ultra II End Prep Enzyme Mix (keep in cooling rack or on ice)</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Combines the purified cDNA with the end-prep reaction mix, which contains enzymes for repairing DNA ends and adding a 3' adenine overhang.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure proper enzyme activity and storage. Use high-quality reagents.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Incubate using thermomix: 10 min Room Temperature, 5 min 65 °C, Place on ice 1 min.</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Provides optimal temperatures for the end-repair and A-tailing reactions.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Verify accurate temperature control and timing in the thermomixer.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Directly add:<br>
o 2.5 μl NB(XX) barcode<br>
▪ e.g. Sample 1 = Barcode 06; Sample 2 = Barcode 07; Sample 03 = Barcode 08<br>
▪ Note down which barcode was used for which sample (!)<br>
o 14,5 μl Ultra II Ligation Master Mix<br>
o 0,5 μl Ligation Enhancer</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Adds a unique barcode to each sample, allowing for multiplexing during sequencing. The ligation master mix and enhancer facilitate the joining of the barcode to the cDNA.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Use fresh, high-quality reagents. Keep the ligation mix and enhancer on ice until use.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">4. Incubate using thermomix: 20 min Room Temperature, 10 min 70 °C, Place on ice 1 min</td>
<td style="text-align: left; width: 40%;">Provides optimal temperatures for the ligation reaction.</td>
<td style="text-align: left; width: 30%;">Verify accurate temperature control and timing in the thermomixer.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">5. POOL ALL SAMPLES in a single tube!</td>
<td style="text-align: left; width: 40%;">Combines all barcoded samples into a single tube for further processing and sequencing.</td>
<td style="text-align: left; width: 30%;">Ensure thorough mixing of all samples before pooling.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="cleanup-of-barcoded-dna" class="level3"><h3 class="anchored" data-anchor-id="cleanup-of-barcoded-dna">Cleanup of barcoded DNA</h3>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-sm small" data-quarto-postprocess="true">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead><tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Step
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Mechanistic Details
</div></th>
<th data-quarto-table-cell-role="th" style="text-align: center; border-bottom: hidden; padding-bottom: 0; padding-left: 3px; padding-right: 3px;"><div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
Troubleshooting
</div></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">1. Add 29.5 μl AMPure XP beads per sample [equal volume (1:1) sample:beads]</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">AMPure XP beads selectively bind DNA, allowing for purification and concentration.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">2. Mix by pipetting and quickly centrifuge</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Ensures efficient binding of DNA to the beads.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Mix gently to avoid foaming or DNA damage.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">3. Incubate for 5 min at room temperature</td>
<td style="text-align: left; width: 40%; background-color: #f2f2f2 !important;">Incubation allows the DNA to bind to the beads.</td>
<td style="text-align: left; width: 30%; background-color: #f2f2f2 !important;">Use a thermomixer or rotator for consistent mixing.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">4. Place the sample on the magnetic rack</td>
<td style="text-align: left; width: 40%;">Magnetic separation facilitates removal of unwanted components.</td>
<td style="text-align: left; width: 30%;">Ensure complete pelleting of beads before removing supernatant.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">5. Incubate for at least 2 min until the supernatant is completely clear</td>
<td style="text-align: left; width: 40%;">Ensures complete binding of DNA to the beads.</td>
<td style="text-align: left; width: 30%;">Use a strong magnet and avoid disturbing the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">6. Carefully remove the supernatant</td>
<td style="text-align: left; width: 40%;">Careful removal prevents disturbing the bound DNA.</td>
<td style="text-align: left; width: 30%;">Remove supernatant carefully to avoid losing beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">7. Add 300 μl 70% ethanol to the tube still on the magnetic rack</td>
<td style="text-align: left; width: 40%;">Ethanol washes remove contaminants and salts.</td>
<td style="text-align: left; width: 30%;">Use fresh 70% ethanol.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">8. Incubate 30 secs</td>
<td style="text-align: left; width: 40%;">Allows ethanol to penetrate and wash the beads.</td>
<td style="text-align: left; width: 30%;">Avoid excessive agitation, which can dislodge the beads.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">9. Carefully remove the supernatant</td>
<td style="text-align: left; width: 40%;">Removes the ethanol wash.</td>
<td style="text-align: left; width: 30%;">Remove supernatant completely without disturbing the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">10. Repeat last three steps (2nd ethanol wash)</td>
<td style="text-align: left; width: 40%;">Second wash further improves purity.</td>
<td style="text-align: left; width: 30%;">Repeat washing if the supernatant is still cloudy.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">11. Spin down and remove residual ethanol with the tube placed on the magnetic rack using a clean filter tip and a pipette.</td>
<td style="text-align: left; width: 40%;">Removes residual ethanol, which can interfere with downstream steps.</td>
<td style="text-align: left; width: 30%;">Ensure complete removal of ethanol before elution. Avoid over-drying the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">12. Air dry 1 min</td>
<td style="text-align: left; width: 40%;">Allows residual ethanol to evaporate.</td>
<td style="text-align: left; width: 30%;">Do not over-dry the beads, as this can make elution difficult.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">13. Resuspend in 30 μl EB by pipetting</td>
<td style="text-align: left; width: 40%;">Elution buffer releases the purified DNA from the beads.</td>
<td style="text-align: left; width: 30%;">Use the correct volume of elution buffer. Ensure complete resuspension of the beads.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">14. Incubate off the magnetic rack for 2 min</td>
<td style="text-align: left; width: 40%;">Incubation allows the DNA to fully rehydrate and elute from the beads.</td>
<td style="text-align: left; width: 30%;">Incubate at the correct temperature and time.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">15. Replace on magnetic rack</td>
<td style="text-align: left; width: 40%;">Magnetic separation collects the beads, leaving the purified DNA in the supernatant.</td>
<td style="text-align: left; width: 30%;">Use a strong magnet and allow sufficient time for clear eluate.</td>
</tr>
<tr class="even">
<td style="text-align: left; width: 30%;">16. Transfer eluted DNA to a fresh 1.5 mL DNA LoBind tube</td>
<td style="text-align: left; width: 40%;">Collects the purified, barcoded DNA for further processing.</td>
<td style="text-align: left; width: 30%;">Avoid transferring any beads with the eluate.</td>
</tr>
<tr class="odd">
<td style="text-align: left; width: 30%;">17. Label tube accordingly with an ethanol-resistant labmarker</td>
<td style="text-align: left; width: 40%;">Ensures proper identification of the sample.</td>
<td style="text-align: left; width: 30%;">Use a marker that won't smudge or fade when exposed to ethanol.</td>
</tr>
</tbody>
</table>
</div>
</div>
</section><section id="sispa-vs-maxima-h-minus-rtssp-mmlv-rt" class="level3"><h3 class="anchored" data-anchor-id="sispa-vs-maxima-h-minus-rtssp-mmlv-rt">SISPA vs Maxima H Minus RT/SSP (MMLV RT)</h3>
<p>MMLV RT just make ssRNA to ds-cDNA. But for this, we need to add poly-A tail first. On the other hand, SISPA protocol has both ds-cDNA making step and amplification step of the cDNA using standard PCR protocol. So, we will have pre-amplified DNA samples out of the starting RNA material. And we don’t need 3´ poly-A tailing since the primers are random hexamers (with barcode) here in this protocol.</p>
</section></section><section id="troubleshooting-data-analysis" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-data-analysis">Troubleshooting: Data Analysis</h2>
<p>We knew the concentration/titer of the (Usutu) virus used to spike the mosquito pools. So, we have an expectation of the concentration of viruses in our samples. I tried to do the calculation for the expectation and compared with the predicted ones.</p>
<p>Different steps of the genomic pipeline is done separately. Working on making the automated snakemake pipeline making.</p>
</section><section id="troubleshooting-flowcell-count-issue" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-flowcell-count-issue">Troubleshooting: FlowCell Count Issue</h2>
<p>It is fine. We used same FlowCell for multiple times (e.g.&nbsp;all LMD sequencing were done on 1 FlowCell, WNV_sup sample was run on the FlowCell used for the 5RNAvir cocktail run)</p>
</section><section id="troubleshooting-contamination-issue" class="level2"><h2 class="anchored" data-anchor-id="troubleshooting-contamination-issue">Troubleshooting: Contamination Issue</h2>
<p>I used the DCS reagent (1 µL) for all the samples using the LMD sequencing protocol. I should have use the direct cDNA sequencing protocol (SQK-DCS109) where I was supposed to use 1 µL more of the sample omitting the use of the DCS.</p>


<!-- -->

</section><div id="quarto-appendix" class="default"><section class="quarto-appendix-contents"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{rasheduzzaman2024,
  author = {Md Rasheduzzaman},
  title = {Lab {Work} {Organization}},
  date = {2024-08-27},
  langid = {en},
  abstract = {Sample prep, RNA extraction, RT-qPCR, cDNA conversion,
    library prep, ONT sequencing, etc.}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-rasheduzzaman2024" class="csl-entry quarto-appendix-citeas" role="listitem">
Md Rasheduzzaman. 2024. <span>“Lab Work Organization.”</span> August 27,
2024.
</div></div></section></div></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb1" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Lab Work Organization"</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="an">abstract:</span><span class="co"> "Sample prep, RNA extraction, RT-qPCR, cDNA conversion, library prep, ONT sequencing, etc."</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="an">prefer-html:</span><span class="co"> true</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: Sample Preparation</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo = FALSE}</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>pacman<span class="sc">::</span><span class="fu">p_load</span>(</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>  tidyverse, </span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>  kableExtra,</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>  downlit,</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>  xml2</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>sample_prep <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Get 50 mosquitoes from insectary."</span>,</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Make 5 pools each consisting of 10 mosquitoes in 2ml tubes containing steel balls."</span>,</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Make 3 Medium aliquots containing 2ml medium each."</span>,</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Add 10^4/ml particles to one 2ml aliquot -&gt; mix."</span>,</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Add 10^6/ml particles to the second 2ml aliquot -&gt; mix."</span>,</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Add 750µl of the 10^4 solution to two pools each and 250µl of the 10^4 solution to Trizol LS (Sample 1)."</span>,</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Add 750µl of the 10^6 solution to two pools each and 250µl of the 10^6 solution to Trizol LS (Sample 2)."</span>,</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Add 750µl of the control medium to the last mosquito pool and 250µl of the medium to Trizol LS (Sample 3)."</span>,</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Store Trizol LS samples @ -80°C."</span>,</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Homogenize the pools 2 min @ 30 Hz."</span>,</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Centrifuge all samples for 3 min @ 20000g."</span>,</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Take 250µl of the supernatant of the smaller control and pools and add it to Trizol LS -&gt; -80°C (Samples 4-6)."</span>,</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Transfer the remaining supernatant to a fresh tube (for all pools)."</span>,</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Centrifuge for 10 min @ 20000g."</span>,</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_1. Take out the plunger from a filtering syringe, and put the empty cylinder on top of a new tube."</span>,</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_2. Transfer the sample into the empty cylinder."</span>,</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_3. Hold the cylinder firmly, and insert the plunger. Push it to filter the sample through the filter. Some dead volume will not be passed through."</span>,</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a>    <span class="st">"15. Transfer supernatant/filtered volume to ultracentrifuge tubes."</span>,</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">"16. Underlay the sample with 20% Sucrose in PBS."</span>,</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a>    <span class="st">"17. Pellet particles 1h @ 483 750 g (or higher)."</span>,</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>    <span class="st">"18. Carefully remove supernatant (best by careful decanting)."</span>,</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a>    <span class="st">"19. Leave the tubes bottoms up for a few minutes to drain any fluid."</span>,</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a>    <span class="st">"20. Add 1ml Trizol to the pellets (Samples 7-11) -&gt; store @ -80°C till processing."</span></span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Obtain the necessary number of mosquitoes for the experiment."</span>,</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Pooling increases sample size and reduces variability."</span>,</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Medium is used as a negative control."</span>,</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Create a spike-in sample with a known concentration of particles."</span>,</span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Create a spike-in sample with a higher concentration of particles."</span>,</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Spike-in controls help assess extraction efficiency."</span>,</span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Spike-in controls help assess extraction efficiency."</span>,</span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. The control medium sample serves as a baseline for comparison."</span>,</span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Trizol LS preserves RNA/DNA integrity."</span>,</span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Disrupts mosquito tissues and releases nucleic acids."</span>,</span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Separates cellular debris from the supernatant containing nucleic acids."</span>,</span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Collects a representative sample for later RNA/DNA extraction."</span>,</span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Removes debris and potential inhibitors from the supernatant."</span>,</span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Further clarifies the supernatant."</span>,</span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_1. Prepare the syringe for filtration."</span>,</span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_2. Begin the filtration process."</span>,</span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_3. Complete the filtration step."</span>,</span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a>    <span class="st">"15. Prepare for ultracentrifugation."</span>,</span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a>    <span class="st">"16. Sucrose cushion helps to concentrate particles during ultracentrifugation."</span>,</span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a>    <span class="st">"17. Ultracentrifugation pellets particles."</span>,</span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a>    <span class="st">"18. Removes unwanted supernatant."</span>,</span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a>    <span class="st">"19. Ensure all supernatant is removed."</span>,</span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a>    <span class="st">"20. Trizol preserves RNA/DNA in the pelleted particles."</span></span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Ensure mosquitoes are healthy and of the correct species/strain."</span>,</span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Avoid cross-contamination between pools."</span>,</span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Prepare medium under sterile conditions."</span>,</span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Verify particle concentration and mix thoroughly."</span>,</span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Verify particle concentration and mix thoroughly."</span>,</span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Pipette accurately to ensure proper spike-in concentrations."</span>,</span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Pipette accurately to ensure proper spike-in concentrations."</span>,</span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Ensure control medium is sterile and particle-free."</span>,</span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Maintain consistent -80°C storage for all samples."</span>,</span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Check homogenizer settings and ensure thorough disruption."</span>,</span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Use appropriate centrifuge settings and time."</span>,</span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Avoid disturbing the pellet when collecting supernatant."</span>,</span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Transfer supernatant carefully to avoid contamination."</span>,</span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Check centrifuge settings and time."</span>,</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_1. Ensure the filter is compatible with the sample type."</span>,</span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_2. Avoid spilling or losing sample during transfer."</span>,</span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14_3. Apply consistent pressure to the plunger for even filtration."</span>,</span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a>    <span class="st">"15. Use appropriate ultracentrifuge tubes.&lt;br&gt;</span></span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a><span class="st">    These tubes are in the ultrcentrifugation room in the closet."</span>,</span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a>    <span class="st">"16. Prepare sucrose solution accurately.&lt;br&gt; </span></span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a><span class="st">    Find out the heaviest one and add PBS to other tubes to make the same volume.&lt;br&gt; </span></span>
<span id="cb1-89"><a href="#cb1-89" aria-hidden="true" tabindex="-1"></a><span class="st">    Add 1.2mL PBS first to the sample and then add the 20% sucrose."</span>,</span>
<span id="cb1-90"><a href="#cb1-90" aria-hidden="true" tabindex="-1"></a>    <span class="st">"17. Use correct ultracentrifuge settings and time.&lt;br&gt; </span></span>
<span id="cb1-91"><a href="#cb1-91" aria-hidden="true" tabindex="-1"></a><span class="st">    We need 60WS to get 48375g."</span>,</span>
<span id="cb1-92"><a href="#cb1-92" aria-hidden="true" tabindex="-1"></a>    <span class="st">"18. Avoid disturbing the pellet when removing supernatant."</span>,</span>
<span id="cb1-93"><a href="#cb1-93" aria-hidden="true" tabindex="-1"></a>    <span class="st">"19. Complete drainage before proceeding to the next step."</span>,</span>
<span id="cb1-94"><a href="#cb1-94" aria-hidden="true" tabindex="-1"></a>    <span class="st">"20. Add Trizol directly to the pellet to prevent degradation."</span></span>
<span id="cb1-95"><a href="#cb1-95" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-96"><a href="#cb1-96" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-97"><a href="#cb1-97" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-98"><a href="#cb1-98" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-99"><a href="#cb1-99" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Extract all trizol samples according to manufacturers protocol</span>
<span id="cb1-100"><a href="#cb1-100" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Add glycogen (0.5µl) for the samples after the sucrose cushion (samples 7-11).</span>
<span id="cb1-101"><a href="#cb1-101" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Determine viral RNA content through RT-qPCR.</span>
<span id="cb1-102"><a href="#cb1-102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-103"><a href="#cb1-103" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-104"><a href="#cb1-104" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate HTML table</span></span>
<span id="cb1-105"><a href="#cb1-105" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(sample_prep, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-106"><a href="#cb1-106" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-107"><a href="#cb1-107" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"50%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-108"><a href="#cb1-108" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"25%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-109"><a href="#cb1-109" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"25%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-110"><a href="#cb1-110" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb1-111"><a href="#cb1-111" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Details"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Comments for Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-112"><a href="#cb1-112" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-113"><a href="#cb1-113" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-114"><a href="#cb1-114" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: RNA Extraction</span></span>
<span id="cb1-115"><a href="#cb1-115" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-116"><a href="#cb1-116" aria-hidden="true" tabindex="-1"></a><span class="fu">### Procedural Guidelines</span></span>
<span id="cb1-117"><a href="#cb1-117" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-118"><a href="#cb1-118" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Perform all steps at room temperature (20–25°C) unless otherwise noted.\</span>
<span id="cb1-119"><a href="#cb1-119" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Use cold TRIzol™ Reagent if the starting material contains high levels of RNase, such as spleen or pancreas samples.\</span>
<span id="cb1-120"><a href="#cb1-120" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Use disposable, individually wrapped, sterile plastic ware and sterile, disposable RNase-free pipettes &amp; pipette tips, and tubes.\</span>
<span id="cb1-121"><a href="#cb1-121" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Wear disposable gloves while handling reagents and RNA samples to prevent RNase contamination from the surface of the skin; change gloves frequently, particularly as the protocol progresses from crude extracts to more purified materials.\</span>
<span id="cb1-122"><a href="#cb1-122" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Always use proper microbiological aseptic techniques when working with RNA.\</span>
<span id="cb1-123"><a href="#cb1-123" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Use RNaseZap™ RNase Decontamination Solution (Cat. no. AM9780) to remove RNase contamination from work surfaces and non-disposable items such as centrifuges and pipettes used during purification.</span>
<span id="cb1-124"><a href="#cb1-124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-125"><a href="#cb1-125" aria-hidden="true" tabindex="-1"></a><span class="fu">### Lyse samples and separate phases</span></span>
<span id="cb1-126"><a href="#cb1-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-127"><a href="#cb1-127" aria-hidden="true" tabindex="-1"></a><span class="ss">1.  </span>Lyse and homogenize samples in TRIzol™ Reagent according to your starting material.</span>
<span id="cb1-128"><a href="#cb1-128" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-129"><a href="#cb1-129" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Tissues:\</span>
<span id="cb1-130"><a href="#cb1-130" aria-hidden="true" tabindex="-1"></a>    Add 1 mL of TRIzol™ Reagent per 50–100 mg of tissue to the sample and homogenize using a homogenizer.</span>
<span id="cb1-131"><a href="#cb1-131" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-132"><a href="#cb1-132" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Cell grown in monolayer:\</span>
<span id="cb1-133"><a href="#cb1-133" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-134"><a href="#cb1-134" aria-hidden="true" tabindex="-1"></a>a.  Remove growth media.\</span>
<span id="cb1-135"><a href="#cb1-135" aria-hidden="true" tabindex="-1"></a>b.  Add 0.3–0.4 mL of TRIzol™ Reagent per 1 × $10^{5}—10^{7}$ cells directly to the culture dish to lyse the cells.\</span>
<span id="cb1-136"><a href="#cb1-136" aria-hidden="true" tabindex="-1"></a>c.  Pipet the lysate up and down several times to homogenize.</span>
<span id="cb1-137"><a href="#cb1-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-138"><a href="#cb1-138" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Cells grown in suspension:\</span>
<span id="cb1-139"><a href="#cb1-139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-140"><a href="#cb1-140" aria-hidden="true" tabindex="-1"></a>a.  Pellet the cells by centrifugation and discard the supernatant.\</span>
<span id="cb1-141"><a href="#cb1-141" aria-hidden="true" tabindex="-1"></a>b.  Add 0.75 mL of TRIzol™ Reagent per 0.25 mL of sample (5–10 × $10^{6}$ cells from animal, plant, or yeasty origin or 1 × $10^{7}$ cells of bacterial origin) to the pellet.\</span>
<span id="cb1-142"><a href="#cb1-142" aria-hidden="true" tabindex="-1"></a>    *Note:* Do not wash cells before addition of TRIzol™ Reagent to avoid mRNA degradation.\</span>
<span id="cb1-143"><a href="#cb1-143" aria-hidden="true" tabindex="-1"></a>c.  Pipette the lysate up and down several times to homogenize.\</span>
<span id="cb1-144"><a href="#cb1-144" aria-hidden="true" tabindex="-1"></a>    **Note:** The sample volume should not exceed 10% of the volume of TRIzol™ Reagent used for lysis.</span>
<span id="cb1-145"><a href="#cb1-145" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-146"><a href="#cb1-146" aria-hidden="true" tabindex="-1"></a>**STOPPING POINT** Samples can be stored at 4°C overnight or at –20°C for up to a year.</span>
<span id="cb1-147"><a href="#cb1-147" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-148"><a href="#cb1-148" aria-hidden="true" tabindex="-1"></a><span class="ss">2.  </span>**(Optional)** If samples have a high fat content, centrifuge the lysate for 5 minutes at 12,000 g at 4–10°C, then transfer the clear supernatant to a new tube.</span>
<span id="cb1-149"><a href="#cb1-149" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-150"><a href="#cb1-150" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo = FALSE}</span></span>
<span id="cb1-151"><a href="#cb1-151" aria-hidden="true" tabindex="-1"></a>lyse_sample <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-152"><a href="#cb1-152" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-153"><a href="#cb1-153" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Incubate for 5 minutes to permit complete dissociation of the nucleoproteins complex."</span>,</span>
<span id="cb1-154"><a href="#cb1-154" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Add 0.2 mL of chloroform per 1 mL of TRIzol™ Reagent used for lysis, then securely cap the tube."</span>,</span>
<span id="cb1-155"><a href="#cb1-155" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Incubate for 2–3 minutes."</span>,</span>
<span id="cb1-156"><a href="#cb1-156" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Centrifuge the sample for 15 minutes at 12,000 × g at 4°C. The mixture separates into a lower red phenol-chloroform, and interphase, and a colorless upper aqueous phase."</span>,</span>
<span id="cb1-157"><a href="#cb1-157" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Transfer the aqueous phase containing the RNA to a new tube by angling the tube at 45° and pipette the solution out."</span></span>
<span id="cb1-158"><a href="#cb1-158" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-159"><a href="#cb1-159" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Detail</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-160"><a href="#cb1-160" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Incubation allows the denaturing agents in TRIzol to disrupt cellular structures and dissolve membranes."</span>,</span>
<span id="cb1-161"><a href="#cb1-161" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Chloroform further denatures proteins and promotes phase separation."</span>,</span>
<span id="cb1-162"><a href="#cb1-162" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. This step enhances phase separation and ensures thorough mixing."</span>,</span>
<span id="cb1-163"><a href="#cb1-163" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Centrifugation causes the sample to separate into distinct layers based on density."</span>,</span>
<span id="cb1-164"><a href="#cb1-164" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. The aqueous phase contains RNA, while DNA and proteins are in the interphase and organic phase, respectively."</span></span>
<span id="cb1-165"><a href="#cb1-165" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-166"><a href="#cb1-166" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-167"><a href="#cb1-167" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. If complete dissociation doesn't occur, increase incubation time or use a higher concentration of TRIzol."</span>,</span>
<span id="cb1-168"><a href="#cb1-168" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Ensure proper mixing of chloroform and TRIzol. Vortex briefly after adding chloroform."</span>,</span>
<span id="cb1-169"><a href="#cb1-169" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Check sample temperature to ensure it's close to room temperature."</span>,</span>
<span id="cb1-170"><a href="#cb1-170" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Ensure correct centrifuge speed and time. Incomplete separation may indicate issues with centrifugation."</span>,</span>
<span id="cb1-171"><a href="#cb1-171" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Use a micropipette with a wide-bore tip to avoid shearing RNA. Be careful not to disturb the interphase."</span></span>
<span id="cb1-172"><a href="#cb1-172" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-173"><a href="#cb1-173" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-174"><a href="#cb1-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-175"><a href="#cb1-175" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate HTML table</span></span>
<span id="cb1-176"><a href="#cb1-176" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(lyse_sample, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-177"><a href="#cb1-177" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-178"><a href="#cb1-178" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"50%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-179"><a href="#cb1-179" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"25%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-180"><a href="#cb1-180" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"25%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-181"><a href="#cb1-181" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb1-182"><a href="#cb1-182" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Details"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Comments for Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-183"><a href="#cb1-183" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-184"><a href="#cb1-184" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-185"><a href="#cb1-185" aria-hidden="true" tabindex="-1"></a>**IMPORTANT!** Avoid transferring any of the interphase or organic layer into the pipette when removing the aqueous phase. <span class="kw">&lt;br&gt;</span></span>
<span id="cb1-186"><a href="#cb1-186" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-187"><a href="#cb1-187" aria-hidden="true" tabindex="-1"></a>Proceed directly to "Isolate RNA" on page 2.<span class="kw">&lt;br&gt;</span></span>
<span id="cb1-188"><a href="#cb1-188" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-189"><a href="#cb1-189" aria-hidden="true" tabindex="-1"></a>Save the interphase and organic phase if you want to isolate DNA or protein. See "Isolate DNA" on page 3 or "Isolate proteins" on page 4 for detailed procedures. The organic phase can be stored at 4°C overnight.</span>
<span id="cb1-190"><a href="#cb1-190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-191"><a href="#cb1-191" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: RT-qPCR</span></span>
<span id="cb1-192"><a href="#cb1-192" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-193"><a href="#cb1-193" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>I will make a shiny app for the calculation online for any number of <span class="in">`#sample`</span>.</span>
<span id="cb1-194"><a href="#cb1-194" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-195"><a href="#cb1-195" aria-hidden="true" tabindex="-1"></a>**N.B.** For the NTC, we need everything except the RNA sample (use RNase-free water instead of sample).</span>
<span id="cb1-196"><a href="#cb1-196" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-197"><a href="#cb1-197" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Using <span class="in">`WNV-Ag-ID-Path.prcl`</span> One-Step RT-PCR protocol.</span>
<span id="cb1-198"><a href="#cb1-198" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-199"><a href="#cb1-199" aria-hidden="true" tabindex="-1"></a>::: {.column width="90%"}</span>
<span id="cb1-200"><a href="#cb1-200" aria-hidden="true" tabindex="-1"></a><span class="al">![RT-qPCR Run Info](../../img/RTqPCR_info.png)</span></span>
<span id="cb1-201"><a href="#cb1-201" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb1-202"><a href="#cb1-202" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-203"><a href="#cb1-203" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>There are 6 steps in the RT-qPCR. Here goes the mechanistic details of the steps:</span>
<span id="cb1-204"><a href="#cb1-204" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-205"><a href="#cb1-205" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-206"><a href="#cb1-206" aria-hidden="true" tabindex="-1"></a>RTqPCR_mechanism <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-207"><a href="#cb1-207" aria-hidden="true" tabindex="-1"></a>  <span class="at">Steps =</span> <span class="fu">c</span>(</span>
<span id="cb1-208"><a href="#cb1-208" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 1: Reverse Transcription (RT)"</span>,</span>
<span id="cb1-209"><a href="#cb1-209" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 2: Initial Denaturation and Enzyme Activation"</span>,</span>
<span id="cb1-210"><a href="#cb1-210" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 3: Denaturation"</span>,</span>
<span id="cb1-211"><a href="#cb1-211" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 4: Annealing"</span>,</span>
<span id="cb1-212"><a href="#cb1-212" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 5: Extension"</span>,</span>
<span id="cb1-213"><a href="#cb1-213" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Step 6: Data Acquisition and Analysis"</span></span>
<span id="cb1-214"><a href="#cb1-214" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-215"><a href="#cb1-215" aria-hidden="true" tabindex="-1"></a>  <span class="at">Mechanism =</span> <span class="fu">c</span>(</span>
<span id="cb1-216"><a href="#cb1-216" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The reverse transcriptase enzyme is active at this temperature, converting the RNA template into complementary DNA (cDNA). This is a crucial step as qPCR amplifies DNA, not RNA."</span>,</span>
<span id="cb1-217"><a href="#cb1-217" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The high temperature serves two purposes: &lt;br&gt;</span></span>
<span id="cb1-218"><a href="#cb1-218" aria-hidden="true" tabindex="-1"></a><span class="st">    Denaturation: Separates double-stranded RNA/DNA into single strands, making them accessible for primers. &lt;br&gt;</span></span>
<span id="cb1-219"><a href="#cb1-219" aria-hidden="true" tabindex="-1"></a><span class="st">    Hot-start activation: Activates the hot-start Taq polymerase, which prevents non-specific amplification at lower temperatures."</span>,</span>
<span id="cb1-220"><a href="#cb1-220" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeated in each cycle, this step ensures complete separation of newly synthesized DNA strands, allowing primers to bind in the next step."</span>,</span>
<span id="cb1-221"><a href="#cb1-221" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The primers, designed to be complementary to specific regions of the target DNA, bind (anneal) to their respective targets. Lowering the temperature facilitates this binding."</span>,</span>
<span id="cb1-222"><a href="#cb1-222" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The Taq polymerase extends the primers, synthesizing new complementary DNA strands. This results in the doubling of the target DNA in each cycle."</span>,</span>
<span id="cb1-223"><a href="#cb1-223" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The duration varies (usually a brief step at a temperature between annealing and extension) &lt;br&gt;</span></span>
<span id="cb1-224"><a href="#cb1-224" aria-hidden="true" tabindex="-1"></a><span class="st">    FAM Fluorescence: The machine measures the fluorescence of the FAM dye, which is typically incorporated into the amplifying DNA (either through a probe or intercalating dye like SYBR Green). The FAM signal increases proportionally with the amount of target DNA. &lt;br&gt;</span></span>
<span id="cb1-225"><a href="#cb1-225" aria-hidden="true" tabindex="-1"></a><span class="st">    ROX Fluorescence: The ROX dye acts as a passive reference. Its fluorescence should remain constant throughout the run. The machine uses the ROX signal to normalize the FAM signal, correcting for well-to-well variations and potential errors in pipetting."</span> </span>
<span id="cb1-226"><a href="#cb1-226" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-227"><a href="#cb1-227" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-228"><a href="#cb1-228" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-229"><a href="#cb1-229" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(RTqPCR_mechanism, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-230"><a href="#cb1-230" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-231"><a href="#cb1-231" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-232"><a href="#cb1-232" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"80%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-233"><a href="#cb1-233" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>)</span>
<span id="cb1-234"><a href="#cb1-234" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-235"><a href="#cb1-235" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-236"><a href="#cb1-236" aria-hidden="true" tabindex="-1"></a><span class="fu">### Organizational Stuffs</span></span>
<span id="cb1-237"><a href="#cb1-237" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-238"><a href="#cb1-238" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Reagents are in -20°C fridge (opposite of -80°C fridge).\</span>
<span id="cb1-239"><a href="#cb1-239" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Turn on all the switches on the sample hood and rotate the key to turn on. Take the slash up (folded).\</span>
<span id="cb1-240"><a href="#cb1-240" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Look if the scissor is there to cut your plate and sealing accordingly.\</span>
<span id="cb1-241"><a href="#cb1-241" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Print out the design and keep there to organize the Master Mix (MM) and samples accordingly. Use FAM and ROX for this experiment.\</span>
<span id="cb1-242"><a href="#cb1-242" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Keep the enzyme mix always on the cooling block till use.\</span>
<span id="cb1-243"><a href="#cb1-243" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take gloves.\</span>
<span id="cb1-244"><a href="#cb1-244" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>The cooling blocks change color depending on how hot they are.\</span>
<span id="cb1-245"><a href="#cb1-245" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take 10µL, 20µL, 100µL, 200µL tip boxes if needed. Pipettes are there already inside the hoods.\</span>
<span id="cb1-246"><a href="#cb1-246" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Samples on ice.\</span>
<span id="cb1-247"><a href="#cb1-247" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take 1 black box and take ice in it (from downstairs, left hand side while entering). Put the ice boxes in it.\</span>
<span id="cb1-248"><a href="#cb1-248" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take another black box with plate &amp; sealing (touch only with gloves).\</span>
<span id="cb1-249"><a href="#cb1-249" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take 2 Eppendorf tube holders, one for sample and one for H2O, Buffer.\</span>
<span id="cb1-250"><a href="#cb1-250" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Touch the RNA sample and reagents only with gloves.\</span>
<span id="cb1-251"><a href="#cb1-251" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Keep FAM and Enzyme inside the black box, because it is color sensitive.\</span>
<span id="cb1-252"><a href="#cb1-252" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take the small white box to keep the well plate (with sealing) on the ice box inside it to remove potential contamination issue. Take out when needed after making the MM.\</span>
<span id="cb1-253"><a href="#cb1-253" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Clean the MM hood and Sample hood with alcohol first and then start working/putting things inside.\</span>
<span id="cb1-254"><a href="#cb1-254" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>The centrifuge machine for well plate works this way: Turn on --<span class="sc">\&gt;</span> Place the plate --<span class="sc">\&gt;</span> Run for 10-20 secs --<span class="sc">\&gt;</span> Take out --<span class="sc">\&gt;</span> Turn off\</span>
<span id="cb1-255"><a href="#cb1-255" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Use the first point to pipette off the 11.5 µL MM, don't push too much. A bit will remain in the tip. That's fine. It will be the same for all the wells. No air bubble (aerosol) will be formed this way.</span>
<span id="cb1-256"><a href="#cb1-256" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-257"><a href="#cb1-257" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-258"><a href="#cb1-258" aria-hidden="true" tabindex="-1"></a>RT_qPCR_org <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-259"><a href="#cb1-259" aria-hidden="true" tabindex="-1"></a>  <span class="at">Consumables =</span> <span class="fu">c</span>(</span>
<span id="cb1-260"><a href="#cb1-260" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Cooling blocks (blue and yellow) for PCR tubes and reagents"</span>,</span>
<span id="cb1-261"><a href="#cb1-261" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RNase-free water"</span>,</span>
<span id="cb1-262"><a href="#cb1-262" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2x RT-PCR buffer (has things for ROX)"</span>,</span>
<span id="cb1-263"><a href="#cb1-263" aria-hidden="true" tabindex="-1"></a>    <span class="st">"25x RT-PCR enzyme mix (Rev Transcriptase + Pol)"</span>,</span>
<span id="cb1-264"><a href="#cb1-264" aria-hidden="true" tabindex="-1"></a>    <span class="st">"FAM mix (Primer + Probe)"</span>,</span>
<span id="cb1-265"><a href="#cb1-265" aria-hidden="true" tabindex="-1"></a>    <span class="st">"96-Well-Plate + sealing"</span>,</span>
<span id="cb1-266"><a href="#cb1-266" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RNA Sample(s)"</span>,</span>
<span id="cb1-267"><a href="#cb1-267" aria-hidden="true" tabindex="-1"></a>    <span class="st">"A waste bag"</span>,</span>
<span id="cb1-268"><a href="#cb1-268" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Scissors"</span></span>
<span id="cb1-269"><a href="#cb1-269" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-270"><a href="#cb1-270" aria-hidden="true" tabindex="-1"></a>  <span class="at">Location =</span> <span class="fu">c</span>(</span>
<span id="cb1-271"><a href="#cb1-271" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Shelf XIV, middle room"</span>,</span>
<span id="cb1-272"><a href="#cb1-272" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Rack A, Box 3"</span>,</span>
<span id="cb1-273"><a href="#cb1-273" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Rack A, Box 3"</span>,</span>
<span id="cb1-274"><a href="#cb1-274" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Rack A, Box 3"</span>,</span>
<span id="cb1-275"><a href="#cb1-275" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Rack B2, Box 2"</span>,</span>
<span id="cb1-276"><a href="#cb1-276" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Top-right drawer of RNA side"</span>,</span>
<span id="cb1-277"><a href="#cb1-277" aria-hidden="true" tabindex="-1"></a>    <span class="st">"-80°C fridge (Rack N, Box 4)"</span>,</span>
<span id="cb1-278"><a href="#cb1-278" aria-hidden="true" tabindex="-1"></a>    <span class="st">"You know where"</span>,</span>
<span id="cb1-279"><a href="#cb1-279" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Inside MM hood or you take"</span></span>
<span id="cb1-280"><a href="#cb1-280" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-281"><a href="#cb1-281" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-282"><a href="#cb1-282" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-283"><a href="#cb1-283" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-284"><a href="#cb1-284" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate HTML table</span></span>
<span id="cb1-285"><a href="#cb1-285" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(RT_qPCR_org, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-286"><a href="#cb1-286" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-287"><a href="#cb1-287" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"65%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-288"><a href="#cb1-288" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"35%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-289"><a href="#cb1-289" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>)</span>
<span id="cb1-290"><a href="#cb1-290" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-291"><a href="#cb1-291" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-292"><a href="#cb1-292" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>No. of sample = depends on your experiment &amp; used in the equation to calculate the total volume of MM.\</span>
<span id="cb1-293"><a href="#cb1-293" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Take the waste bag out of MM hood to the sample hood, never the other way around.\</span>
<span id="cb1-294"><a href="#cb1-294" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Design the well-plate beforehand using correct fluorophores (FAM/ROX), standard samples, concentration of the standards, unknown and/or negative samples, NTC, biological/technical replicates, etc.\</span>
<span id="cb1-295"><a href="#cb1-295" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Prepare the reaction mix following this exact order. **&amp; See the points on precautions below:**</span>
<span id="cb1-296"><a href="#cb1-296" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-297"><a href="#cb1-297" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-298"><a href="#cb1-298" aria-hidden="true" tabindex="-1"></a>no_sample <span class="ot">&lt;-</span> <span class="dv">18</span></span>
<span id="cb1-299"><a href="#cb1-299" aria-hidden="true" tabindex="-1"></a>no_NTC <span class="ot">=</span> <span class="dv">2</span></span>
<span id="cb1-300"><a href="#cb1-300" aria-hidden="true" tabindex="-1"></a>no_repeat <span class="ot">=</span> <span class="dv">2</span></span>
<span id="cb1-301"><a href="#cb1-301" aria-hidden="true" tabindex="-1"></a>tot_well <span class="ot">=</span> no_sample<span class="sc">*</span>no_repeat <span class="sc">+</span> no_NTC</span>
<span id="cb1-302"><a href="#cb1-302" aria-hidden="true" tabindex="-1"></a>RT_qPCR <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-303"><a href="#cb1-303" aria-hidden="true" tabindex="-1"></a>  <span class="at">Reagent =</span> <span class="fu">c</span>(</span>
<span id="cb1-304"><a href="#cb1-304" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RNase-free water (100µL aliquot)"</span>,</span>
<span id="cb1-305"><a href="#cb1-305" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2x RT-PCR buffer (130µL aliquot?)"</span>,</span>
<span id="cb1-306"><a href="#cb1-306" aria-hidden="true" tabindex="-1"></a>    <span class="st">"FAM mix (10µL aliquot)"</span>,</span>
<span id="cb1-307"><a href="#cb1-307" aria-hidden="true" tabindex="-1"></a>    <span class="st">"25x RT-PCR enzyme mix (10µL aliquot)"</span>,</span>
<span id="cb1-308"><a href="#cb1-308" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Total vol of MM"</span>,</span>
<span id="cb1-309"><a href="#cb1-309" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RNA template (20µL)"</span>,</span>
<span id="cb1-310"><a href="#cb1-310" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Total vol of reaction mix"</span></span>
<span id="cb1-311"><a href="#cb1-311" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-312"><a href="#cb1-312" aria-hidden="true" tabindex="-1"></a>  <span class="at">x_1 =</span> <span class="fu">c</span>(<span class="fl">3.75</span>, <span class="fl">6.25</span>, <span class="fl">1.0</span>, <span class="fl">0.5</span>, <span class="fl">11.5</span>, <span class="dv">1</span>, <span class="fl">12.5</span>)</span>
<span id="cb1-313"><a href="#cb1-313" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-314"><a href="#cb1-314" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-315"><a href="#cb1-315" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate 20x efficiently AFTER the data frame is created</span></span>
<span id="cb1-316"><a href="#cb1-316" aria-hidden="true" tabindex="-1"></a>RT_qPCR<span class="sc">$</span>x_new <span class="ot">&lt;-</span> RT_qPCR<span class="sc">$</span>x_1 <span class="sc">*</span> tot_well</span>
<span id="cb1-317"><a href="#cb1-317" aria-hidden="true" tabindex="-1"></a><span class="co">#as.factor(RT_qPCR$x_1)</span></span>
<span id="cb1-318"><a href="#cb1-318" aria-hidden="true" tabindex="-1"></a><span class="co">#set the RNA sample row correctly, total amount/vol doesn't make sense here.</span></span>
<span id="cb1-319"><a href="#cb1-319" aria-hidden="true" tabindex="-1"></a>RT_qPCR<span class="sc">$</span>x_new <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(RT_qPCR<span class="sc">$</span>Reagent <span class="sc">==</span> <span class="st">"RNA template (20µL)"</span>, <span class="st">"x"</span>, RT_qPCR<span class="sc">$</span>x_new)</span>
<span id="cb1-320"><a href="#cb1-320" aria-hidden="true" tabindex="-1"></a>RT_qPCR<span class="sc">$</span>Mechanism <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb1-321"><a href="#cb1-321" aria-hidden="true" tabindex="-1"></a>  <span class="st">"To make the correct volume"</span>,</span>
<span id="cb1-322"><a href="#cb1-322" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Has buffering activity and ROX for internal control"</span>,</span>
<span id="cb1-323"><a href="#cb1-323" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Has primer pairs and probe for amplification and fluorophore detection, respectively"</span>,</span>
<span id="cb1-324"><a href="#cb1-324" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Has the polymerase for amplification"</span>,</span>
<span id="cb1-325"><a href="#cb1-325" aria-hidden="true" tabindex="-1"></a>  <span class="st">"X"</span>,</span>
<span id="cb1-326"><a href="#cb1-326" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Inital element/RNA for RT-qPCR"</span>,</span>
<span id="cb1-327"><a href="#cb1-327" aria-hidden="true" tabindex="-1"></a>  <span class="st">"X"</span>)</span>
<span id="cb1-328"><a href="#cb1-328" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-329"><a href="#cb1-329" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(RT_qPCR, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-330"><a href="#cb1-330" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-331"><a href="#cb1-331" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-332"><a href="#cb1-332" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"15%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-333"><a href="#cb1-333" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"15%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-334"><a href="#cb1-334" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">4</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-335"><a href="#cb1-335" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-336"><a href="#cb1-336" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Reagent"</span>, <span class="st">"1x (µL)"</span>, <span class="fu">paste0</span>(tot_well, <span class="st">"x (µL)"</span>), <span class="st">"Mechanism"</span>)))</span>
<span id="cb1-337"><a href="#cb1-337" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-338"><a href="#cb1-338" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-339"><a href="#cb1-339" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Cut the plate and the sealing accordingly using the scissors. Name/date &amp; orientation should be written.\</span>
<span id="cb1-340"><a href="#cb1-340" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Turn on air circulation, &amp; white light in the MM hood.\</span>
<span id="cb1-341"><a href="#cb1-341" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Vortex everything (MM, samples) before adding (so, prep the MM in a Eppendorf tube).\</span>
<span id="cb1-342"><a href="#cb1-342" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>**MM handling:** Put the MM in each and every well carefully. Don't release the pipette inside the well or near the plate. Keep it held. Release only on the waste bag and discard. Otherwise, you might take something from the well and make aerosol (contaminant).\</span>
<span id="cb1-343"><a href="#cb1-343" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>**Sample handling:** Don't press hard (second point), pipette multiple times and take carefully. Then put in the well carefully. Again follow the same procedure of pipetting.\</span>
<span id="cb1-344"><a href="#cb1-344" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Put on the sealing nicely; otherwise aerosol will form and disperse.\</span>
<span id="cb1-345"><a href="#cb1-345" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Clean, turn on the UV light in th MM hood.\</span>
<span id="cb1-346"><a href="#cb1-346" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Clean, turn off the switches in the sample hood. Turn on the UV. Go back after 20-30 mins to turn of the UV (for the sample hood, MM hood is automatic). Turn off using the key.\</span>
<span id="cb1-347"><a href="#cb1-347" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Centrifuge the plate in the other lab (Short pulse for 20 secs). **Procedure:** Open lid --<span class="sc">\&gt;</span> Place on the plate --<span class="sc">\&gt;</span> Switch on --<span class="sc">\&gt;</span> Close lid --<span class="sc">\&gt;</span> Hold the button for "Impuls" for 20 secs --<span class="sc">\&gt;</span> Take out and turn off\</span>
<span id="cb1-348"><a href="#cb1-348" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>RT-qPCR running: Open the program --<span class="sc">\&gt;</span> Click on "open lid" --<span class="sc">\&gt;</span> place on the well-plate as designed --<span class="sc">\&gt;</span> Put a balance for even heat circulation (if not using the full plate) --<span class="sc">\&gt;</span> Click on "close lid" --<span class="sc">\&gt;</span> Choose the right options.\</span>
<span id="cb1-349"><a href="#cb1-349" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Run the RT-qPCR using the right protocol (<span class="in">`WNV-Ag-ID-Path.prcl`</span>) and set the right location for saving the result.\</span>
<span id="cb1-350"><a href="#cb1-350" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span>Open the result using Bio-Rad CFX Maestro and explore!</span>
<span id="cb1-351"><a href="#cb1-351" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-352"><a href="#cb1-352" aria-hidden="true" tabindex="-1"></a><span class="fu">### Troubleshooting: Reverse Transcription</span></span>
<span id="cb1-353"><a href="#cb1-353" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-354"><a href="#cb1-354" aria-hidden="true" tabindex="-1"></a><span class="ss">-   </span><span class="co">[</span><span class="ot">The main paper</span><span class="co">](https://www.tandfonline.com/doi/pdf/10.2144/01304pf02)</span> should help us understanding and designing the primers. Primer (1) and (3) are needed.</span>
<span id="cb1-355"><a href="#cb1-355" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-356"><a href="#cb1-356" aria-hidden="true" tabindex="-1"></a><span class="kw">&lt;iframe</span> <span class="er">src</span><span class="ot">=</span><span class="st">"../../pdf/cDNA_MMLV.pdf"</span> <span class="er">width</span><span class="ot">=</span><span class="st">"100%"</span> <span class="er">height</span><span class="ot">=</span><span class="st">"600px"</span><span class="kw">&gt;</span></span>
<span id="cb1-357"><a href="#cb1-357" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-358"><a href="#cb1-358" aria-hidden="true" tabindex="-1"></a><span class="kw">&lt;/iframe&gt;</span></span>
<span id="cb1-359"><a href="#cb1-359" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-360"><a href="#cb1-360" aria-hidden="true" tabindex="-1"></a>The ONT protocol for direct cDNA sequencing also provides with the information about the primers required.</span>
<span id="cb1-361"><a href="#cb1-361" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-362"><a href="#cb1-362" aria-hidden="true" tabindex="-1"></a>::: {.column width="90%"}</span>
<span id="cb1-363"><a href="#cb1-363" aria-hidden="true" tabindex="-1"></a><span class="al">![cDNA conversion by ONT](../../img/cDNA_ONT.png)</span></span>
<span id="cb1-364"><a href="#cb1-364" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb1-365"><a href="#cb1-365" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-366"><a href="#cb1-366" aria-hidden="true" tabindex="-1"></a><span class="co">[</span><span class="ot">Another website</span><span class="co">]</span>(https://eu.idtdna.com/pages/education/decoded/article/use-of-template-switching-oligos-(ts-oligos-tsos)-for-efficient-cdna-library-construction) is quite good for understanding as well.</span>
<span id="cb1-367"><a href="#cb1-367" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-368"><a href="#cb1-368" aria-hidden="true" tabindex="-1"></a>**Here goes the step-by-step mechanism.**</span>
<span id="cb1-369"><a href="#cb1-369" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-370"><a href="#cb1-370" aria-hidden="true" tabindex="-1"></a><span class="al">![cDNA conversion mechanism](../../Video/cDNA_mechanism.gif)</span>{width="100%"}</span>
<span id="cb1-371"><a href="#cb1-371" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-372"><a href="#cb1-372" aria-hidden="true" tabindex="-1"></a><span class="fu">### DNase I treatment</span></span>
<span id="cb1-373"><a href="#cb1-373" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-374"><a href="#cb1-374" aria-hidden="true" tabindex="-1"></a><span class="fu">### Removal of DNase I</span></span>
<span id="cb1-375"><a href="#cb1-375" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-376"><a href="#cb1-376" aria-hidden="true" tabindex="-1"></a><span class="fu">### Polyadenylating using E. coli poly(A) polymerase</span></span>
<span id="cb1-377"><a href="#cb1-377" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-378"><a href="#cb1-378" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-379"><a href="#cb1-379" aria-hidden="true" tabindex="-1"></a>polyadenylation_protocol <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-380"><a href="#cb1-380" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-381"><a href="#cb1-381" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Start with clean, non-polyadenylated RNA of interest. The sample should be EDTA-free, with no salts present."</span>,</span>
<span id="cb1-382"><a href="#cb1-382" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. In a 1.5 ml microcentrifuge tube, set up 3\u2019 polyadenylation reaction as follows: &lt;br&gt; </span></span>
<span id="cb1-383"><a href="#cb1-383" aria-hidden="true" tabindex="-1"></a><span class="st">     * Non-polyadenylated RNA X \u03BCl \u2264 10 \u03BCg &lt;br&gt;</span></span>
<span id="cb1-384"><a href="#cb1-384" aria-hidden="true" tabindex="-1"></a><span class="st">     * 10X E. coli poly(A) polymerase buffer 2 \u03BCl 1X &lt;br&gt;</span></span>
<span id="cb1-385"><a href="#cb1-385" aria-hidden="true" tabindex="-1"></a><span class="st">     * ATP (10 mM) 2 \u03BCl 1 mM &lt;br&gt;</span></span>
<span id="cb1-386"><a href="#cb1-386" aria-hidden="true" tabindex="-1"></a><span class="st">     * Nuclease-free water 15-X \u03BCl &lt;br&gt;</span></span>
<span id="cb1-387"><a href="#cb1-387" aria-hidden="true" tabindex="-1"></a><span class="st">     * E. coli poly(A) polymerase (5 U/\u03BCl) 1 \u03BCl 5U &lt;br&gt;</span></span>
<span id="cb1-388"><a href="#cb1-388" aria-hidden="true" tabindex="-1"></a><span class="st">     * TOTAL 20 \u03BCl"</span>,</span>
<span id="cb1-389"><a href="#cb1-389" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Incubate at 37\u00B0C for 0.5\u20131.5 minutes."</span>,</span>
<span id="cb1-390"><a href="#cb1-390" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. After 0.5\u20131.5 minutes of incubation, stop the reaction by adding 5 \u03BCl of 50 mM EDTA (to a final concentration of 10 mM EDTA). Final volume will be 25 \u03BCl."</span>,</span>
<span id="cb1-391"><a href="#cb1-391" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Add 45 \u03BCl of RNase-free SPRI beads to the reaction."</span>,</span>
<span id="cb1-392"><a href="#cb1-392" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Incubate on a Hula Mixer for 5 minutes at room temperature."</span>,</span>
<span id="cb1-393"><a href="#cb1-393" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Spin down the sample and pellet on a magnet. Keep the tube on the magnet and pipette off and discard the supernatant."</span>,</span>
<span id="cb1-394"><a href="#cb1-394" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Keep the tube on the magnet and wash the beads with 200 \u03BCl of freshly-prepared 70% ethanol. Carefully turn the tube 180\u00B0 twice in the rack to wash pelleted beads. Pippette off and discard the supernatant."</span>,</span>
<span id="cb1-395"><a href="#cb1-395" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Repeat the previous step for a total of two washes."</span>,</span>
<span id="cb1-396"><a href="#cb1-396" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Briefly spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for 30 seconds, but do not dry the pellet to the point of cracking."</span>,</span>
<span id="cb1-397"><a href="#cb1-397" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Remove the tube from the magnetic rack and resuspend the pellet in 12 \u03BCl of nuclease-free water. Incubate on ice for 5 minutes."</span>,</span>
<span id="cb1-398"><a href="#cb1-398" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Pellet the beads on a magnet until the eluate is clear and colourless."</span>,</span>
<span id="cb1-399"><a href="#cb1-399" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Remove and retain 12 \u03BCl of eluate containing the 3\u2019-polyadenylated RNA in a clean 1.5 ml microcentrifuge tube."</span>,</span>
<span id="cb1-400"><a href="#cb1-400" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Remove 1 \u03BCl of final eluate and quantify the final concentration using HS RNA Qubit kit."</span>,</span>
<span id="cb1-401"><a href="#cb1-401" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Optional step: Dilute the final eluate to 100 ng/\u03BCl and quantify 1 \u03BCl using Agilent 6000 Nano RNA kit, using total RNA or mRNA setting depending on the sample."</span>,</span>
<span id="cb1-402"><a href="#cb1-402" aria-hidden="true" tabindex="-1"></a>    <span class="st">"15. Store the final product at -80\u00B0C."</span></span>
<span id="cb1-403"><a href="#cb1-403" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-404"><a href="#cb1-404" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-405"><a href="#cb1-405" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-406"><a href="#cb1-406" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures that the RNA sample is free of contaminants that could interfere with the polyadenylation reaction."</span>,</span>
<span id="cb1-407"><a href="#cb1-407" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the RNA template with the necessary reagents for polyadenylation, including the enzyme, buffer, ATP (the source of adenosine nucleotides), and nuclease-free water to adjust the final volume."</span>,</span>
<span id="cb1-408"><a href="#cb1-408" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides the optimal temperature for the E. coli poly(A) polymerase enzyme to add adenosine nucleotides to the 3' end of the RNA molecules."</span>,</span>
<span id="cb1-409"><a href="#cb1-409" aria-hidden="true" tabindex="-1"></a>    <span class="st">"EDTA chelates divalent cations (like Mg2+), which are essential for the polymerase enzyme's activity, effectively stopping the reaction."</span>,</span>
<span id="cb1-410"><a href="#cb1-410" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads selectively bind to RNA, allowing for purification and removal of contaminants like enzymes, salts, and unincorporated nucleotides."</span>,</span>
<span id="cb1-411"><a href="#cb1-411" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mixing promotes efficient binding of RNA to the beads."</span>,</span>
<span id="cb1-412"><a href="#cb1-412" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation facilitates removal of unwanted components in the supernatant."</span>,</span>
<span id="cb1-413"><a href="#cb1-413" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol washes remove residual contaminants and salts from the beads."</span>,</span>
<span id="cb1-414"><a href="#cb1-414" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-415"><a href="#cb1-415" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any remaining ethanol, which can interfere with downstream steps."</span>,</span>
<span id="cb1-416"><a href="#cb1-416" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elution buffer releases the purified RNA from the beads. Incubation on ice helps maintain RNA integrity."</span>,</span>
<span id="cb1-417"><a href="#cb1-417" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified RNA in the supernatant."</span>,</span>
<span id="cb1-418"><a href="#cb1-418" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified, polyadenylated RNA for further analysis or downstream applications."</span>,</span>
<span id="cb1-419"><a href="#cb1-419" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Measures the concentration of the purified, polyadenylated RNA to assess yield and quality."</span>,</span>
<span id="cb1-420"><a href="#cb1-420" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides additional information about RNA size distribution and integrity."</span>,</span>
<span id="cb1-421"><a href="#cb1-421" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Long-term storage preserves the RNA for future use."</span></span>
<span id="cb1-422"><a href="#cb1-422" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-423"><a href="#cb1-423" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-424"><a href="#cb1-424" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-425"><a href="#cb1-425" aria-hidden="true" tabindex="-1"></a>    <span class="st">"If the RNA sample contains EDTA or salts, perform a cleanup step using ethanol precipitation or a commercial RNA purification kit."</span>,</span>
<span id="cb1-426"><a href="#cb1-426" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate pipetting, especially with small volumes. Use high-quality, RNase-free reagents."</span>,</span>
<span id="cb1-427"><a href="#cb1-427" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate temperature control and timing during incubation. Longer incubation times will result in longer poly(A) tails, but may decrease overall sequencing yield."</span>,</span>
<span id="cb1-428"><a href="#cb1-428" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure thorough mixing after adding EDTA to stop the reaction completely."</span>,</span>
<span id="cb1-429"><a href="#cb1-429" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio."</span>,</span>
<span id="cb1-430"><a href="#cb1-430" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a thermomixer or rotator for consistent mixing."</span>,</span>
<span id="cb1-431"><a href="#cb1-431" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and avoid disturbing the beads."</span>,</span>
<span id="cb1-432"><a href="#cb1-432" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 70% ethanol. Carefully turn the tube to ensure all beads are washed."</span>,</span>
<span id="cb1-433"><a href="#cb1-433" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat washing if the supernatant is still cloudy."</span>,</span>
<span id="cb1-434"><a href="#cb1-434" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid over-drying the beads, which can make elution difficult."</span>,</span>
<span id="cb1-435"><a href="#cb1-435" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use nuclease-free water for elution. Ensure complete resuspension of the beads."</span>,</span>
<span id="cb1-436"><a href="#cb1-436" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-437"><a href="#cb1-437" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span>,</span>
<span id="cb1-438"><a href="#cb1-438" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate calibration of the Qubit fluorometer. If yield is low, troubleshoot previous steps."</span>,</span>
<span id="cb1-439"><a href="#cb1-439" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper chip preparation and instrument calibration for the Agilent Bioanalyzer."</span>,</span>
<span id="cb1-440"><a href="#cb1-440" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Maintain consistent -80\u00B0C storage to prevent RNA degradation."</span></span>
<span id="cb1-441"><a href="#cb1-441" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-442"><a href="#cb1-442" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-443"><a href="#cb1-443" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-444"><a href="#cb1-444" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(polyadenylation_protocol, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-445"><a href="#cb1-445" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-446"><a href="#cb1-446" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-447"><a href="#cb1-447" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-448"><a href="#cb1-448" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-449"><a href="#cb1-449" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-450"><a href="#cb1-450" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-451"><a href="#cb1-451" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-452"><a href="#cb1-452" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-453"><a href="#cb1-453" aria-hidden="true" tabindex="-1"></a><span class="fu">### Reverse Transcription and Strand-switching</span></span>
<span id="cb1-454"><a href="#cb1-454" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-455"><a href="#cb1-455" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-456"><a href="#cb1-456" aria-hidden="true" tabindex="-1"></a>rt_ss <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-457"><a href="#cb1-457" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-458"><a href="#cb1-458" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Prepare the following reaction in a 0.2 ml PCR tube: &lt;br&gt;</span></span>
<span id="cb1-459"><a href="#cb1-459" aria-hidden="true" tabindex="-1"></a><span class="st">    </span></span>
<span id="cb1-460"><a href="#cb1-460" aria-hidden="true" tabindex="-1"></a><span class="st">      x μl poly A+ RNA, 100 ng &lt;br&gt;</span></span>
<span id="cb1-461"><a href="#cb1-461" aria-hidden="true" tabindex="-1"></a><span class="st">      2.5 μl VNP &lt;br&gt;</span></span>
<span id="cb1-462"><a href="#cb1-462" aria-hidden="true" tabindex="-1"></a><span class="st">      1 μl 10 mM dNTPs &lt;br&gt;</span></span>
<span id="cb1-463"><a href="#cb1-463" aria-hidden="true" tabindex="-1"></a><span class="st">      7.5-x μl RNase-free water"</span>,</span>
<span id="cb1-464"><a href="#cb1-464" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Mix gently by flicking the tube, and spin down"</span>,</span>
<span id="cb1-465"><a href="#cb1-465" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Incubate at 65°C for 5 minutes and then snap cool on a pre-chilled freezer block for 1 minute"</span>,</span>
<span id="cb1-466"><a href="#cb1-466" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. In a separate tube, mix together the following: &lt;br&gt;</span></span>
<span id="cb1-467"><a href="#cb1-467" aria-hidden="true" tabindex="-1"></a><span class="st">    </span></span>
<span id="cb1-468"><a href="#cb1-468" aria-hidden="true" tabindex="-1"></a><span class="st">      4 μl 5x RT Buffer &lt;br&gt;</span></span>
<span id="cb1-469"><a href="#cb1-469" aria-hidden="true" tabindex="-1"></a><span class="st">      1 μl RNaseOUT &lt;br&gt;</span></span>
<span id="cb1-470"><a href="#cb1-470" aria-hidden="true" tabindex="-1"></a><span class="st">      1 μl Nuclease-free water &lt;br&gt;</span></span>
<span id="cb1-471"><a href="#cb1-471" aria-hidden="true" tabindex="-1"></a><span class="st">      2 μl Strand-Switching Primer (SSP)"</span>,</span>
<span id="cb1-472"><a href="#cb1-472" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Mix gently by flicking the tube, and spin down."</span>,</span>
<span id="cb1-473"><a href="#cb1-473" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Add the strand-switching buffer to the snap-cooled, annealed mRNA, mix by flicking the tube and spin down."</span>,</span>
<span id="cb1-474"><a href="#cb1-474" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Incubate at 42°C for 2 minutes in the thermal cycler."</span>,</span>
<span id="cb1-475"><a href="#cb1-475" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Add 1 µl of Maxima H Minus Reverse Transcriptase. The total volume is now 20 µl."</span>,</span>
<span id="cb1-476"><a href="#cb1-476" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Mix gently by flicking the tube, and spin down."</span>,</span>
<span id="cb1-477"><a href="#cb1-477" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Incubate using the following protocol using a thermal cycler: &lt;br&gt;</span></span>
<span id="cb1-478"><a href="#cb1-478" aria-hidden="true" tabindex="-1"></a><span class="st">    Reverse transcription and strand-switching 90 mins @ 42°C (1 cycle) &lt;br&gt;</span></span>
<span id="cb1-479"><a href="#cb1-479" aria-hidden="true" tabindex="-1"></a><span class="st">    Heat inactivation 5 mins @ 85°C (1 cycle) &lt;br&gt;</span></span>
<span id="cb1-480"><a href="#cb1-480" aria-hidden="true" tabindex="-1"></a><span class="st">    Hold @ 4°C"</span></span>
<span id="cb1-481"><a href="#cb1-481" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-482"><a href="#cb1-482" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-483"><a href="#cb1-483" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines mRNA template, VNP (a template switching oligo), dNTPs, and water. VNP anneals to the polyA tail of mRNA and provides a template for reverse transcription."</span>,</span>
<span id="cb1-484"><a href="#cb1-484" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures all components are well mixed and collects any condensation on the tube walls."</span>,</span>
<span id="cb1-485"><a href="#cb1-485" aria-hidden="true" tabindex="-1"></a>    <span class="st">"High temperature denatures RNA secondary structures. Snap cooling prevents re-formation of secondary structures and promotes annealing of VNP to the polyA tail."</span>,</span>
<span id="cb1-486"><a href="#cb1-486" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepares the reverse transcription and strand-switching buffer containing the necessary reagents."</span>,</span>
<span id="cb1-487"><a href="#cb1-487" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures thorough mixing of components."</span>,</span>
<span id="cb1-488"><a href="#cb1-488" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the mRNA template with the RT and strand-switching buffer."</span>,</span>
<span id="cb1-489"><a href="#cb1-489" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Allows the SSP to anneal to the mRNA/cDNA hybrid."</span>,</span>
<span id="cb1-490"><a href="#cb1-490" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Adds the reverse transcriptase enzyme."</span>,</span>
<span id="cb1-491"><a href="#cb1-491" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures proper mixing of the enzyme with the reaction mixture."</span>,</span>
<span id="cb1-492"><a href="#cb1-492" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Reverse transcriptase synthesizes cDNA from the mRNA template, and the strand-switching activity of the enzyme generates the second cDNA strand. Heat inactivation denatures the enzyme and stops the reaction."</span></span>
<span id="cb1-493"><a href="#cb1-493" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-494"><a href="#cb1-494" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-495"><a href="#cb1-495" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate quantification and integrity of RNA. Use high-quality, RNase-free water."</span>,</span>
<span id="cb1-496"><a href="#cb1-496" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid vigorous mixing, which can shear RNA."</span>,</span>
<span id="cb1-497"><a href="#cb1-497" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a pre-chilled freezer block for efficient snap cooling."</span>,</span>
<span id="cb1-498"><a href="#cb1-498" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare the buffer fresh and keep it on ice. Ensure RNase-free conditions."</span>,</span>
<span id="cb1-499"><a href="#cb1-499" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid vigorous mixing, which can shear RNA."</span>,</span>
<span id="cb1-500"><a href="#cb1-500" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix gently to avoid disrupting the annealed mRNA/VNP complex."</span>,</span>
<span id="cb1-501"><a href="#cb1-501" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate temperature and timing in the thermal cycler."</span>,</span>
<span id="cb1-502"><a href="#cb1-502" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Keep the enzyme on ice until use. Avoid pipetting errors."</span>,</span>
<span id="cb1-503"><a href="#cb1-503" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid vigorous mixing, which can shear the cDNA."</span>,</span>
<span id="cb1-504"><a href="#cb1-504" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate temperature and timing in the thermal cycler."</span></span>
<span id="cb1-505"><a href="#cb1-505" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-506"><a href="#cb1-506" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-507"><a href="#cb1-507" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-508"><a href="#cb1-508" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(rt_ss, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-509"><a href="#cb1-509" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-510"><a href="#cb1-510" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-511"><a href="#cb1-511" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-512"><a href="#cb1-512" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-513"><a href="#cb1-513" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-514"><a href="#cb1-514" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-515"><a href="#cb1-515" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-516"><a href="#cb1-516" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-517"><a href="#cb1-517" aria-hidden="true" tabindex="-1"></a><span class="fu">### RNA Degradation and Second Strand Synthesis</span></span>
<span id="cb1-518"><a href="#cb1-518" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-519"><a href="#cb1-519" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-520"><a href="#cb1-520" aria-hidden="true" tabindex="-1"></a>RNA_deg <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-521"><a href="#cb1-521" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"1. Add 1 µl RiboShredder or RNase Cocktail Enzyme Mix (ThermoFisher, cat # AM2286) to the reverse transcription reaction."</span>,</span>
<span id="cb1-522"><a href="#cb1-522" aria-hidden="true" tabindex="-1"></a><span class="st">"2. Incubate the reaction for 10 minutes at 37° C in a thermal cycler."</span>,</span>
<span id="cb1-523"><a href="#cb1-523" aria-hidden="true" tabindex="-1"></a><span class="st">"3. Resuspend the AMPure XP beads by vortexing."</span>,</span>
<span id="cb1-524"><a href="#cb1-524" aria-hidden="true" tabindex="-1"></a><span class="st">"4. Transfer the sample to a clean 1.5 ml Eppendorf DNA LoBind tube."</span>,</span>
<span id="cb1-525"><a href="#cb1-525" aria-hidden="true" tabindex="-1"></a><span class="st">"5. Add 17 µl of resuspended AMPure XP beads to the reaction and mix by flicking the tube."</span>,</span>
<span id="cb1-526"><a href="#cb1-526" aria-hidden="true" tabindex="-1"></a><span class="st">"6. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT."</span>,</span>
<span id="cb1-527"><a href="#cb1-527" aria-hidden="true" tabindex="-1"></a><span class="st">"7. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water."</span>,</span>
<span id="cb1-528"><a href="#cb1-528" aria-hidden="true" tabindex="-1"></a><span class="st">"8. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant."</span>,</span>
<span id="cb1-529"><a href="#cb1-529" aria-hidden="true" tabindex="-1"></a><span class="st">"9. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet. Remove the ethanol using a pipette and discard."</span>,</span>
<span id="cb1-530"><a href="#cb1-530" aria-hidden="true" tabindex="-1"></a><span class="st">"10. Repeat the previous step."</span>,</span>
<span id="cb1-531"><a href="#cb1-531" aria-hidden="true" tabindex="-1"></a><span class="st">"11. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking."</span>,</span>
<span id="cb1-532"><a href="#cb1-532" aria-hidden="true" tabindex="-1"></a><span class="st">"12. Remove the tube from the magnetic rack and resuspend pellet in 20 µl Nuclease-free water."</span>,</span>
<span id="cb1-533"><a href="#cb1-533" aria-hidden="true" tabindex="-1"></a><span class="st">"13. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT."</span>,</span>
<span id="cb1-534"><a href="#cb1-534" aria-hidden="true" tabindex="-1"></a><span class="st">"14. Pellet the beads on the magnet until the eluate is clear and colourless."</span>,</span>
<span id="cb1-535"><a href="#cb1-535" aria-hidden="true" tabindex="-1"></a><span class="st">"15. Remove and retain 20 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube."</span>,</span>
<span id="cb1-536"><a href="#cb1-536" aria-hidden="true" tabindex="-1"></a><span class="st">"16. Prepare the following reaction in a 0.2 ml thin-walled PCR tube: &lt;br&gt;</span></span>
<span id="cb1-537"><a href="#cb1-537" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-538"><a href="#cb1-538" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; 25 μl 2x LongAmp Taq Master Mix &lt;br&gt;</span></span>
<span id="cb1-539"><a href="#cb1-539" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; 2 μl PR2 Primer, 10 μM &lt;br&gt;</span></span>
<span id="cb1-540"><a href="#cb1-540" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; 20 μl Reverse-transcribed sample from above &lt;br&gt;</span></span>
<span id="cb1-541"><a href="#cb1-541" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; 3 μl Nuclease-free water"</span>,</span>
<span id="cb1-542"><a href="#cb1-542" aria-hidden="true" tabindex="-1"></a><span class="st">"17. Incubate using the following protocol: &lt;br&gt;</span></span>
<span id="cb1-543"><a href="#cb1-543" aria-hidden="true" tabindex="-1"></a><span class="st">94°C &lt;br&gt;</span></span>
<span id="cb1-544"><a href="#cb1-544" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-545"><a href="#cb1-545" aria-hidden="true" tabindex="-1"></a><span class="st">- Time: 1 mins &lt;br&gt;</span></span>
<span id="cb1-546"><a href="#cb1-546" aria-hidden="true" tabindex="-1"></a><span class="st">- No. of cycles: 1 &lt;br&gt;</span></span>
<span id="cb1-547"><a href="#cb1-547" aria-hidden="true" tabindex="-1"></a><span class="st">50°C &lt;br&gt;</span></span>
<span id="cb1-548"><a href="#cb1-548" aria-hidden="true" tabindex="-1"></a><span class="st">- Time: 1 mins &lt;br&gt;</span></span>
<span id="cb1-549"><a href="#cb1-549" aria-hidden="true" tabindex="-1"></a><span class="st">- No. of cycles: 1 &lt;br&gt;</span></span>
<span id="cb1-550"><a href="#cb1-550" aria-hidden="true" tabindex="-1"></a><span class="st">65°C &lt;br&gt;</span></span>
<span id="cb1-551"><a href="#cb1-551" aria-hidden="true" tabindex="-1"></a><span class="st">- Time: 15 mins &lt;br&gt;</span></span>
<span id="cb1-552"><a href="#cb1-552" aria-hidden="true" tabindex="-1"></a><span class="st">- No. of cycles: 1 &lt;br&gt;</span></span>
<span id="cb1-553"><a href="#cb1-553" aria-hidden="true" tabindex="-1"></a><span class="st">4°C &lt;br&gt;</span></span>
<span id="cb1-554"><a href="#cb1-554" aria-hidden="true" tabindex="-1"></a><span class="st">- Time: ∞"</span>,</span>
<span id="cb1-555"><a href="#cb1-555" aria-hidden="true" tabindex="-1"></a><span class="st">"18. Resuspend the AMPure XP beads by vortexing."</span>,</span>
<span id="cb1-556"><a href="#cb1-556" aria-hidden="true" tabindex="-1"></a><span class="st">"19. Transfer the sample to a clean 1.5 ml Eppendorf DNA LoBind tube."</span>,</span>
<span id="cb1-557"><a href="#cb1-557" aria-hidden="true" tabindex="-1"></a><span class="st">"20. Add 40 µl of resuspended AMPure XP beads to the reaction and mix by flicking the tube."</span>,</span>
<span id="cb1-558"><a href="#cb1-558" aria-hidden="true" tabindex="-1"></a><span class="st">"21. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT."</span>,</span>
<span id="cb1-559"><a href="#cb1-559" aria-hidden="true" tabindex="-1"></a><span class="st">"22. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water."</span>,</span>
<span id="cb1-560"><a href="#cb1-560" aria-hidden="true" tabindex="-1"></a><span class="st">"23. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant."</span>,</span>
<span id="cb1-561"><a href="#cb1-561" aria-hidden="true" tabindex="-1"></a><span class="st">"24. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet."</span>,</span>
<span id="cb1-562"><a href="#cb1-562" aria-hidden="true" tabindex="-1"></a><span class="st">"25. Remove the ethanol using a pipette and discard."</span>,</span>
<span id="cb1-563"><a href="#cb1-563" aria-hidden="true" tabindex="-1"></a><span class="st">"26. Repeat the previous step."</span>,</span>
<span id="cb1-564"><a href="#cb1-564" aria-hidden="true" tabindex="-1"></a><span class="st">"27. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking."</span>,</span>
<span id="cb1-565"><a href="#cb1-565" aria-hidden="true" tabindex="-1"></a><span class="st">"28. Remove the tube from the magnetic rack and resuspend pellet in 21 µl Nuclease-free water."</span>,</span>
<span id="cb1-566"><a href="#cb1-566" aria-hidden="true" tabindex="-1"></a><span class="st">"29. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT."</span>,</span>
<span id="cb1-567"><a href="#cb1-567" aria-hidden="true" tabindex="-1"></a><span class="st">"30. Pellet the beads on the magnet until the eluate is clear and colourless."</span>,</span>
<span id="cb1-568"><a href="#cb1-568" aria-hidden="true" tabindex="-1"></a><span class="st">"31. Remove and retain 21 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube."</span>,</span>
<span id="cb1-569"><a href="#cb1-569" aria-hidden="true" tabindex="-1"></a><span class="st">"32. Analyse 1 µl of the strand-switched DNA for size, quantity and quality."</span></span>
<span id="cb1-570"><a href="#cb1-570" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-571"><a href="#cb1-571" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-572"><a href="#cb1-572" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-573"><a href="#cb1-573" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Degrades any remaining RNA, preventing it from interfering with downstream applications."</span>,</span>
<span id="cb1-574"><a href="#cb1-574" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides optimal temperature for RNase activity."</span>,</span>
<span id="cb1-575"><a href="#cb1-575" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures consistent bead suspension for efficient binding of cDNA."</span>,</span>
<span id="cb1-576"><a href="#cb1-576" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Transfers the sample to a tube compatible with magnetic separation."</span>,</span>
<span id="cb1-577"><a href="#cb1-577" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads selectively bind to cDNA, allowing for purification."</span>,</span>
<span id="cb1-578"><a href="#cb1-578" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mixing promotes efficient binding of cDNA to the beads."</span>,</span>
<span id="cb1-579"><a href="#cb1-579" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepares ethanol for washing the beads."</span>,</span>
<span id="cb1-580"><a href="#cb1-580" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic beads are attracted to the magnet, separating them from the supernatant."</span>,</span>
<span id="cb1-581"><a href="#cb1-581" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol wash removes impurities and salts from the beads."</span>,</span>
<span id="cb1-582"><a href="#cb1-582" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-583"><a href="#cb1-583" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any remaining ethanol."</span>,</span>
<span id="cb1-584"><a href="#cb1-584" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elute the purified cDNA from the beads."</span>,</span>
<span id="cb1-585"><a href="#cb1-585" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Allows the cDNA to rehydrate and detach from the beads."</span>,</span>
<span id="cb1-586"><a href="#cb1-586" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified cDNA in the supernatant."</span>,</span>
<span id="cb1-587"><a href="#cb1-587" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified cDNA for downstream applications."</span>,</span>
<span id="cb1-588"><a href="#cb1-588" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Sets up the PCR reaction with LongAmp Taq polymerase, primers, and the cDNA template."</span>,</span>
<span id="cb1-589"><a href="#cb1-589" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Initial denaturation, primer annealing, extension (second strand synthesis), and final hold."</span>,</span>
<span id="cb1-590"><a href="#cb1-590" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Resuspends the AMPure XP beads for purification."</span>,</span>
<span id="cb1-591"><a href="#cb1-591" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Transfers the sample to a tube compatible with magnetic separation."</span>,</span>
<span id="cb1-592"><a href="#cb1-592" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads bind to the amplified cDNA, allowing for purification."</span>,</span>
<span id="cb1-593"><a href="#cb1-593" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mixing promotes efficient binding of cDNA to the beads."</span>,</span>
<span id="cb1-594"><a href="#cb1-594" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepares ethanol for washing the beads."</span>,</span>
<span id="cb1-595"><a href="#cb1-595" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, separating them from the supernatant."</span>,</span>
<span id="cb1-596"><a href="#cb1-596" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol wash removes impurities and salts from the beads."</span>,</span>
<span id="cb1-597"><a href="#cb1-597" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes the ethanol wash."</span>,</span>
<span id="cb1-598"><a href="#cb1-598" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-599"><a href="#cb1-599" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any remaining ethanol."</span>,</span>
<span id="cb1-600"><a href="#cb1-600" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elute the purified double-stranded cDNA from the beads."</span>,</span>
<span id="cb1-601"><a href="#cb1-601" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Allows the cDNA to rehydrate and detach from the beads."</span>,</span>
<span id="cb1-602"><a href="#cb1-602" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified cDNA in the supernatant."</span>,</span>
<span id="cb1-603"><a href="#cb1-603" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified cDNA for downstream applications."</span>,</span>
<span id="cb1-604"><a href="#cb1-604" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Assess the size distribution, concentration, and integrity of the cDNA to ensure it's suitable for further analysis."</span></span>
<span id="cb1-605"><a href="#cb1-605" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-606"><a href="#cb1-606" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-607"><a href="#cb1-607" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-608"><a href="#cb1-608" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh RNase enzyme. Ensure complete RNA degradation by checking for absence of RNA bands on a gel."</span>,</span>
<span id="cb1-609"><a href="#cb1-609" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control in the thermal cycler."</span>,</span>
<span id="cb1-610"><a href="#cb1-610" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Vortex thoroughly until no clumps remain."</span>,</span>
<span id="cb1-611"><a href="#cb1-611" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid contamination by using clean tubes and pipette tips."</span>,</span>
<span id="cb1-612"><a href="#cb1-612" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of beads for the sample volume. Mix gently to avoid foaming."</span>,</span>
<span id="cb1-613"><a href="#cb1-613" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-614"><a href="#cb1-614" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare fresh ethanol and use high-quality, nuclease-free water."</span>,</span>
<span id="cb1-615"><a href="#cb1-615" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure the magnet is strong enough to hold the beads. Avoid disturbing the pellet when removing supernatant."</span>,</span>
<span id="cb1-616"><a href="#cb1-616" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 70% ethanol and pipette gently to avoid disturbing the beads."</span>,</span>
<span id="cb1-617"><a href="#cb1-617" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat washing if the supernatant is still cloudy."</span>,</span>
<span id="cb1-618"><a href="#cb1-618" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid over-drying the beads, which can make elution difficult."</span>,</span>
<span id="cb1-619"><a href="#cb1-619" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of nuclease-free water. Ensure complete resuspension."</span>,</span>
<span id="cb1-620"><a href="#cb1-620" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-621"><a href="#cb1-621" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-622"><a href="#cb1-622" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span>,</span>
<span id="cb1-623"><a href="#cb1-623" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare the PCR master mix carefully. Avoid pipetting errors and ensure proper primer concentrations."</span>,</span>
<span id="cb1-624"><a href="#cb1-624" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify correct thermal cycler settings. Check for amplification using gel electrophoresis."</span>,</span>
<span id="cb1-625"><a href="#cb1-625" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Vortex thoroughly until no clumps remain."</span>,</span>
<span id="cb1-626"><a href="#cb1-626" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid contamination by using clean tubes and pipette tips."</span>,</span>
<span id="cb1-627"><a href="#cb1-627" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of beads for the sample volume. Mix gently to avoid foaming."</span>,</span>
<span id="cb1-628"><a href="#cb1-628" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-629"><a href="#cb1-629" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare fresh ethanol and use high-quality, nuclease-free water."</span>,</span>
<span id="cb1-630"><a href="#cb1-630" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure the magnet is strong enough to hold the beads. Avoid disturbing the pellet when removing supernatant."</span>,</span>
<span id="cb1-631"><a href="#cb1-631" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 70% ethanol and pipette gently to avoid disturbing the beads."</span>,</span>
<span id="cb1-632"><a href="#cb1-632" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete removal of ethanol."</span>,</span>
<span id="cb1-633"><a href="#cb1-633" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat washing if the supernatant is still cloudy."</span>,</span>
<span id="cb1-634"><a href="#cb1-634" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid over-drying the beads, which can make elution difficult."</span>,</span>
<span id="cb1-635"><a href="#cb1-635" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of nuclease-free water. Ensure complete resuspension."</span>,</span>
<span id="cb1-636"><a href="#cb1-636" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-637"><a href="#cb1-637" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-638"><a href="#cb1-638" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span>,</span>
<span id="cb1-639"><a href="#cb1-639" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use appropriate methods for analysis, such as Bioanalyzer or gel electrophoresis."</span></span>
<span id="cb1-640"><a href="#cb1-640" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-641"><a href="#cb1-641" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-642"><a href="#cb1-642" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-643"><a href="#cb1-643" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(RNA_deg, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-644"><a href="#cb1-644" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-645"><a href="#cb1-645" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-646"><a href="#cb1-646" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-647"><a href="#cb1-647" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-648"><a href="#cb1-648" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-649"><a href="#cb1-649" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-650"><a href="#cb1-650" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-651"><a href="#cb1-651" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-652"><a href="#cb1-652" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: MinION Sequencing</span></span>
<span id="cb1-653"><a href="#cb1-653" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-654"><a href="#cb1-654" aria-hidden="true" tabindex="-1"></a>I am basically doing gDNA and cDNA sequencing. There are some subtle differences in the protocols. Please, refer to the ONT guidelines for details. I am attaching the pdf here though. The reagents highlighted yellow are different in the 2 protocols. Check them out below.</span>
<span id="cb1-655"><a href="#cb1-655" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-656"><a href="#cb1-656" aria-hidden="true" tabindex="-1"></a>For gDNA sequencing using SQK-LSK109:\</span>
<span id="cb1-657"><a href="#cb1-657" aria-hidden="true" tabindex="-1"></a><span class="kw">&lt;iframe</span> <span class="er">src</span><span class="ot">=</span><span class="st">"../../pdf/SQK-LSK109-XL.pdf"</span> <span class="er">width</span><span class="ot">=</span><span class="st">"100%"</span> <span class="er">height</span><span class="ot">=</span><span class="st">"600px"</span><span class="kw">&gt;&lt;/iframe&gt;</span></span>
<span id="cb1-658"><a href="#cb1-658" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-659"><a href="#cb1-659" aria-hidden="true" tabindex="-1"></a>For polyA tailing:\</span>
<span id="cb1-660"><a href="#cb1-660" aria-hidden="true" tabindex="-1"></a><span class="kw">&lt;iframe</span> <span class="er">src</span><span class="ot">=</span><span class="st">"../../pdf/3-poly-rna-ecoli-pap.pdf"</span> <span class="er">width</span><span class="ot">=</span><span class="st">"100%"</span> <span class="er">height</span><span class="ot">=</span><span class="st">"600px"</span><span class="kw">&gt;&lt;/iframe&gt;</span></span>
<span id="cb1-661"><a href="#cb1-661" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-662"><a href="#cb1-662" aria-hidden="true" tabindex="-1"></a>For direct cDNA sequencing using SQK-DCS109:\</span>
<span id="cb1-663"><a href="#cb1-663" aria-hidden="true" tabindex="-1"></a><span class="kw">&lt;iframe</span> <span class="er">src</span><span class="ot">=</span><span class="st">"../../pdf/cDNA_SQK-DCS109.pdf"</span> <span class="er">width</span><span class="ot">=</span><span class="st">"100%"</span> <span class="er">height</span><span class="ot">=</span><span class="st">"600px"</span><span class="kw">&gt;&lt;/iframe&gt;</span></span>
<span id="cb1-664"><a href="#cb1-664" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-665"><a href="#cb1-665" aria-hidden="true" tabindex="-1"></a>**The cDNA sequencing kit uses the same Ligation sequencing kit (SQK-LSK109) + Priming kit (EXP-FLP002). But the adapter mix could be different here (inherently) to ease addition of the adapters to the ds-cDNA sequences.**</span>
<span id="cb1-666"><a href="#cb1-666" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-667"><a href="#cb1-667" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo = FALSE}</span></span>
<span id="cb1-668"><a href="#cb1-668" aria-hidden="true" tabindex="-1"></a>seq_diff <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-669"><a href="#cb1-669" aria-hidden="true" tabindex="-1"></a>  <span class="at">Reagent =</span> <span class="fu">c</span>(</span>
<span id="cb1-670"><a href="#cb1-670" aria-hidden="true" tabindex="-1"></a>    <span class="st">"NEBNext FFPE Repair Mix (NEB, M6630)"</span>,</span>
<span id="cb1-671"><a href="#cb1-671" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Adapter Mix (AMX) and Quick T4 Ligase"</span>,</span>
<span id="cb1-672"><a href="#cb1-672" aria-hidden="true" tabindex="-1"></a>    <span class="st">"E. coli Poly(A) Polymerase (NEB: M0276L)"</span>,</span>
<span id="cb1-673"><a href="#cb1-673" aria-hidden="true" tabindex="-1"></a>    <span class="st">"50 mM RNase-free EDTA (Invitrogen: AM9260G)"</span>,</span>
<span id="cb1-674"><a href="#cb1-674" aria-hidden="true" tabindex="-1"></a>    <span class="st">"NEB Blunt/TA Ligase Master Mix (M0367)"</span>,</span>
<span id="cb1-675"><a href="#cb1-675" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10 mM dNTP solution (e.g. NEB N0447)"</span>,</span>
<span id="cb1-676"><a href="#cb1-676" aria-hidden="true" tabindex="-1"></a>    <span class="st">"LongAmp Taq 2X Master Mix (e.g. NEB M0287)"</span>,</span>
<span id="cb1-677"><a href="#cb1-677" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Maxima H Minus Reverse Transcriptase (200 U/μl) with 5x RT Buffer (ThermoFisher, cat # EP0751)"</span>,</span>
<span id="cb1-678"><a href="#cb1-678" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RNaseOUTTM, 40 U/μl (Life Technologies, cat # 10777019)"</span>,</span>
<span id="cb1-679"><a href="#cb1-679" aria-hidden="true" tabindex="-1"></a>    <span class="st">"RiboShredder (Epicentre, cat # RS12500), or RNase Cocktail Enzyme Mix (ThermoFisher, cat # AM2286)"</span>,</span>
<span id="cb1-680"><a href="#cb1-680" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2.5 μl VNP/DTT"</span>,</span>
<span id="cb1-681"><a href="#cb1-681" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2 μl Strand-Switching Primer (SSP)"</span>,</span>
<span id="cb1-682"><a href="#cb1-682" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2 μl PR2 Primer, 10 μM"</span></span>
<span id="cb1-683"><a href="#cb1-683" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-684"><a href="#cb1-684" aria-hidden="true" tabindex="-1"></a>  <span class="at">Function =</span> <span class="fu">c</span>(<span class="st">"Repair the end of the dsDNA. Not needed for cDNA"</span>,</span>
<span id="cb1-685"><a href="#cb1-685" aria-hidden="true" tabindex="-1"></a>               <span class="st">"T4 DNA ligase lygate the adapter mix"</span>,</span>
<span id="cb1-686"><a href="#cb1-686" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Adds poly-A tail"</span>,</span>
<span id="cb1-687"><a href="#cb1-687" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Stops poly-A tailing after the required time period"</span>,</span>
<span id="cb1-688"><a href="#cb1-688" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Ligate the dA tail"</span>,</span>
<span id="cb1-689"><a href="#cb1-689" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Nucleotides to be added in the cDNA strands"</span>,</span>
<span id="cb1-690"><a href="#cb1-690" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Polymerase for 2nd strand synthesis"</span>,</span>
<span id="cb1-691"><a href="#cb1-691" aria-hidden="true" tabindex="-1"></a>               <span class="st">"For 1st strand synthesis"</span>,</span>
<span id="cb1-692"><a href="#cb1-692" aria-hidden="true" tabindex="-1"></a>               <span class="st">"To protect RNA template from any RNase"</span>,</span>
<span id="cb1-693"><a href="#cb1-693" aria-hidden="true" tabindex="-1"></a>               <span class="st">"To degrade the RNA template before the 2nd strand synthesis"</span>,</span>
<span id="cb1-694"><a href="#cb1-694" aria-hidden="true" tabindex="-1"></a>               <span class="st">"As a primer for the 1st strand synthesis, complementary to the poly-A tail in the RNA"</span>,</span>
<span id="cb1-695"><a href="#cb1-695" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Required for the Maxima H RT (modified MMLV) for full length+ 1st strand synthesis"</span>,</span>
<span id="cb1-696"><a href="#cb1-696" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Primer to initiate the 2nd strand synthesis"</span></span>
<span id="cb1-697"><a href="#cb1-697" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-698"><a href="#cb1-698" aria-hidden="true" tabindex="-1"></a>  <span class="at">Protocol =</span> <span class="fu">c</span>(<span class="st">"gDNA seq"</span>,</span>
<span id="cb1-699"><a href="#cb1-699" aria-hidden="true" tabindex="-1"></a>               <span class="st">"gDNA seq"</span>,</span>
<span id="cb1-700"><a href="#cb1-700" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-701"><a href="#cb1-701" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-702"><a href="#cb1-702" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-703"><a href="#cb1-703" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-704"><a href="#cb1-704" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-705"><a href="#cb1-705" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-706"><a href="#cb1-706" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-707"><a href="#cb1-707" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-708"><a href="#cb1-708" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-709"><a href="#cb1-709" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span>,</span>
<span id="cb1-710"><a href="#cb1-710" aria-hidden="true" tabindex="-1"></a>               <span class="st">"cDNA seq"</span></span>
<span id="cb1-711"><a href="#cb1-711" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-712"><a href="#cb1-712" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-713"><a href="#cb1-713" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-714"><a href="#cb1-714" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate HTML table</span></span>
<span id="cb1-715"><a href="#cb1-715" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(seq_diff, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-716"><a href="#cb1-716" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-717"><a href="#cb1-717" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-718"><a href="#cb1-718" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-719"><a href="#cb1-719" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-720"><a href="#cb1-720" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="co">#%&gt;% </span></span>
<span id="cb1-721"><a href="#cb1-721" aria-hidden="true" tabindex="-1"></a>  <span class="co">#add_header_above(c("Step" = 1, "Mechanistic Details" = 1, "Comments for Troubleshooting" = 1))</span></span>
<span id="cb1-722"><a href="#cb1-722" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-723"><a href="#cb1-723" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-724"><a href="#cb1-724" aria-hidden="true" tabindex="-1"></a><span class="fu">### DNA Repair and End-prep</span></span>
<span id="cb1-725"><a href="#cb1-725" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-726"><a href="#cb1-726" aria-hidden="true" tabindex="-1"></a><span class="fu">#### End-prep for cDNA sequencing</span></span>
<span id="cb1-727"><a href="#cb1-727" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-728"><a href="#cb1-728" aria-hidden="true" tabindex="-1"></a>**IMPORTANT** If you have prepared your own cDNA instead of performing reverse transcription using the Direct cDNA Sequencing Kit, please start this step with 70-200 ng cDNA in 20 µl Nuclease-free water.</span>
<span id="cb1-729"><a href="#cb1-729" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-730"><a href="#cb1-730" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-731"><a href="#cb1-731" aria-hidden="true" tabindex="-1"></a>end_prep <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-732"><a href="#cb1-732" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"1. Perform end repair and dA-tailing of the cDNA sample as follows: &lt;br&gt; &nbsp;</span></span>
<span id="cb1-733"><a href="#cb1-733" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;20 µl cDNA sample &lt;br&gt;</span></span>
<span id="cb1-734"><a href="#cb1-734" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;30 µl Nuclease-free water &lt;br&gt;</span></span>
<span id="cb1-735"><a href="#cb1-735" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;7 µl Ultra II End-prep reaction buffer &lt;br&gt;</span></span>
<span id="cb1-736"><a href="#cb1-736" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;3 µl Ultra II End-prep enzyme mix"</span>,</span>
<span id="cb1-737"><a href="#cb1-737" aria-hidden="true" tabindex="-1"></a>  <span class="st">"2. Mix gently by pipetting and spin down."</span>,</span>
<span id="cb1-738"><a href="#cb1-738" aria-hidden="true" tabindex="-1"></a>  <span class="st">"3. Using a thermal cycler, incubate at 20°C for 5 minutes and 65°C for 5 minutes."</span>,</span>
<span id="cb1-739"><a href="#cb1-739" aria-hidden="true" tabindex="-1"></a>  <span class="st">"4. Resuspend the AMPure XP beads by vortexing."</span>,</span>
<span id="cb1-740"><a href="#cb1-740" aria-hidden="true" tabindex="-1"></a>  <span class="st">"5. Transfer the sample to a 1.5 ml DNA LoBind Eppendorf tube."</span>,</span>
<span id="cb1-741"><a href="#cb1-741" aria-hidden="true" tabindex="-1"></a>  <span class="st">"6. Add 60 µl of resuspended AMPure XP beads to the end-prep reaction and mix by pipetting."</span>,</span>
<span id="cb1-742"><a href="#cb1-742" aria-hidden="true" tabindex="-1"></a>  <span class="st">"7. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT."</span>,</span>
<span id="cb1-743"><a href="#cb1-743" aria-hidden="true" tabindex="-1"></a>  <span class="st">"8. Prepare 500 μl of fresh 70% ethanol in Nuclease-free water."</span>,</span>
<span id="cb1-744"><a href="#cb1-744" aria-hidden="true" tabindex="-1"></a>  <span class="st">"9. Spin down the sample and pellet on a magnet. Keep the tube on the magnet, and pipette off the supernatant."</span>,</span>
<span id="cb1-745"><a href="#cb1-745" aria-hidden="true" tabindex="-1"></a>  <span class="st">"10. Keep the tube on the magnet and wash the beads with 200 µl of freshly prepared 70% ethanol without disturbing the pellet. Remove the ethanol using a pipette and discard."</span>,</span>
<span id="cb1-746"><a href="#cb1-746" aria-hidden="true" tabindex="-1"></a>  <span class="st">"11. Repeat the previous step."</span>,</span>
<span id="cb1-747"><a href="#cb1-747" aria-hidden="true" tabindex="-1"></a>  <span class="st">"12. Spin down and place the tube back on the magnet. Pipette off any residual ethanol. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking."</span>,</span>
<span id="cb1-748"><a href="#cb1-748" aria-hidden="true" tabindex="-1"></a>  <span class="st">"13. Remove the tube from the magnetic rack and resuspend pellet in 30 µl Nuclease-free water. Incubate for 2 minutes at RT."</span>,</span>
<span id="cb1-749"><a href="#cb1-749" aria-hidden="true" tabindex="-1"></a>  <span class="st">"14. Pellet the beads on the magnet until the eluate is clear and colourless, for at least 1 minute"</span>,</span>
<span id="cb1-750"><a href="#cb1-750" aria-hidden="true" tabindex="-1"></a>  <span class="st">"15. Remove and retain 30 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube."</span></span>
<span id="cb1-751"><a href="#cb1-751" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-752"><a href="#cb1-752" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-753"><a href="#cb1-753" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-754"><a href="#cb1-754" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repairs any damaged DNA ends and adds an adenosine (A) nucleotide to the 3' ends of the cDNA, preparing it for ligation to sequencing adapters."</span>,</span>
<span id="cb1-755"><a href="#cb1-755" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures proper mixing of reagents."</span>,</span>
<span id="cb1-756"><a href="#cb1-756" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides optimal temperature for enzymatic activity: 20°C for end repair and 65°C for A-tailing."</span>,</span>
<span id="cb1-757"><a href="#cb1-757" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures consistent bead suspension for efficient binding of cDNA."</span>,</span>
<span id="cb1-758"><a href="#cb1-758" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Transfers the sample to a tube compatible with magnetic separation."</span>,</span>
<span id="cb1-759"><a href="#cb1-759" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads bind to the cDNA, allowing for purification and removal of excess reagents."</span>,</span>
<span id="cb1-760"><a href="#cb1-760" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mixing promotes efficient binding of cDNA to the beads."</span>,</span>
<span id="cb1-761"><a href="#cb1-761" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepares ethanol for washing the beads."</span>,</span>
<span id="cb1-762"><a href="#cb1-762" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic beads are attracted to the magnet, separating them from the supernatant."</span>,</span>
<span id="cb1-763"><a href="#cb1-763" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol wash removes impurities and salts from the beads."</span>,</span>
<span id="cb1-764"><a href="#cb1-764" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-765"><a href="#cb1-765" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any remaining ethanol."</span>,</span>
<span id="cb1-766"><a href="#cb1-766" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elution buffer rehydrates the DNA and allows it to detach from the beads."</span>,</span>
<span id="cb1-767"><a href="#cb1-767" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified DNA in the supernatant."</span>,</span>
<span id="cb1-768"><a href="#cb1-768" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified, end-repaired, and A-tailed cDNA for further processing."</span></span>
<span id="cb1-769"><a href="#cb1-769" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-770"><a href="#cb1-770" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-771"><a href="#cb1-771" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-772"><a href="#cb1-772" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper enzyme activity and storage. Use high-quality reagents and follow the manufacturer's instructions."</span>,</span>
<span id="cb1-773"><a href="#cb1-773" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid vigorous mixing, which can damage DNA."</span>,</span>
<span id="cb1-774"><a href="#cb1-774" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate temperature control and timing in the thermal cycler."</span>,</span>
<span id="cb1-775"><a href="#cb1-775" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Vortex thoroughly until no clumps remain."</span>,</span>
<span id="cb1-776"><a href="#cb1-776" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a clean tube to avoid contamination."</span>,</span>
<span id="cb1-777"><a href="#cb1-777" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of beads. Mix gently to avoid foaming."</span>,</span>
<span id="cb1-778"><a href="#cb1-778" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-779"><a href="#cb1-779" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare fresh ethanol and use high-quality nuclease-free water."</span>,</span>
<span id="cb1-780"><a href="#cb1-780" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and avoid disturbing the pellet when removing supernatant."</span>,</span>
<span id="cb1-781"><a href="#cb1-781" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 70% ethanol. Pipette gently to avoid disturbing the beads."</span>,</span>
<span id="cb1-782"><a href="#cb1-782" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat the wash if the supernatant is cloudy."</span>,</span>
<span id="cb1-783"><a href="#cb1-783" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid over-drying the beads, which can hinder elution."</span>,</span>
<span id="cb1-784"><a href="#cb1-784" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of nuclease-free water. Ensure complete resuspension."</span>,</span>
<span id="cb1-785"><a href="#cb1-785" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-786"><a href="#cb1-786" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span></span>
<span id="cb1-787"><a href="#cb1-787" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-788"><a href="#cb1-788" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-789"><a href="#cb1-789" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-790"><a href="#cb1-790" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(end_prep, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-791"><a href="#cb1-791" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-792"><a href="#cb1-792" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-793"><a href="#cb1-793" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-794"><a href="#cb1-794" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-795"><a href="#cb1-795" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-796"><a href="#cb1-796" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-797"><a href="#cb1-797" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-798"><a href="#cb1-798" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-799"><a href="#cb1-799" aria-hidden="true" tabindex="-1"></a>Take forward 30 µl of end-prepped cDNA into adapter ligation.</span>
<span id="cb1-800"><a href="#cb1-800" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-801"><a href="#cb1-801" aria-hidden="true" tabindex="-1"></a><span class="fu">### Adapter Ligation for cDNA</span></span>
<span id="cb1-802"><a href="#cb1-802" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-803"><a href="#cb1-803" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-804"><a href="#cb1-804" aria-hidden="true" tabindex="-1"></a>ad_lig_cDNA <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-805"><a href="#cb1-805" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"1. Check the contents of each tube are clear of any precipitate and are thoroughly mixed before setting up the reaction. &lt;br&gt;</span></span>
<span id="cb1-806"><a href="#cb1-806" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Mix the contents of each tube by flicking.&lt;br&gt;</span></span>
<span id="cb1-807"><a href="#cb1-807" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Check that there is no precipitate present (DTT in the Blunt/TA Master Mix can sometimes form a precipitate) &lt;br&gt;</span></span>
<span id="cb1-808"><a href="#cb1-808" aria-hidden="true" tabindex="-1"></a><span class="st">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Spin down briefly before accurately pipetting the contents in the reaction."</span>,</span>
<span id="cb1-809"><a href="#cb1-809" aria-hidden="true" tabindex="-1"></a><span class="st">"2. Taking the end-prepped DNA, perform adapter ligation as follows, mixing by flicking the tube between each</span></span>
<span id="cb1-810"><a href="#cb1-810" aria-hidden="true" tabindex="-1"></a><span class="st">sequential addition. &lt;br&gt;</span></span>
<span id="cb1-811"><a href="#cb1-811" aria-hidden="true" tabindex="-1"></a><span class="st">30 µl End-prepped DNA &lt;br&gt;</span></span>
<span id="cb1-812"><a href="#cb1-812" aria-hidden="true" tabindex="-1"></a><span class="st">5 µl Adapter Mix &lt;br&gt;</span></span>
<span id="cb1-813"><a href="#cb1-813" aria-hidden="true" tabindex="-1"></a><span class="st">50 µl Blunt/TA Ligation Master Mix &lt;br&gt;</span></span>
<span id="cb1-814"><a href="#cb1-814" aria-hidden="true" tabindex="-1"></a><span class="st">15 µl Nuclease-free water"</span>,</span>
<span id="cb1-815"><a href="#cb1-815" aria-hidden="true" tabindex="-1"></a><span class="st">"3. Mix gently by flicking the tube, and spin down."</span>,</span>
<span id="cb1-816"><a href="#cb1-816" aria-hidden="true" tabindex="-1"></a><span class="st">"4. Incubate the reaction for 10 minutes at RT."</span>),</span>
<span id="cb1-817"><a href="#cb1-817" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-818"><a href="#cb1-818" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures that all reagents are properly mixed and no precipitates are present, which could interfere with the ligation reaction."</span>,</span>
<span id="cb1-819"><a href="#cb1-819" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the end-prepped cDNA with adapters and the ligation master mix. The adapters contain sequences necessary for sequencing and bridge amplification."</span>,</span>
<span id="cb1-820"><a href="#cb1-820" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures all components are well mixed and collects any condensation on the tube walls."</span>,</span>
<span id="cb1-821"><a href="#cb1-821" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides optimal temperature for the ligase enzyme to join the adapters to the cDNA ends."</span></span>
<span id="cb1-822"><a href="#cb1-822" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-823"><a href="#cb1-823" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-824"><a href="#cb1-824" aria-hidden="true" tabindex="-1"></a>    <span class="st">"If precipitate is observed, warm the tube briefly at 37°C and vortex gently until the precipitate dissolves."</span>,</span>
<span id="cb1-825"><a href="#cb1-825" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh, high-quality reagents. Ensure accurate pipetting and avoid contamination."</span>,</span>
<span id="cb1-826"><a href="#cb1-826" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid vigorous mixing, which can damage DNA."</span>,</span>
<span id="cb1-827"><a href="#cb1-827" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate temperature control. Avoid extending the incubation time, as this could lead to non-specific ligation."</span></span>
<span id="cb1-828"><a href="#cb1-828" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-829"><a href="#cb1-829" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-830"><a href="#cb1-830" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-831"><a href="#cb1-831" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(ad_lig_cDNA, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-832"><a href="#cb1-832" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-833"><a href="#cb1-833" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-834"><a href="#cb1-834" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-835"><a href="#cb1-835" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-836"><a href="#cb1-836" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-837"><a href="#cb1-837" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-838"><a href="#cb1-838" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-839"><a href="#cb1-839" aria-hidden="true" tabindex="-1"></a>**Output: Adapted and tethered DNA library.**</span>
<span id="cb1-840"><a href="#cb1-840" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-841"><a href="#cb1-841" aria-hidden="true" tabindex="-1"></a><span class="fu">### AMPure XP bead binding</span></span>
<span id="cb1-842"><a href="#cb1-842" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-843"><a href="#cb1-843" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-844"><a href="#cb1-844" aria-hidden="true" tabindex="-1"></a>AMPure <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-845"><a href="#cb1-845" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"1. Resuspend the AMPure XP beads by vortexing."</span>,</span>
<span id="cb1-846"><a href="#cb1-846" aria-hidden="true" tabindex="-1"></a>            <span class="st">"2. Add 40 µl of resuspended AMPure XP beads to the adapter ligation reaction from the previous step and mix by pipetting."</span>,</span>
<span id="cb1-847"><a href="#cb1-847" aria-hidden="true" tabindex="-1"></a>            <span class="st">"3. Incubate on a Hula mixer (rotator mixer) for 5 minutes at RT."</span>,</span>
<span id="cb1-848"><a href="#cb1-848" aria-hidden="true" tabindex="-1"></a>            <span class="st">"4. Place on a magnetic rack, allow beads to pellet and pipette off supernatant."</span>,</span>
<span id="cb1-849"><a href="#cb1-849" aria-hidden="true" tabindex="-1"></a>            <span class="st">"5. Add 200 μl of Wash Buffer (WSB) to the beads. Resuspend the beads by pipetting up and down. Return the tube to the magnetic rack, allow beads to pellet and pipette off the supernatant."</span>,</span>
<span id="cb1-850"><a href="#cb1-850" aria-hidden="true" tabindex="-1"></a>            <span class="st">"6. Repeat the previous step."</span>,</span>
<span id="cb1-851"><a href="#cb1-851" aria-hidden="true" tabindex="-1"></a>            <span class="st">"7. Spin down and place the tube back on the magnet. Pipette off any residual supernatant. Allow to dry for ~30 seconds, but do not dry the pellet to the point of cracking."</span>,</span>
<span id="cb1-852"><a href="#cb1-852" aria-hidden="true" tabindex="-1"></a>            <span class="st">"8. Remove the tube from the magnetic rack and resuspend pellet in 13 µl of Elution Buffer (EB)."</span>,</span>
<span id="cb1-853"><a href="#cb1-853" aria-hidden="true" tabindex="-1"></a>            <span class="st">"9. Incubate on a Hula mixer (rotator mixer) for 10 minutes at RT."</span>,</span>
<span id="cb1-854"><a href="#cb1-854" aria-hidden="true" tabindex="-1"></a>            <span class="st">"10. Pellet the beads on a magnet until the eluate is clear and colourless."</span>,</span>
<span id="cb1-855"><a href="#cb1-855" aria-hidden="true" tabindex="-1"></a>            <span class="st">"11. Remove and retain 13 µl of eluate into a clean 1.5 ml Eppendorf DNA LoBind tube."</span>,</span>
<span id="cb1-856"><a href="#cb1-856" aria-hidden="true" tabindex="-1"></a>            <span class="st">"12. Quantify 1 µl of eluted cDNA using a Qubit fluorometer - recovery aim ~60 ng."</span></span>
<span id="cb1-857"><a href="#cb1-857" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-858"><a href="#cb1-858" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-859"><a href="#cb1-859" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures consistent bead suspension for efficient binding of DNA."</span>,</span>
<span id="cb1-860"><a href="#cb1-860" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads selectively bind to DNA fragments larger than a certain size, allowing for size selection and purification."</span>,</span>
<span id="cb1-861"><a href="#cb1-861" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mixing promotes efficient binding of DNA to the beads."</span>,</span>
<span id="cb1-862"><a href="#cb1-862" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, separating them from the supernatant containing unwanted fragments and reagents."</span>,</span>
<span id="cb1-863"><a href="#cb1-863" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Wash buffer removes impurities and salts from the beads."</span>,</span>
<span id="cb1-864"><a href="#cb1-864" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-865"><a href="#cb1-865" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any remaining wash buffer, which can interfere with downstream applications."</span>,</span>
<span id="cb1-866"><a href="#cb1-866" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elution buffer rehydrates the DNA and allows it to detach from the beads."</span>,</span>
<span id="cb1-867"><a href="#cb1-867" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Incubation allows the DNA to fully rehydrate and elute from the beads."</span>,</span>
<span id="cb1-868"><a href="#cb1-868" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified DNA in the supernatant."</span>,</span>
<span id="cb1-869"><a href="#cb1-869" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified DNA for downstream applications."</span>,</span>
<span id="cb1-870"><a href="#cb1-870" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Measures the concentration of the purified cDNA to assess yield and quality."</span></span>
<span id="cb1-871"><a href="#cb1-871" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-872"><a href="#cb1-872" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-873"><a href="#cb1-873" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Vortex thoroughly until no clumps remain."</span>,</span>
<span id="cb1-874"><a href="#cb1-874" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of beads for the sample volume. Mix gently to avoid foaming."</span>,</span>
<span id="cb1-875"><a href="#cb1-875" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-876"><a href="#cb1-876" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and avoid disturbing the pellet when removing supernatant."</span>,</span>
<span id="cb1-877"><a href="#cb1-877" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh wash buffer and pipette gently to avoid disturbing the beads."</span>,</span>
<span id="cb1-878"><a href="#cb1-878" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat washing if the supernatant is still cloudy."</span>,</span>
<span id="cb1-879"><a href="#cb1-879" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid over-drying the beads, which can make elution difficult."</span>,</span>
<span id="cb1-880"><a href="#cb1-880" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of elution buffer. Ensure complete resuspension."</span>,</span>
<span id="cb1-881"><a href="#cb1-881" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper rotation on the Hula mixer."</span>,</span>
<span id="cb1-882"><a href="#cb1-882" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-883"><a href="#cb1-883" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span>,</span>
<span id="cb1-884"><a href="#cb1-884" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate calibration of the Qubit fluorometer. If yield is low, troubleshoot previous steps."</span></span>
<span id="cb1-885"><a href="#cb1-885" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-886"><a href="#cb1-886" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-887"><a href="#cb1-887" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-888"><a href="#cb1-888" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(AMPure, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-889"><a href="#cb1-889" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-890"><a href="#cb1-890" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-891"><a href="#cb1-891" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-892"><a href="#cb1-892" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-893"><a href="#cb1-893" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-894"><a href="#cb1-894" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-895"><a href="#cb1-895" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-896"><a href="#cb1-896" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-897"><a href="#cb1-897" aria-hidden="true" tabindex="-1"></a>**The prepared library is used for loading onto the flow cell. Store the library on ice until ready to load.**</span>
<span id="cb1-898"><a href="#cb1-898" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-899"><a href="#cb1-899" aria-hidden="true" tabindex="-1"></a><span class="fu">### Priming and Loading on SpotON flowCell</span></span>
<span id="cb1-900"><a href="#cb1-900" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-901"><a href="#cb1-901" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-902"><a href="#cb1-902" aria-hidden="true" tabindex="-1"></a>loading <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-903"><a href="#cb1-903" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"1. Thaw the Sequencing Buffer (SQB), Loading Beads (LB), Flush Tether (FLT) and one tube of Flush Buffer (FB) at RT before mixing the reagents by vortexing, and spin down the SQB, FB and FLT at RT."</span>,</span>
<span id="cb1-904"><a href="#cb1-904" aria-hidden="true" tabindex="-1"></a>             <span class="st">"2. Prepare the flow cell priming mix: Add 30 µl of thawed and mixed Flush Tether (FLT) directly to the tube of thawed and mixed Flush Buffer (FB), and mix by vortexing."</span>,</span>
<span id="cb1-905"><a href="#cb1-905" aria-hidden="true" tabindex="-1"></a>             <span class="st">"3. Open the MinION Mk1C lid and slide the flow cell under the clip."</span>,</span>
<span id="cb1-906"><a href="#cb1-906" aria-hidden="true" tabindex="-1"></a>             <span class="st">"4. Slide the priming port cover clockwise to open the priming port."</span>,</span>
<span id="cb1-907"><a href="#cb1-907" aria-hidden="true" tabindex="-1"></a>             <span class="st">"5. IMPORTANT: Take care when drawing back buffer from the flow cell. Do not remove more than 20-30 µl, and make sure that the array of pores are covered by buffer at all times. Introducing air bubbles into the array can irreversibly damage pores. &lt;br&gt;</span></span>
<span id="cb1-908"><a href="#cb1-908" aria-hidden="true" tabindex="-1"></a><span class="st">               After opening the priming port, check for a small air bubble under the cover. Draw back a small volume to remove any bubbles: &lt;br&gt;</span></span>
<span id="cb1-909"><a href="#cb1-909" aria-hidden="true" tabindex="-1"></a><span class="st">               Set a P1000 pipette to 200 µl &lt;br&gt;</span></span>
<span id="cb1-910"><a href="#cb1-910" aria-hidden="true" tabindex="-1"></a><span class="st">               Insert the tip into the priming port &lt;br&gt;</span></span>
<span id="cb1-911"><a href="#cb1-911" aria-hidden="true" tabindex="-1"></a><span class="st">               Turn the wheel until the dial shows 220-230 µl, to draw back 20-30 µl, or until you can see a small volume of buffer entering the pipette tip &lt;br&gt;</span></span>
<span id="cb1-912"><a href="#cb1-912" aria-hidden="true" tabindex="-1"></a><span class="st">               &lt;br&gt;</span></span>
<span id="cb1-913"><a href="#cb1-913" aria-hidden="true" tabindex="-1"></a><span class="st">               Note: Visually check that there is continuous buffer from the priming port across the sensor array."</span>,</span>
<span id="cb1-914"><a href="#cb1-914" aria-hidden="true" tabindex="-1"></a>             <span class="st">"6. Load 800 µl of the priming mix into the flow cell via the priming port, avoiding the introduction of air bubbles. Wait for five minutes. During this time, prepare the library for loading by following the steps below."</span>,</span>
<span id="cb1-915"><a href="#cb1-915" aria-hidden="true" tabindex="-1"></a>             <span class="st">"7. Thoroughly mix the contents of the Loading Beads (LB) by pipetting."</span>,</span>
<span id="cb1-916"><a href="#cb1-916" aria-hidden="true" tabindex="-1"></a>             <span class="st">"8. IMPORTANT: The Loading Beads (LB) tube contains a suspension of beads. These beads settle very quickly. It is vital that they are mixed immediately before use &lt;br&gt;</span></span>
<span id="cb1-917"><a href="#cb1-917" aria-hidden="true" tabindex="-1"></a><span class="st">               In a new tube, prepare the library for loading as follows: &lt;br&gt;</span></span>
<span id="cb1-918"><a href="#cb1-918" aria-hidden="true" tabindex="-1"></a><span class="st">               &lt;br&gt;</span></span>
<span id="cb1-919"><a href="#cb1-919" aria-hidden="true" tabindex="-1"></a><span class="st">               37.5 µl Sequencing Buffer (SQB) &lt;br&gt;</span></span>
<span id="cb1-920"><a href="#cb1-920" aria-hidden="true" tabindex="-1"></a><span class="st">               25.5 µl Loading Beads (LB), mixed immediately before use &lt;br&gt;</span></span>
<span id="cb1-921"><a href="#cb1-921" aria-hidden="true" tabindex="-1"></a><span class="st">               12 µl DNA library"</span>,</span>
<span id="cb1-922"><a href="#cb1-922" aria-hidden="true" tabindex="-1"></a>             <span class="st">"9. Complete the flow cell priming: &lt;br&gt;</span></span>
<span id="cb1-923"><a href="#cb1-923" aria-hidden="true" tabindex="-1"></a><span class="st">               Gently lift the SpotON sample port cover to make the SpotON sample port accessible. &lt;br&gt;</span></span>
<span id="cb1-924"><a href="#cb1-924" aria-hidden="true" tabindex="-1"></a><span class="st">               Load 200 µl of the priming mix into the flow cell priming port (not the SpotON sample port), avoiding the introduction of air bubbles."</span>,</span>
<span id="cb1-925"><a href="#cb1-925" aria-hidden="true" tabindex="-1"></a>             <span class="st">"10. Mix the prepared library gently by pipetting up and down just prior to loading."</span>,</span>
<span id="cb1-926"><a href="#cb1-926" aria-hidden="true" tabindex="-1"></a>             <span class="st">"11. Add 75 μl of the prepared library to the flow cell via the SpotON sample port in a dropwise fashion. Ensure each drop flows into the port before adding the next."</span>,</span>
<span id="cb1-927"><a href="#cb1-927" aria-hidden="true" tabindex="-1"></a>             <span class="st">"12. Gently replace the SpotON sample port cover, making sure the bung enters the SpotON port and close the priming port."</span>,</span>
<span id="cb1-928"><a href="#cb1-928" aria-hidden="true" tabindex="-1"></a>             <span class="st">"13. IMPORTANT: Install the light shield on your flow cell as soon as the library has been loaded for optimal sequencing output. &lt;br&gt;</span></span>
<span id="cb1-929"><a href="#cb1-929" aria-hidden="true" tabindex="-1"></a><span class="st">               &lt;br&gt;</span></span>
<span id="cb1-930"><a href="#cb1-930" aria-hidden="true" tabindex="-1"></a><span class="st">               Place the light shield onto the flow cell, as follows: &lt;br&gt;</span></span>
<span id="cb1-931"><a href="#cb1-931" aria-hidden="true" tabindex="-1"></a><span class="st">               Carefully place the leading edge of the light shield against the clip. &lt;br&gt;</span></span>
<span id="cb1-932"><a href="#cb1-932" aria-hidden="true" tabindex="-1"></a><span class="st">               Note: Do not force the light shield underneath the clip. &lt;br&gt;</span></span>
<span id="cb1-933"><a href="#cb1-933" aria-hidden="true" tabindex="-1"></a><span class="st">               Gently lower the light shield onto the flow cell. The light shield should sit around the SpotON cover, covering the entire top section of the flow cell."</span></span>
<span id="cb1-934"><a href="#cb1-934" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-935"><a href="#cb1-935" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-936"><a href="#cb1-936" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-937"><a href="#cb1-937" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Thawing and mixing reagents ensures they are at the correct temperature and consistency for optimal performance."</span>,</span>
<span id="cb1-938"><a href="#cb1-938" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The priming mix prepares the flow cell for loading by coating the channels and pores with reagents that facilitate DNA binding and sequencing."</span>,</span>
<span id="cb1-939"><a href="#cb1-939" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides access to the flow cell ports for loading reagents."</span>,</span>
<span id="cb1-940"><a href="#cb1-940" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Opens the port for loading the priming mix."</span>,</span>
<span id="cb1-941"><a href="#cb1-941" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes any air bubbles that may have formed under the priming port cover, which could interfere with flow cell function."</span>,</span>
<span id="cb1-942"><a href="#cb1-942" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The priming mix helps to wet the flow cell channels and pores, ensuring proper fluid flow and DNA capture."</span>,</span>
<span id="cb1-943"><a href="#cb1-943" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures uniform distribution of beads for optimal library loading."</span>,</span>
<span id="cb1-944"><a href="#cb1-944" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the library with sequencing buffer and loading beads. The loading beads help to concentrate the library and facilitate its entry into the flow cell pores."</span>,</span>
<span id="cb1-945"><a href="#cb1-945" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Completes the priming process, ensuring all channels and pores are properly coated and ready for sequencing."</span>,</span>
<span id="cb1-946"><a href="#cb1-946" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures the library is well mixed and evenly distributed before loading."</span>,</span>
<span id="cb1-947"><a href="#cb1-947" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The SpotON port allows for gentle and controlled loading of the library onto the flow cell."</span>,</span>
<span id="cb1-948"><a href="#cb1-948" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prevents contamination and leakage of reagents."</span>,</span>
<span id="cb1-949"><a href="#cb1-949" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The light shield protects the flow cell from ambient light, which can interfere with sequencing."</span></span>
<span id="cb1-950"><a href="#cb1-950" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-951"><a href="#cb1-951" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-952"><a href="#cb1-952" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-953"><a href="#cb1-953" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure all reagents are completely thawed and mixed before use. Check for any signs of contamination or degradation."</span>,</span>
<span id="cb1-954"><a href="#cb1-954" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid introducing air bubbles into the priming mix. Mix gently to avoid foaming."</span>,</span>
<span id="cb1-955"><a href="#cb1-955" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Handle the flow cell with care to avoid damage. Make sure it is properly seated under the clip."</span>,</span>
<span id="cb1-956"><a href="#cb1-956" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Open the priming port slowly and carefully to prevent damage."</span>,</span>
<span id="cb1-957"><a href="#cb1-957" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use caution when drawing back buffer. Avoid removing too much or introducing air bubbles."</span>,</span>
<span id="cb1-958"><a href="#cb1-958" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid introducing air bubbles when loading the priming mix. Ensure the full volume is loaded."</span>,</span>
<span id="cb1-959"><a href="#cb1-959" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix gently to avoid damaging the beads."</span>,</span>
<span id="cb1-960"><a href="#cb1-960" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix the loading beads immediately before use to prevent settling. Avoid over-mixing, which can damage DNA."</span>,</span>
<span id="cb1-961"><a href="#cb1-961" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid introducing air bubbles when loading the priming mix. Ensure the full volume is loaded."</span>,</span>
<span id="cb1-962"><a href="#cb1-962" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix gently to avoid damaging the DNA library."</span>,</span>
<span id="cb1-963"><a href="#cb1-963" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Load the library slowly and carefully to avoid air bubbles or spills. Ensure complete loading."</span>,</span>
<span id="cb1-964"><a href="#cb1-964" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Close the ports securely to prevent leakage."</span>,</span>
<span id="cb1-965"><a href="#cb1-965" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Install the light shield gently to avoid damaging the flow cell. Make sure it is properly seated."</span></span>
<span id="cb1-966"><a href="#cb1-966" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-967"><a href="#cb1-967" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-968"><a href="#cb1-968" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-969"><a href="#cb1-969" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(loading, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-970"><a href="#cb1-970" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-971"><a href="#cb1-971" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-972"><a href="#cb1-972" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-973"><a href="#cb1-973" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"20%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-974"><a href="#cb1-974" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-975"><a href="#cb1-975" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">setNames</span>(<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>), <span class="fu">c</span>(<span class="st">"Step"</span>, <span class="st">"Mechanistic Details"</span>, <span class="st">"Troubleshooting"</span>)))</span>
<span id="cb1-976"><a href="#cb1-976" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-977"><a href="#cb1-977" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-978"><a href="#cb1-978" aria-hidden="true" tabindex="-1"></a>Close the device lid and set up a sequencing run on MinKNOW.</span>
<span id="cb1-979"><a href="#cb1-979" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-980"><a href="#cb1-980" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: SISPA Protocol  </span></span>
<span id="cb1-981"><a href="#cb1-981" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-982"><a href="#cb1-982" aria-hidden="true" tabindex="-1"></a><span class="fu">### cDNA Synthesis  </span></span>
<span id="cb1-983"><a href="#cb1-983" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-984"><a href="#cb1-984" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-985"><a href="#cb1-985" aria-hidden="true" tabindex="-1"></a>cDNA_synthesis <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-986"><a href="#cb1-986" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-987"><a href="#cb1-987" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Mix in a clean 0.2 ml PCR tube: &lt;br&gt; * 1 \u03BCl dNTP Mix (10 mM) &lt;br&gt; * 1 \u03BCl FR26RV-N (Stock = 50 \u03BCM) &lt;br&gt; * 11 \u03BCl Viral RNA (ideally the Cq should be &lt;23-26, depending on the virus)"</span>,</span>
<span id="cb1-988"><a href="#cb1-988" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Incubate for 5 min at 65 \u00B0C in the thermal cycler. Place on ice for 1 min."</span>,</span>
<span id="cb1-989"><a href="#cb1-989" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Pre-warm 5x SSIV Buffer at 37 \u00B0C for 10 minutes before use, mix by vortexing and spin down."</span>,</span>
<span id="cb1-990"><a href="#cb1-990" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Add per sample: &lt;br&gt; * 4 \u03BCl SSIV buffer 5x &lt;br&gt; * 1 \u03BCl DTT 100 mM &lt;br&gt; * 1 \u03BCl RNase Inhibitor (RNase OUT; 40 U/\u03BCl) &lt;br&gt; * 1 \u03BCl SSIV Reverse Transcriptase (200 U/\u03BCl)"</span>,</span>
<span id="cb1-991"><a href="#cb1-991" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Close lids thoroughly, spin down quickly using bench top centrifuge"</span>,</span>
<span id="cb1-992"><a href="#cb1-992" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Place in the thermal cycler: 23 \u00B0C for 10 min, 50 \u00B0C for 50 min, 80 \u00B0C for 10 min, Hold at 4 \u00B0C"</span></span>
<span id="cb1-993"><a href="#cb1-993" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-994"><a href="#cb1-994" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-995"><a href="#cb1-995" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Detail</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-996"><a href="#cb1-996" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the viral RNA template, dNTPs (building blocks for DNA synthesis), and the first SISPA primer (FR26RV-N). The primer contains a barcode sequence and random hexamers that will anneal to the viral RNA."</span>,</span>
<span id="cb1-997"><a href="#cb1-997" aria-hidden="true" tabindex="-1"></a>    <span class="st">"The high temperature denatures RNA secondary structures, making it more accessible for primer binding. The subsequent cooling on ice helps maintain the denatured state and promotes primer annealing."</span>,</span>
<span id="cb1-998"><a href="#cb1-998" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Pre-warming the buffer ensures optimal enzyme activity during reverse transcription. Mixing and spinning down ensure homogeneity and prevent reagent loss due to condensation."</span>,</span>
<span id="cb1-999"><a href="#cb1-999" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides the necessary buffer conditions, reducing agents (DTT), and RNase inhibitor to protect the RNA template and ensure efficient reverse transcription. The reverse transcriptase enzyme catalyzes the synthesis of cDNA from the RNA template."</span>,</span>
<span id="cb1-1000"><a href="#cb1-1000" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prevents evaporation and contamination during the reaction."</span>,</span>
<span id="cb1-1001"><a href="#cb1-1001" aria-hidden="true" tabindex="-1"></a>    <span class="st">"* 23 \u00B0C: Allows for primer annealing to the RNA template. &lt;br&gt; * 50 \u00B0C: Optimal temperature for reverse transcriptase activity, enabling cDNA synthesis. &lt;br&gt; * 80 \u00B0C: Inactivates the reverse transcriptase enzyme. &lt;br&gt; * 4 \u00B0C: Holds the reaction until further processing."</span></span>
<span id="cb1-1002"><a href="#cb1-1002" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1003"><a href="#cb1-1003" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1004"><a href="#cb1-1004" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1005"><a href="#cb1-1005" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate quantification and integrity of RNA. Use high-quality, RNase-free reagents. The Cq value of the viral RNA should be within the optimal range for efficient reverse transcription."</span>,</span>
<span id="cb1-1006"><a href="#cb1-1006" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control in the thermal cycler. Use a pre-chilled ice bath for rapid cooling."</span>,</span>
<span id="cb1-1007"><a href="#cb1-1007" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a water bath or heat block for accurate temperature control. Mix gently to avoid introducing air bubbles."</span>,</span>
<span id="cb1-1008"><a href="#cb1-1008" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prepare the reaction mix on ice to minimize RNase activity. Use fresh, high-quality reagents. Avoid pipetting errors, especially with small volumes."</span>,</span>
<span id="cb1-1009"><a href="#cb1-1009" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down."</span>,</span>
<span id="cb1-1010"><a href="#cb1-1010" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control and timing in the thermal cycler."</span></span>
<span id="cb1-1011"><a href="#cb1-1011" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1012"><a href="#cb1-1012" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1013"><a href="#cb1-1013" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1014"><a href="#cb1-1014" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(cDNA_synthesis, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1015"><a href="#cb1-1015" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1016"><a href="#cb1-1016" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1017"><a href="#cb1-1017" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1018"><a href="#cb1-1018" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1019"><a href="#cb1-1019" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1020"><a href="#cb1-1020" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Detail"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1021"><a href="#cb1-1021" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1022"><a href="#cb1-1022" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1023"><a href="#cb1-1023" aria-hidden="true" tabindex="-1"></a><span class="fu">### Double-strand synthesis</span></span>
<span id="cb1-1024"><a href="#cb1-1024" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1025"><a href="#cb1-1025" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-1026"><a href="#cb1-1026" aria-hidden="true" tabindex="-1"></a>double_strand <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-1027"><a href="#cb1-1027" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1028"><a href="#cb1-1028" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Add 1 \u03BCl of Klenow polymerase (5 U/\u03BCl) per sample directly to the tube."</span>,</span>
<span id="cb1-1029"><a href="#cb1-1029" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Close lids thoroughly, spin down quickly using bench top centrifuge"</span>,</span>
<span id="cb1-1030"><a href="#cb1-1030" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Place in the thermal cycler: 37 \u00B0C for 60 min, 75 \u00B0C for 10 min"</span></span>
<span id="cb1-1031"><a href="#cb1-1031" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1032"><a href="#cb1-1032" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1033"><a href="#cb1-1033" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Detail</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1034"><a href="#cb1-1034" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Klenow polymerase, a DNA polymerase lacking 5'-&gt;3' exonuclease activity, synthesizes the second cDNA strand using the annealed SISPA primer as a template. This results in double-stranded cDNA with barcode sequences incorporated into both strands."</span>,</span>
<span id="cb1-1035"><a href="#cb1-1035" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prevents evaporation and contamination during the reaction."</span>,</span>
<span id="cb1-1036"><a href="#cb1-1036" aria-hidden="true" tabindex="-1"></a>    <span class="st">"* 37 \u00B0C: Optimal temperature for Klenow polymerase activity, allowing for efficient second strand synthesis. &lt;br&gt; * 75 \u00B0C: Inactivates the Klenow polymerase enzyme."</span></span>
<span id="cb1-1037"><a href="#cb1-1037" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1038"><a href="#cb1-1038" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1039"><a href="#cb1-1039" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1040"><a href="#cb1-1040" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Keep the Klenow polymerase on ice until use to maintain its activity. Avoid pipetting errors, especially with small volumes."</span>,</span>
<span id="cb1-1041"><a href="#cb1-1041" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down."</span>,</span>
<span id="cb1-1042"><a href="#cb1-1042" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control and timing in the thermal cycler."</span></span>
<span id="cb1-1043"><a href="#cb1-1043" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1044"><a href="#cb1-1044" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1045"><a href="#cb1-1045" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1046"><a href="#cb1-1046" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(double_strand, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1047"><a href="#cb1-1047" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1048"><a href="#cb1-1048" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1049"><a href="#cb1-1049" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1050"><a href="#cb1-1050" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1051"><a href="#cb1-1051" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1052"><a href="#cb1-1052" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Detail"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1053"><a href="#cb1-1053" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1054"><a href="#cb1-1054" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1055"><a href="#cb1-1055" aria-hidden="true" tabindex="-1"></a><span class="fu">### cDNA Amplification</span></span>
<span id="cb1-1056"><a href="#cb1-1056" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1057"><a href="#cb1-1057" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-1058"><a href="#cb1-1058" aria-hidden="true" tabindex="-1"></a>cDNA_amplification <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-1059"><a href="#cb1-1059" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1060"><a href="#cb1-1060" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Mix: &lt;br&gt; * 5 \u03BCl dsDNA &lt;br&gt; * 1 \u03BCl FR20RV (Stock 40 \u03BCM, final concentration 0.8 \u03BCM) &lt;br&gt; * 5 \u03BCl 10 X PfU Ultra II reaction Buffer &lt;br&gt; * 1.25 \u03BCl dNTP Mix 10 mM &lt;br&gt; * 1 \u03BCl PfU Ultra II Fusion HS DNA polymerase (keep on cooling rack) &lt;br&gt; * 36.75 \u03BCl Nuclease-free Water"</span>,</span>
<span id="cb1-1061"><a href="#cb1-1061" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Close lids thoroughly, spin down quickly using bench top centrifuge"</span>,</span>
<span id="cb1-1062"><a href="#cb1-1062" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Place in the thermal cycler: &lt;br&gt; * Denaturation 95 \u00B0C 1 min &lt;br&gt; * 45 Cycles &lt;br&gt; * 95 \u00B0C 20 sec &lt;br&gt; * 65 \u00B0C 20 sec &lt;br&gt; * 72 \u00B0C 3 min &lt;br&gt; * Final Extension 72 \u00B0C 3 min &lt;br&gt; * Hold 4 \u00B0C"</span></span>
<span id="cb1-1063"><a href="#cb1-1063" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1064"><a href="#cb1-1064" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1065"><a href="#cb1-1065" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Detail</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1066"><a href="#cb1-1066" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the double-stranded cDNA template, the second SISPA primer (FR20RV), dNTPs, buffer, and a high-fidelity DNA polymerase (PfU Ultra II Fusion HS). The second SISPA primer binds to the barcode sequences incorporated in the previous steps, enabling amplification of the cDNA."</span>,</span>
<span id="cb1-1067"><a href="#cb1-1067" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Prevents evaporation and contamination during the reaction."</span>,</span>
<span id="cb1-1068"><a href="#cb1-1068" aria-hidden="true" tabindex="-1"></a>    <span class="st">"* 95 \u00B0C (Denaturation): Separates the double-stranded cDNA into single strands. &lt;br&gt; * 65 \u00B0C (Annealing): Allows the second SISPA primer to bind to the cDNA. &lt;br&gt; * 72 \u00B0C (Extension): The polymerase enzyme synthesizes new DNA strands, amplifying the cDNA. &lt;br&gt; * 45 Cycles: Repeats the denaturation, annealing, and extension steps to generate a sufficient amount of amplified cDNA. &lt;br&gt; * Final Extension: Ensures complete extension of all cDNA strands. &lt;br&gt; * 4 \u00B0C: Holds the reaction until further processing."</span></span>
<span id="cb1-1069"><a href="#cb1-1069" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1070"><a href="#cb1-1070" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1071"><a href="#cb1-1071" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1072"><a href="#cb1-1072" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure accurate pipetting, especially with small volumes. Keep the polymerase enzyme on ice until use to maintain its activity. Use high-quality, nuclease-free water."</span>,</span>
<span id="cb1-1073"><a href="#cb1-1073" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure tubes are properly sealed. Use a balanced centrifuge for quick and efficient spin-down."</span>,</span>
<span id="cb1-1074"><a href="#cb1-1074" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control and timing in the thermal cycler. Optimize the number of PCR cycles if needed to achieve sufficient amplification without introducing artifacts."</span></span>
<span id="cb1-1075"><a href="#cb1-1075" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1076"><a href="#cb1-1076" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1077"><a href="#cb1-1077" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1078"><a href="#cb1-1078" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(cDNA_amplification, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1079"><a href="#cb1-1079" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1080"><a href="#cb1-1080" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1081"><a href="#cb1-1081" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1082"><a href="#cb1-1082" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1083"><a href="#cb1-1083" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1084"><a href="#cb1-1084" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Detail"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1085"><a href="#cb1-1085" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1086"><a href="#cb1-1086" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1087"><a href="#cb1-1087" aria-hidden="true" tabindex="-1"></a><span class="fu">### Clean-up of cDNA amplicon</span></span>
<span id="cb1-1088"><a href="#cb1-1088" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1089"><a href="#cb1-1089" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-1090"><a href="#cb1-1090" aria-hidden="true" tabindex="-1"></a>cleanup_cDNA <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-1091"><a href="#cb1-1091" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1092"><a href="#cb1-1092" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Transfer 25 \u03BCl of the dsDNA to a 1.5 mL DNA LoBind tubes; the spare dsDNA should be stored at 4 \u00B0C as backup."</span>,</span>
<span id="cb1-1093"><a href="#cb1-1093" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Bring AMPure beads to room temperature. Mix the beads thoroughly before use! Add 1.8 x (sample volume) beads, e.g. for 25 \u03BCl sample add 45 \u03BCl beads;"</span>, </span>
<span id="cb1-1094"><a href="#cb1-1094" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Mix sample thoroughly by pipetting up and down at least 15 x"</span>,</span>
<span id="cb1-1095"><a href="#cb1-1095" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Incubate at room temperature for 7 min on a thermomix (room temperature, 550 rpm)"</span>,</span>
<span id="cb1-1096"><a href="#cb1-1096" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Quickly centrifuge"</span>,</span>
<span id="cb1-1097"><a href="#cb1-1097" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Place tubes in the magnetic stand and leave tubes in the magnetic rack until elution (!)"</span>,</span>
<span id="cb1-1098"><a href="#cb1-1098" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. After 5 minutes remove the supernatant slowly; hold the tube with one hand while opening the cap!"</span>,</span>
<span id="cb1-1099"><a href="#cb1-1099" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Add 1 ml 80 % ethanol and incubate for 30 sec at room temperature"</span>,</span>
<span id="cb1-1100"><a href="#cb1-1100" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Slightly rotate the tubes in the magnetic rack to concentrate the beads"</span>,</span>
<span id="cb1-1101"><a href="#cb1-1101" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Slowly remove the supernatant and discard the ethanol"</span>,</span>
<span id="cb1-1102"><a href="#cb1-1102" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Add another 1 ml 80 % ethanol and incubate for 30 sec at room temperature"</span>,</span>
<span id="cb1-1103"><a href="#cb1-1103" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Slowly remove the supernatant and discard the ethanol"</span>,</span>
<span id="cb1-1104"><a href="#cb1-1104" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Remove all remaining ethanol with a 10 \u03BCl pipet and a clean tip and let air dry for 10 min with the tube opened"</span>,</span>
<span id="cb1-1105"><a href="#cb1-1105" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Elution: Take the tube from the magnetic stand and add 25 \u03BCl nuclease-free water directly onto the beads. Resuspend the beads by pipette mixing. Incubate 2 min at room temperature. Place tube on magnetic stand and incubate for 10 min. Transfer the eluate containing the DNA into a clean reaction tube"</span> </span>
<span id="cb1-1106"><a href="#cb1-1106" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1107"><a href="#cb1-1107" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1108"><a href="#cb1-1108" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1109"><a href="#cb1-1109" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Transfer the amplified cDNA to a suitable tube for purification."</span>,</span>
<span id="cb1-1110"><a href="#cb1-1110" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads selectively bind DNA, allowing for purification and concentration."</span>,</span>
<span id="cb1-1111"><a href="#cb1-1111" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures efficient binding of cDNA to the beads."</span>,</span>
<span id="cb1-1112"><a href="#cb1-1112" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Incubation allows the cDNA to bind to the beads."</span>,</span>
<span id="cb1-1113"><a href="#cb1-1113" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Brief centrifugation ensures all liquid is at the bottom of the tube."</span>,</span>
<span id="cb1-1114"><a href="#cb1-1114" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation facilitates removal of unwanted components."</span>,</span>
<span id="cb1-1115"><a href="#cb1-1115" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Careful removal prevents disturbing the bound cDNA."</span>,</span>
<span id="cb1-1116"><a href="#cb1-1116" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol washes remove contaminants and salts."</span>,</span>
<span id="cb1-1117"><a href="#cb1-1117" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Helps concentrate the beads for efficient washing."</span>,</span>
<span id="cb1-1118"><a href="#cb1-1118" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes the ethanol wash."</span>,</span>
<span id="cb1-1119"><a href="#cb1-1119" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-1120"><a href="#cb1-1120" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes residual ethanol, which can interfere with downstream steps."</span>,</span>
<span id="cb1-1121"><a href="#cb1-1121" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete removal of ethanol before elution. Avoid over-drying the beads."</span>,</span>
<span id="cb1-1122"><a href="#cb1-1122" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elution buffer releases the purified cDNA from the beads."</span></span>
<span id="cb1-1123"><a href="#cb1-1123" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1124"><a href="#cb1-1124" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1125"><a href="#cb1-1125" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1126"><a href="#cb1-1126" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use LoBind tubes to minimize DNA loss."</span>,</span>
<span id="cb1-1127"><a href="#cb1-1127" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio."</span>,</span>
<span id="cb1-1128"><a href="#cb1-1128" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix gently to avoid foaming or DNA damage."</span>,</span>
<span id="cb1-1129"><a href="#cb1-1129" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a thermomixer or rotator for consistent mixing."</span>,</span>
<span id="cb1-1130"><a href="#cb1-1130" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete pelleting of beads before removing supernatant."</span>,</span>
<span id="cb1-1131"><a href="#cb1-1131" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and avoid disturbing the beads."</span>,</span>
<span id="cb1-1132"><a href="#cb1-1132" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Remove supernatant carefully to avoid losing beads."</span>,</span>
<span id="cb1-1133"><a href="#cb1-1133" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 80% ethanol."</span>,</span>
<span id="cb1-1134"><a href="#cb1-1134" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid excessive agitation, which can dislodge the beads."</span>,</span>
<span id="cb1-1135"><a href="#cb1-1135" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Remove supernatant completely without disturbing the beads."</span>,</span>
<span id="cb1-1136"><a href="#cb1-1136" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 80% ethanol."</span>,</span>
<span id="cb1-1137"><a href="#cb1-1137" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Remove supernatant completely without disturbing the beads."</span>,</span>
<span id="cb1-1138"><a href="#cb1-1138" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete removal of ethanol before elution. Avoid over-drying the beads."</span>,</span>
<span id="cb1-1139"><a href="#cb1-1139" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use nuclease-free water for elution. Ensure complete resuspension of the beads."</span></span>
<span id="cb1-1140"><a href="#cb1-1140" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1141"><a href="#cb1-1141" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1142"><a href="#cb1-1142" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1143"><a href="#cb1-1143" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(cleanup_cDNA, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1144"><a href="#cb1-1144" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1145"><a href="#cb1-1145" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1146"><a href="#cb1-1146" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1147"><a href="#cb1-1147" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1148"><a href="#cb1-1148" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1149"><a href="#cb1-1149" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Details"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1150"><a href="#cb1-1150" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1151"><a href="#cb1-1151" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1152"><a href="#cb1-1152" aria-hidden="true" tabindex="-1"></a><span class="fu">### End Prep and Barcode Ligation</span></span>
<span id="cb1-1153"><a href="#cb1-1153" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1154"><a href="#cb1-1154" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-1155"><a href="#cb1-1155" aria-hidden="true" tabindex="-1"></a>end_prep_barcode <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-1156"><a href="#cb1-1156" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1157"><a href="#cb1-1157" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Mix in a clean 1.5 ml DNA LoBind tube: &lt;br&gt; o 10 μl DNA &lt;br&gt; o 1.4 μl Ultra II End Reaction Buffer &lt;br&gt; o 0.6 μl Ultra II End Prep Enzyme Mix (keep in cooling rack or on ice)"</span>,</span>
<span id="cb1-1158"><a href="#cb1-1158" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Incubate using thermomix: 10 min Room Temperature, 5 min 65 °C, Place on ice 1 min."</span>,</span>
<span id="cb1-1159"><a href="#cb1-1159" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Directly add: &lt;br&gt; o 2.5 μl NB(XX) barcode &lt;br&gt; ▪ e.g. Sample 1 = Barcode 06; Sample 2 = Barcode 07; Sample 03 = Barcode 08 &lt;br&gt; ▪ Note down which barcode was used for which sample (!) &lt;br&gt; o 14,5 μl Ultra II Ligation Master Mix &lt;br&gt; o 0,5 μl Ligation Enhancer"</span>,</span>
<span id="cb1-1160"><a href="#cb1-1160" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Incubate using thermomix: 20 min Room Temperature, 10 min 70 °C, Place on ice 1 min"</span>,</span>
<span id="cb1-1161"><a href="#cb1-1161" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. POOL ALL SAMPLES in a single tube!"</span></span>
<span id="cb1-1162"><a href="#cb1-1162" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1163"><a href="#cb1-1163" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1164"><a href="#cb1-1164" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1165"><a href="#cb1-1165" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines the purified cDNA with the end-prep reaction mix, which contains enzymes for repairing DNA ends and adding a 3' adenine overhang."</span>,</span>
<span id="cb1-1166"><a href="#cb1-1166" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides optimal temperatures for the end-repair and A-tailing reactions."</span>,</span>
<span id="cb1-1167"><a href="#cb1-1167" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Adds a unique barcode to each sample, allowing for multiplexing during sequencing. The ligation master mix and enhancer facilitate the joining of the barcode to the cDNA."</span>,</span>
<span id="cb1-1168"><a href="#cb1-1168" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Provides optimal temperatures for the ligation reaction."</span>,</span>
<span id="cb1-1169"><a href="#cb1-1169" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Combines all barcoded samples into a single tube for further processing and sequencing."</span></span>
<span id="cb1-1170"><a href="#cb1-1170" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1171"><a href="#cb1-1171" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1172"><a href="#cb1-1172" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1173"><a href="#cb1-1173" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure proper enzyme activity and storage. Use high-quality reagents."</span>,</span>
<span id="cb1-1174"><a href="#cb1-1174" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control and timing in the thermomixer."</span>,</span>
<span id="cb1-1175"><a href="#cb1-1175" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh, high-quality reagents. Keep the ligation mix and enhancer on ice until use."</span>,</span>
<span id="cb1-1176"><a href="#cb1-1176" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Verify accurate temperature control and timing in the thermomixer."</span>,</span>
<span id="cb1-1177"><a href="#cb1-1177" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure thorough mixing of all samples before pooling."</span></span>
<span id="cb1-1178"><a href="#cb1-1178" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1179"><a href="#cb1-1179" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1180"><a href="#cb1-1180" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1181"><a href="#cb1-1181" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(end_prep_barcode, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1182"><a href="#cb1-1182" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1183"><a href="#cb1-1183" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1184"><a href="#cb1-1184" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1185"><a href="#cb1-1185" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1186"><a href="#cb1-1186" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1187"><a href="#cb1-1187" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Details"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1188"><a href="#cb1-1188" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1189"><a href="#cb1-1189" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1190"><a href="#cb1-1190" aria-hidden="true" tabindex="-1"></a><span class="fu">### Cleanup of barcoded DNA</span></span>
<span id="cb1-1191"><a href="#cb1-1191" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1192"><a href="#cb1-1192" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=FALSE}</span></span>
<span id="cb1-1193"><a href="#cb1-1193" aria-hidden="true" tabindex="-1"></a>cleanup_barcoded <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-1194"><a href="#cb1-1194" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Step</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1195"><a href="#cb1-1195" aria-hidden="true" tabindex="-1"></a>    <span class="st">"1. Add 29.5 μl AMPure XP beads per sample [equal volume (1:1) sample:beads]"</span>,</span>
<span id="cb1-1196"><a href="#cb1-1196" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2. Mix by pipetting and quickly centrifuge"</span>,</span>
<span id="cb1-1197"><a href="#cb1-1197" aria-hidden="true" tabindex="-1"></a>    <span class="st">"3. Incubate for 5 min at room temperature"</span>,</span>
<span id="cb1-1198"><a href="#cb1-1198" aria-hidden="true" tabindex="-1"></a>    <span class="st">"4. Place the sample on the magnetic rack"</span>,</span>
<span id="cb1-1199"><a href="#cb1-1199" aria-hidden="true" tabindex="-1"></a>    <span class="st">"5. Incubate for at least 2 min until the supernatant is completely clear"</span>,</span>
<span id="cb1-1200"><a href="#cb1-1200" aria-hidden="true" tabindex="-1"></a>    <span class="st">"6. Carefully remove the supernatant"</span>,</span>
<span id="cb1-1201"><a href="#cb1-1201" aria-hidden="true" tabindex="-1"></a>    <span class="st">"7. Add 300 μl 70% ethanol to the tube still on the magnetic rack"</span>,</span>
<span id="cb1-1202"><a href="#cb1-1202" aria-hidden="true" tabindex="-1"></a>    <span class="st">"8. Incubate 30 secs"</span>,</span>
<span id="cb1-1203"><a href="#cb1-1203" aria-hidden="true" tabindex="-1"></a>    <span class="st">"9. Carefully remove the supernatant"</span>,</span>
<span id="cb1-1204"><a href="#cb1-1204" aria-hidden="true" tabindex="-1"></a>    <span class="st">"10. Repeat last three steps (2nd ethanol wash)"</span>,</span>
<span id="cb1-1205"><a href="#cb1-1205" aria-hidden="true" tabindex="-1"></a>    <span class="st">"11. Spin down and remove residual ethanol with the tube placed on the magnetic rack using a clean filter tip and a pipette."</span>,</span>
<span id="cb1-1206"><a href="#cb1-1206" aria-hidden="true" tabindex="-1"></a>    <span class="st">"12. Air dry 1 min"</span>,</span>
<span id="cb1-1207"><a href="#cb1-1207" aria-hidden="true" tabindex="-1"></a>    <span class="st">"13. Resuspend in 30 μl EB by pipetting"</span>,</span>
<span id="cb1-1208"><a href="#cb1-1208" aria-hidden="true" tabindex="-1"></a>    <span class="st">"14. Incubate off the magnetic rack for 2 min"</span>,</span>
<span id="cb1-1209"><a href="#cb1-1209" aria-hidden="true" tabindex="-1"></a>    <span class="st">"15. Replace on magnetic rack"</span>,</span>
<span id="cb1-1210"><a href="#cb1-1210" aria-hidden="true" tabindex="-1"></a>    <span class="st">"16. Transfer eluted DNA to a fresh 1.5 mL DNA LoBind tube"</span>,</span>
<span id="cb1-1211"><a href="#cb1-1211" aria-hidden="true" tabindex="-1"></a>    <span class="st">"17. Label tube accordingly with an ethanol-resistant labmarker"</span></span>
<span id="cb1-1212"><a href="#cb1-1212" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1213"><a href="#cb1-1213" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1214"><a href="#cb1-1214" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Mechanistic Details</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1215"><a href="#cb1-1215" aria-hidden="true" tabindex="-1"></a>    <span class="st">"AMPure XP beads selectively bind DNA, allowing for purification and concentration."</span>,</span>
<span id="cb1-1216"><a href="#cb1-1216" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures efficient binding of DNA to the beads."</span>,</span>
<span id="cb1-1217"><a href="#cb1-1217" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Incubation allows the DNA to bind to the beads."</span>,</span>
<span id="cb1-1218"><a href="#cb1-1218" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation facilitates removal of unwanted components."</span>,</span>
<span id="cb1-1219"><a href="#cb1-1219" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures complete binding of DNA to the beads."</span>,</span>
<span id="cb1-1220"><a href="#cb1-1220" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Careful removal prevents disturbing the bound DNA."</span>,</span>
<span id="cb1-1221"><a href="#cb1-1221" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ethanol washes remove contaminants and salts."</span>,</span>
<span id="cb1-1222"><a href="#cb1-1222" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Allows ethanol to penetrate and wash the beads."</span>,</span>
<span id="cb1-1223"><a href="#cb1-1223" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes the ethanol wash."</span>,</span>
<span id="cb1-1224"><a href="#cb1-1224" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Second wash further improves purity."</span>,</span>
<span id="cb1-1225"><a href="#cb1-1225" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Removes residual ethanol, which can interfere with downstream steps."</span>,</span>
<span id="cb1-1226"><a href="#cb1-1226" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Allows residual ethanol to evaporate."</span>,</span>
<span id="cb1-1227"><a href="#cb1-1227" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Elution buffer releases the purified DNA from the beads."</span>,</span>
<span id="cb1-1228"><a href="#cb1-1228" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Incubation allows the DNA to fully rehydrate and elute from the beads."</span>,</span>
<span id="cb1-1229"><a href="#cb1-1229" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Magnetic separation collects the beads, leaving the purified DNA in the supernatant."</span>,</span>
<span id="cb1-1230"><a href="#cb1-1230" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Collects the purified, barcoded DNA for further processing."</span>,</span>
<span id="cb1-1231"><a href="#cb1-1231" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensures proper identification of the sample."</span></span>
<span id="cb1-1232"><a href="#cb1-1232" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb1-1233"><a href="#cb1-1233" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1234"><a href="#cb1-1234" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">Troubleshooting</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">c</span>(</span>
<span id="cb1-1235"><a href="#cb1-1235" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure beads are well-mixed and at room temperature. Use the correct bead-to-sample ratio."</span>,</span>
<span id="cb1-1236"><a href="#cb1-1236" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Mix gently to avoid foaming or DNA damage."</span>,</span>
<span id="cb1-1237"><a href="#cb1-1237" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a thermomixer or rotator for consistent mixing."</span>,</span>
<span id="cb1-1238"><a href="#cb1-1238" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete pelleting of beads before removing supernatant."</span>,</span>
<span id="cb1-1239"><a href="#cb1-1239" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and avoid disturbing the beads."</span>,</span>
<span id="cb1-1240"><a href="#cb1-1240" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Remove supernatant carefully to avoid losing beads."</span>,</span>
<span id="cb1-1241"><a href="#cb1-1241" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use fresh 70% ethanol."</span>,</span>
<span id="cb1-1242"><a href="#cb1-1242" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid excessive agitation, which can dislodge the beads."</span>,</span>
<span id="cb1-1243"><a href="#cb1-1243" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Remove supernatant completely without disturbing the beads."</span>,</span>
<span id="cb1-1244"><a href="#cb1-1244" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Repeat washing if the supernatant is still cloudy."</span>,</span>
<span id="cb1-1245"><a href="#cb1-1245" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Ensure complete removal of ethanol before elution. Avoid over-drying the beads."</span>,</span>
<span id="cb1-1246"><a href="#cb1-1246" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Do not over-dry the beads, as this can make elution difficult."</span>,</span>
<span id="cb1-1247"><a href="#cb1-1247" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use the correct volume of elution buffer. Ensure complete resuspension of the beads."</span>,</span>
<span id="cb1-1248"><a href="#cb1-1248" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Incubate at the correct temperature and time."</span>,</span>
<span id="cb1-1249"><a href="#cb1-1249" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a strong magnet and allow sufficient time for clear eluate."</span>,</span>
<span id="cb1-1250"><a href="#cb1-1250" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Avoid transferring any beads with the eluate."</span>,</span>
<span id="cb1-1251"><a href="#cb1-1251" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Use a marker that won't smudge or fade when exposed to ethanol."</span></span>
<span id="cb1-1252"><a href="#cb1-1252" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-1253"><a href="#cb1-1253" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-1254"><a href="#cb1-1254" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1255"><a href="#cb1-1255" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(cleanup_barcoded, <span class="at">format =</span> <span class="st">"html"</span>, <span class="at">escape =</span> <span class="cn">FALSE</span>, <span class="at">col.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1256"><a href="#cb1-1256" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="fu">c</span>(<span class="st">"striped"</span>, <span class="st">"hover"</span>), <span class="at">full_width =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1257"><a href="#cb1-1257" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1258"><a href="#cb1-1258" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">2</span>, <span class="at">width =</span> <span class="st">"40%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1259"><a href="#cb1-1259" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_spec</span>(<span class="dv">3</span>, <span class="at">width =</span> <span class="st">"30%"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1260"><a href="#cb1-1260" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>, <span class="at">background =</span> <span class="st">"#f2f2f2"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-1261"><a href="#cb1-1261" aria-hidden="true" tabindex="-1"></a>  <span class="fu">add_header_above</span>(<span class="fu">c</span>(<span class="st">"Step"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Mechanistic Details"</span> <span class="ot">=</span> <span class="dv">1</span>, <span class="st">"Troubleshooting"</span> <span class="ot">=</span> <span class="dv">1</span>))</span>
<span id="cb1-1262"><a href="#cb1-1262" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb1-1263"><a href="#cb1-1263" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1264"><a href="#cb1-1264" aria-hidden="true" tabindex="-1"></a><span class="fu">### SISPA vs Maxima H Minus RT/SSP (MMLV RT)  </span></span>
<span id="cb1-1265"><a href="#cb1-1265" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb1-1266"><a href="#cb1-1266" aria-hidden="true" tabindex="-1"></a>MMLV RT just make ssRNA to ds-cDNA. But for this, we need to add poly-A tail first. On the other hand, SISPA protocol has both ds-cDNA making step and amplification step of the cDNA using standard PCR protocol. So, we will have pre-amplified DNA samples out of the starting RNA material. And we don't need 3´ poly-A tailing since the primers are random hexamers (with barcode) here in this protocol.  </span>
<span id="cb1-1267"><a href="#cb1-1267" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1268"><a href="#cb1-1268" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1269"><a href="#cb1-1269" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: Data Analysis</span></span>
<span id="cb1-1270"><a href="#cb1-1270" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1271"><a href="#cb1-1271" aria-hidden="true" tabindex="-1"></a>We knew the concentration/titer of the (Usutu) virus used to spike the mosquito pools. So, we have an expectation of the concentration of viruses in our samples. I tried to do the calculation for the expectation and compared with the predicted ones.</span>
<span id="cb1-1272"><a href="#cb1-1272" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1273"><a href="#cb1-1273" aria-hidden="true" tabindex="-1"></a>Different steps of the genomic pipeline is done separately. Working on making the automated snakemake pipeline making.</span>
<span id="cb1-1274"><a href="#cb1-1274" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1275"><a href="#cb1-1275" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: FlowCell Count Issue</span></span>
<span id="cb1-1276"><a href="#cb1-1276" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1277"><a href="#cb1-1277" aria-hidden="true" tabindex="-1"></a>It is fine. We used same FlowCell for multiple times (e.g. all LMD sequencing were done on 1 FlowCell, WNV_sup sample was run on the FlowCell used for the 5RNAvir cocktail run)</span>
<span id="cb1-1278"><a href="#cb1-1278" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1279"><a href="#cb1-1279" aria-hidden="true" tabindex="-1"></a><span class="fu">## Troubleshooting: Contamination Issue</span></span>
<span id="cb1-1280"><a href="#cb1-1280" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-1281"><a href="#cb1-1281" aria-hidden="true" tabindex="-1"></a>I used the DCS reagent (1 µL) for all the samples using the LMD sequencing protocol. I should have use the direct cDNA sequencing protocol (SQK-DCS109) where I was supposed to use 1 µL more of the sample omitting the use of the DCS.</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->



</body></html>